<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.39">
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
<title>Appendix H — Hypothesis Test B – Is Latitude Associated with Chronotype?</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>

<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../qmd/supplementary-material-9.html" rel="next">
<link href="../qmd/supplementary-material-7.html" rel="prev">
<link href="../images/favicon.png" rel="icon" type="image/png">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-2ade4c202a29add217eadf66f1be2696.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap-f34ee0daa61d89cdb8324c9dd5a46643.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light"><script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script><script type="application/json" class="js-hypothesis-config">
{
  "theme": "clean",
  "openSidebar": false
}
</script><script async="" src="https://hypothes.is/embed.js"></script><script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script><meta property="og:title" content="Appendix H — Hypothesis Test B – Is Latitude Associated with Chronotype?">
<meta property="og:description" content="">
<meta property="og:image" content="/mastersthesis/images/og-image.png">
<meta property="og:image:width" content="1600">
<meta property="og:image:height" content="2400">
<meta property="og:image:alt " content="Is Latitude Associated with Chronotype?">
<link href="../site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet">
<script src="../site_libs/pagedtable-1.1/js/pagedtable.js"></script><script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script><script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script><script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>
<meta property="og:title" content="[Appendix H — Hypothesis Test B]{#sec-sm-hypothesis-test-b .quarto-section-identifier} – Is Latitude Associated with Chronotype?">
<meta property="og:description" content="This document presents the analysis for testing the thesis hypothesis (Test B) using the methodology outlined in Supplementary Material sec-sm-methods.">
<meta property="og:image" content="https://danielvartan.github.io/mastersthesis/qmd/supplementary-material-8_files/figure-html/unnamed-chunk-28-1.png">
<meta property="og:site_name" content="Is Latitude Associated with Chronotype?">
<meta property="og:image:height" content="960">
<meta property="og:image:width" content="1344">
<meta name="citation_title" content="[Appendix H — Hypothesis Test B]{#sec-sm-hypothesis-test-b .quarto-section-identifier}">
<meta name="citation_fulltext_html_url" content="https://danielvartan.github.io/mastersthesis/">
<meta name="citation_pdf_url" content="https://danielvartan.github.io/mastersthesis/print/index.pdf">
<meta name="citation_doi" content="10.17605/OSF.IO/YGKTS">
<meta name="citation_language" content="en">
<meta name="citation_dissertation_institution" content="University of São Paulo">
<meta name="citation_reference" content="citation_title=Quarto;,citation_abstract=Quarto is an open-source scientific and technical publishing system built on Pandoc.;,citation_author=J. J. Allaire;,citation_author=Charles Teague;,citation_author=Yihui Xie;,citation_author=Christophe Dervieux;,citation_fulltext_html_url=https://quarto.org;,citation_doi=10.5281/ZENODO.5960048;,citation_publisher=Zenodo;">
<meta name="citation_reference" content="citation_title=Understanding regression analysis;,citation_author=Michael Patrick Allen;,citation_publication_date=1997;,citation_cover_date=1997;,citation_year=1997;,citation_isbn=978-0-306-45648-0;">
<meta name="citation_reference" content="citation_title=Asymptotic theory of certain &amp;amp;amp;quot;goodness of fit&amp;quot; criteria based on stochastic processes;,citation_abstract=The statistical problem treated is that of testing the hypothesis that n independent, identically distributed random variables have a specified continuous distribution function F(x). If Fn(x) is the empirical cumulative distribution function and ψ(t) is some nonnegative weight function (0 ≤ t ≤ 1), we consider $n^{\frac{1}{2}} \sup_{-\infty};,citation_author=T. W. Anderson;,citation_author=D. A. Darling;,citation_publication_date=1952;,citation_cover_date=1952;,citation_year=1952;,citation_fulltext_html_url=https://www.jstor.org/stable/2236446;,citation_issue=2;,citation_issn=0003-4851;,citation_volume=23;,citation_language=en-US;,citation_journal_title=The Annals of Mathematical Statistics;,citation_publisher=Institute of Mathematical Statistics;">
<meta name="citation_reference" content="citation_title=A test of goodness of fit;,citation_abstract=Some (large sample) significance points are tabulated for a distribution-free test of goodness of fit which was introduced earlier by the authors. The test, which uses the actual observations without grouping, is sensitive to discrepancies at the tails of the distribution rather than near the median. An illustration is given, using a numerical example used previously by Birnbaum in illustrating the Kolmogorov test.;,citation_author=T. W. Anderson;,citation_author=D. A. Darling;,citation_publication_date=1954-12-01;,citation_cover_date=1954-12-01;,citation_year=1954;,citation_fulltext_html_url=https://www.tandfonline.com/doi/abs/10.1080/01621459.1954.10501232;,citation_issue=268;,citation_doi=10.1080/01621459.1954.10501232;,citation_issn=0162-1459;,citation_volume=49;,citation_language=en-US;,citation_journal_title=Journal of the American Statistical Association;,citation_publisher=ASA Website;">
<meta name="citation_reference" content="citation_title=On the distribution of the two-sample Cramér-von Mises criterion;,citation_abstract=The Cramer-von Mises $\omega^2$ criterion for testing that a sample, $x_1, \cdots, x_N$, has been drawn from a specified continuous distribution $F(x)$ is \begin{equation*}\tag{1}\omega^2 = \int^\infty_{-\infty} \lbrack F_N(x) - F(x)\rbrack^2 dF(x),\end{equation*} where $F_N(x)$ is the empirical distribution function of the sample; that is, $F_N(x) = k/N$ if exactly $k$ observations are less than or equal to $x(k = 0, 1, \cdots, N)$. If there is a second sample, $y_1, \cdots, y_M$, a test of the hypothesis that the two samples come from the same (unspecified) continuous distribution can be based on the analogue of $N\omega^2$, namely \begin{equation*}\tag{2} T = \lbrack NM/(N + M)\rbrack \int^\infty_{-\infty} \lbrack F_N(x) - G_M(x)\rbrack^2 dH_{N+M}(x),\end{equation*} where $G_M(x)$ is the empirical distribution function of the second sample and $H_{N+M}(x)$ is the empirical distribution function of the two samples together [that is, $(N + M)H_{N+M}(x) = NF_N(x) + MG_M(x)\rbrack$. The limiting distribution of $N\omega^2$ as $N \rightarrow \infty$ has been tabulated [2], and it has been shown ([3], [4a], and [7]) that $T$ has the same limiting distribution as $N \rightarrow \infty, M \rightarrow \infty$, and $N/M \rightarrow \lambda$, where $\lambda$ is any finite positive constant. In this note we consider the distribution of $T$ for small values of $N$ and $M$ and present tables to permit use of the criterion at some conventional significance levels for small values of $N$ and $M$. The limiting distribution seems a surprisingly good approximation to the exact distribution for moderate sample sizes (corresponding to the same feature for $N\omega^2$ [6]). The accuracy of approximation is better than in the case of the two-sample Kolmogorov-Smirnov statistic studied by Hodges [4].;,citation_author=T. W. Anderson;,citation_publication_date=1962-09;,citation_cover_date=1962-09;,citation_year=1962;,citation_fulltext_html_url=https://projecteuclid.org/journals/annals-of-mathematical-statistics/volume-33/issue-3/On-the-Distribution-of-the-Two-Sample-Cramer-von-Mises/10.1214/aoms/1177704477.full;,citation_issue=3;,citation_doi=10.1214/aoms/1177704477;,citation_issn=0003-4851, 2168-8990;,citation_volume=33;,citation_language=en-US;,citation_journal_title=The Annals of Mathematical Statistics;,citation_publisher=Institute of Mathematical Statistics;">
<meta name="citation_reference" content="citation_title=Predictive models aren’t for causal inference;,citation_author=Suchinta Arif;,citation_author=M. Aaron MacNeil;,citation_publication_date=2022-08;,citation_cover_date=2022-08;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/ele.14033;,citation_issue=8;,citation_doi=10.1111/ele.14033;,citation_issn=1461-023X, 1461-0248;,citation_volume=25;,citation_language=en-US;,citation_journal_title=Ecology Letters;,citation_journal_abbrev=Ecology Letters;">
<meta name="citation_reference" content="citation_title=Exogenous and endogenous components in circadian rhythms;,citation_abstract=Excerpt PREFACE The main topic of the symposium “Biological Clocks” says nothing about the types of clocks we will be concerned with. It is an open question whether the clocks run continuously or stop after one revolution and have to be started anew. Also the term “clock” does not imply that one revolution is finished in 24 hours. Watches are instruments to measure time. Organisms have to measure time-spans of quite different lengths and for different purposes. Each measurement of speed, for instance, needs timing and mostly within the limits of milliseconds. One can expect, therefore, that organisms possess several clocks with perhaps extremely different periods [1]. There is no need for these clocks to run continuously; for some purposes it would be sufficient if the clock were started always at the beginning of timing (principle of a sandglass, stop watch). “Biological clocks” therefore is a more general concept than what...;,citation_author=Jürgen Aschoff;,citation_publication_date=1960-01-01;,citation_cover_date=1960-01-01;,citation_year=1960;,citation_fulltext_html_url=http://symposium.cshlp.org/content/25/11;,citation_doi=10.1101/SQB.1960.025.01.004;,citation_issn=0091-7451, 1943-4456;,citation_volume=25;,citation_language=en-US;,citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;,citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;,citation_publisher=Cold Spring Harbor Laboratory Press;">
<meta name="citation_reference" content="citation_title=Phasenlage der Tagesperiodik in Abhängigkeit von Jahreszeit und Breitengrad;,citation_abstract=Es wird die Hypothese vertreten, daß die regelhaften jahreszeitlichen Änderungen der Phasenbeziehung zwischen Aktivitätsrhythmik und natürlichem Licht-Dunkel-Wechsel auf der Grundlage spezieller Synchronisations-mechanismen erklärt werden können. Im Teil 1.1 werden die endogenen und exogenen Faktoren besprochen, die theoretisch die Phase zu beeinflussen vermögen. Die Überlegungen führen zu dem Schluß, daß das Licht-Dunkel-Verhältnis und die Dauer der Dämmerung im wesentlichen die Phase bestimmen. Experimentelle Belege sind in Teil 1.2 enthalten. In Teil 2 werden Freilandbeobachtungen über Beginn und Ende der Aktivität von Vögeln neu ausgewertet. Diese Ergebnisse, wie auch Meßwerte der Aktivität von Tieren in Innenräumen (Teil 3), stützen die Hypothese. Sie zeigen alle maximal voreilende Phasen im Mittsommer bei tagaktiven Arten bzw. im Mittwinter bei einer nachtaktiven Art. Einige, in Teil 4 erwähnte Ausnahmen von dieser Regel, werfen die Frage auf, in welchem Ausmaß die Ergebnisse verallgemeinert werden dürfen. In der Arbeit werden auch die einheitlichen jahreszeitlichen Änderungen der Aktivitätszeit erörtert.;,citation_author=Jürgen Aschoff;,citation_publication_date=1969-08-01;,citation_cover_date=1969-08-01;,citation_year=1969;,citation_fulltext_html_url=https://doi.org/10.1007/BF00416979;,citation_issue=2;,citation_doi=10.1007/BF00416979;,citation_issn=1432-1939;,citation_volume=3;,citation_language=de-DE;,citation_journal_title=Oecologia;,citation_journal_abbrev=Oecologia;">
<meta name="citation_reference" content="citation_title=Precision of entrained circadian activity rhythms under natural photoperiodic conditions;,citation_author=Jürgen Aschoff;,citation_author=Serge Daan;,citation_author=J. Figala;,citation_author=K. Müller;,citation_publication_date=1972;,citation_cover_date=1972;,citation_year=1972;,citation_issue=6;,citation_volume=59;,citation_language=en-US;,citation_journal_title=Naturwissenschaften;,citation_publisher=Springer Verlag;">
<meta name="citation_reference" content="citation_title=Biological rhythms;,citation_editor=Jürgen Aschoff;,citation_publication_date=1981;,citation_cover_date=1981;,citation_year=1981;,citation_fulltext_html_url=http://link.springer.com/10.1007/978-1-4615-6552-9;,citation_doi=10.1007/978-1-4615-6552-9;,citation_isbn=978-1-4615-6552-9;,citation_language=en-US;,citation_series_title=Handbook of Behavioral Neurobiology;">
<meta name="citation_reference" content="citation_title=Temporal orientation: Circadian clocks in animals and humans;,citation_author=Jürgen Aschoff;,citation_publication_date=1989-06;,citation_cover_date=1989-06;,citation_year=1989;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/0003347289901322;,citation_doi=10.1016/0003-3472(89)90132-2;,citation_issn=00033472;,citation_volume=37;,citation_language=en-US;,citation_journal_title=Animal Behaviour;,citation_journal_abbrev=Animal Behaviour;">
<meta name="citation_reference" content="citation_title=Circadian temporal adaptation and the perception of time;,citation_author=Jürgen Aschoff;,citation_publication_date=1989-08-01;,citation_cover_date=1989-08-01;,citation_year=1989;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/0167876089900718;,citation_issue=2;,citation_doi=10.1016/0167-8760(89)90071-8;,citation_issn=0167-8760;,citation_volume=7;,citation_language=en-US;,citation_journal_title=International Journal of Psychophysiology;,citation_journal_abbrev=International Journal of Psychophysiology;">
<meta name="citation_reference" content="citation_title=Regression diagnostics: Identifying influential data and sources of collinearity;,citation_abstract=The Wiley-Interscience Paperback Series consists of selected books that have been made more accessible to consumers in an effort to increase global appeal and general circulation. With these new unabridged softcover volumes, Wiley hopes to extend the lives of these works by making them available to future generations of statisticians, mathematicians, and scientists. &amp;amp;amp;quot;The title of the book more or less sums up the contents. It appears to me to represent a real breakthrough in the art of dealing in “unconventional” data. . . . I found the whole book both readable and enjoyable. It is suitable for data analysts, academic statisticians, and professional software writers.&amp;quot; –Journal of the Royal Statistical Society &quot;The book assumes a working knowledge of all of the principal results and techniques used in least squares multiple regression, as expressed in vector and matrix notation. Given this background, the book is clear and easy to use. . . . The techniques are illustrated in great detail with practical data sets from econometrics.&quot; –Short Book Reviews, International Statistical Institute Regression Diagnostics: Identifying Influential Data and Sources of Collinearity provides practicing statisticians and econometricians with new tools for assessing quality and reliability of regression estimates. Diagnostic techniques are developed that aid in the systematic location of data points that are unusual or inordinately influential; measure the presence and intensity of collinear relations among the regression data; and help to identify variables involved in each and pinpoint estimated coefficients potentially most adversely affected. The book emphasizes diagnostics and includes suggestions for remedial action.;,citation_author=David A. Belsley;,citation_author=Edwin Kuh;,citation_author=Roy E. Welsch;,citation_publication_date=2004;,citation_cover_date=2004;,citation_year=2004;,citation_doi=10.1002/0471725153;,citation_isbn=0-471-69117-8;,citation_language=en-US;,citation_series_title=Wiley Series in Probability and Statistics;">
<meta name="citation_reference" content="citation_title=Efficient tests for normality, homoscedasticity and serial independence of regression residuals: Monte Carlo evidence;,citation_abstract=In this paper we study the performance of various tests for normality (N), homoscedasticity (H) and serial independence (I) of regression residuals (u) under one, two and three directional departures from HO:u∼NHI.;,citation_author=Anil K. Bera;,citation_author=Carlos M. Jarque;,citation_publication_date=1981-01-01;,citation_cover_date=1981-01-01;,citation_year=1981;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/0165176581900355;,citation_issue=4;,citation_doi=10.1016/0165-1765(81)90035-5;,citation_issn=0165-1765;,citation_volume=7;,citation_language=en-US;,citation_journal_title=Economics Letters;,citation_journal_abbrev=Economics Letters;">
<meta name="citation_reference" content="citation_title=suntools: Calculate sun position, sunrise, sunset, solar noon and twilight;,citation_abstract=Provides a set of convenient functions for calculating sun-related information, including the sun’s position (elevation and azimuth), and the times of sunrise, sunset, solar noon, and twilight for any given geographical location on Earth. These calculations are based on equations provided by the National Oceanic &amp;amp;amp; Atmospheric Administration (NOAA) $&amp;lt;$https://gml.noaa.gov/grad/solcalc/calcdetails.html$&amp;gt;$ as described in &amp;quot;Astronomical Algorithms&amp;quot; by Jean Meeus (1991, ISBN: 978-0-943396-35-4).;,citation_author=Roger Bivand;,citation_author=Sebastian Luque;,citation_fulltext_html_url=https://doi.org/10.32614/CRAN.package.suntools;,citation_doi=10.32614/CRAN.package.suntools;">
<meta name="citation_reference" content="citation_title=Latitude and the human circadian system;,citation_author=J. G. Bohlen;,citation_author=undefined Simpson;,citation_editor=J. N. Mills;,citation_publication_date=1973;,citation_cover_date=1973;,citation_year=1973;,citation_fulltext_html_url=http://link.springer.com/10.1007/978-1-4613-4565-7;,citation_doi=10.1007/978-1-4613-4565-7;,citation_isbn=978-1-4613-4565-7;,citation_language=en-US;,citation_inbook_title=Biological aspects of circadian rhythms;">
<meta name="citation_reference" content="citation_title=A test of normality with high uniform power;,citation_abstract=Kurtosis can be measured in more than one way. A modification of Geary’s measure of kurtosis is shown to be more sensitive to kurtosis in the center of the distribution while Pearson’s measure of kurtosis is more sensitive to kurtosis in the tails of the distribution. The modified Geary measure and the Pearson measure are used to define a joint test of kurtosis that has high uniform power across a very wide range of symmetric nonnormal distributions.;,citation_author=Douglas G Bonett;,citation_author=Edith Seier;,citation_publication_date=2002-09-28;,citation_cover_date=2002-09-28;,citation_year=2002;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0167947302000749;,citation_issue=3;,citation_doi=10.1016/S0167-9473(02)00074-9;,citation_issn=0167-9473;,citation_volume=40;,citation_language=en-US;,citation_journal_title=Computational Statistics &amp;amp;amp; Data Analysis;,citation_journal_abbrev=Computational Statistics &amp;amp; Data Analysis;">
<meta name="citation_reference" content="citation_title=A two process model of sleep regulation;,citation_author=Alexander A. Borbély;,citation_publication_date=1982;,citation_cover_date=1982;,citation_year=1982;,citation_fulltext_html_url=https://pubmed.ncbi.nlm.nih.gov/7185792;,citation_issue=3;,citation_issn=0721-9075;,citation_volume=1;,citation_language=en-US;,citation_journal_title=Human Neurobiology;,citation_journal_abbrev=Hum Neurobiol;">
<meta name="citation_reference" content="citation_title=The two-process model of sleep regulation: A reappraisal;,citation_author=Alexander A. Borbély;,citation_author=Serge Daan;,citation_author=Anna Wirz-Justice;,citation_author=Tom Deboer;,citation_publication_date=2016-04;,citation_cover_date=2016-04;,citation_year=2016;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/jsr.12371;,citation_issue=2;,citation_doi=10.1111/jsr.12371;,citation_issn=09621105;,citation_volume=25;,citation_language=en-US;,citation_journal_title=Journal of Sleep Research;,citation_journal_abbrev=J Sleep Res;">
<meta name="citation_reference" content="citation_title=Distribution of residual autocorrelations in autoregressive-integrated moving average time series models;,citation_abstract=Many statistical models, and in particular autoregressive—moving average time series models, can be regarded as means of transforming the data to white noise, that is, to an uncorrected sequence of errors. If the parameters are known exactly, this random sequence can be computed directly from the observations; when this calculation is made with estimates substituted for the true parameter values, the resulting sequence is referred to as the “residuals,” which can be regarded as estimates of the errors. If the appropriate model has been chosen, there will be zero autocorrelation in the errors. In checking adequacy of fit it is therefore logical to study the sample autocorrelation function of the residuals. For large samples the residuals from a correctly fitted model resemble very closely the true errors of the process; however, care is needed in interpreting the serial correlations of the residuals. It is shown here that the residual autocorrelations are to a close approximation representable as a singular linear transformation of the autocorrelations of the errors so that they possess a singular normal distribution. Failing to allow for this results in a tendency to overlook evidence of lack of fit. Tests of fit and diagnostic checks are devised which take these facts into account.;,citation_author=G. E. P. Box;,citation_author=David A. Pierce;,citation_publication_date=1970-12-01;,citation_cover_date=1970-12-01;,citation_year=1970;,citation_fulltext_html_url=https://www.tandfonline.com/doi/abs/10.1080/01621459.1970.10481180;,citation_issue=332;,citation_doi=10.1080/01621459.1970.10481180;,citation_issn=0162-1459;,citation_volume=65;,citation_language=en-US;,citation_journal_title=Journal of the American Statistical Association;,citation_publisher=ASA Website;">
<meta name="citation_reference" content="citation_title=Action spectrum for melatonin regulation in humans: Evidence for a novel circadian photoreceptor;,citation_abstract=The photopigment in the human eye that transduces light for circadian and neuroendocrine regulation, is unknown. The aim of this study was to establish an action spectrum for light-induced melatonin suppression that could help elucidate the ocular photoreceptor system for regulating the human pineal gland. Subjects (37 females, 35 males, mean age of 24.5 ± 0.3 years) were healthy and had normal color vision. Full-field, monochromatic light exposures took place between 2:00 and 3:30 A.M. while subjects’ pupils were dilated. Blood samples collected before and after light exposures were quantified for melatonin. Each subject was tested with at least seven different irradiances of one wavelength with a minimum of 1 week between each nighttime exposure. Nighttime melatonin suppression tests (n = 627) were completed with wavelengths from 420 to 600 nm. The data were fit to eight univariant, sigmoidal fluence–response curves (R2 = 0.81–0.95). The action spectrum constructed from these data fit an opsin template (R2 = 0.91), which identifies 446–477 nm as the most potent wavelength region providing circadian input for regulating melatonin secretion. The results suggest that, in humans, a single photopigment may be primarily responsible for melatonin suppression, and its peak absorbance appears to be distinct from that of rod and cone cell photopigments for vision. The data also suggest that this new photopigment is retinaldehyde based. These findings suggest that there is a novel opsin photopigment in the human eye that mediates circadian photoreception.;,citation_author=George C. Brainard;,citation_author=John P. Hanifin;,citation_author=Jeffrey M. Greeson;,citation_author=Brenda Byrne;,citation_author=Gena Glickman;,citation_author=Edward Gerner;,citation_author=Mark D. Rollag;,citation_publication_date=2001-08-15;,citation_cover_date=2001-08-15;,citation_year=2001;,citation_fulltext_html_url=https://www.jneurosci.org/content/21/16/6405;,citation_issue=16;,citation_doi=10.1523/JNEUROSCI.21-16-06405.2001;,citation_issn=0270-6474, 1529-2401;,citation_volume=21;,citation_language=en-US;,citation_journal_title=Journal of Neuroscience;,citation_journal_abbrev=J. Neurosci.;,citation_publisher=Society for Neuroscience;">
<meta name="citation_reference" content="citation_title=A simple test for heteroscedasticity and random coefficient variation;,citation_abstract=A simple test for heteroscedastic disturbances in a linear regression model is developed using the framework of the Lagrangian multiplier test. For a wide range of heteroscedastic and random coefficient specifications, the criterion is given as a readily computed function of the OLS residuals. Some finite sampleevidence is presented to supplement the general asymptotic properties of Lagrangian multiplier tests.;,citation_author=T. S. Breusch;,citation_author=A. R. Pagan;,citation_publication_date=1979;,citation_cover_date=1979;,citation_year=1979;,citation_fulltext_html_url=https://www.jstor.org/stable/1911963;,citation_issue=5;,citation_doi=10.2307/1911963;,citation_issn=0012-9682;,citation_volume=47;,citation_language=en-US;,citation_journal_title=Econometrica;,citation_publisher=[Wiley, Econometric Society];">
<meta name="citation_reference" content="citation_title=Molecular components of the mammalian circadian clock;,citation_author=Ethan D. Buhr;,citation_author=Joseph S. Takahashi;,citation_editor=Achim Kramer;,citation_editor=Martha Merrow;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_fulltext_html_url=http://link.springer.com/10.1007/978-3-642-25950-0_1;,citation_doi=10.1007/978-3-642-25950-0_1;,citation_isbn=978-3-642-25949-4;,citation_volume=217;,citation_inbook_title=Circadian Clocks;,citation_series_title=Handbook of Experimental Pharmacology;">
<meta name="citation_reference" content="citation_title={pwrss}: Statistical power and sample size calculation;,citation_abstract=Statistical power and minimum required sample size calculations for (1) testing a proportion (one-sample) against a constant, (2) testing a mean (one-sample) against a constant, (3) testing difference between two proportions (independent samples), (4) testing difference between two means or groups (parametric and non-parametric tests for independent and paired samples), (5) testing a correlation (one-sample) against a constant, (6) testing difference between two correlations (independent samples), (7) testing a single coefficient in multiple linear regression, logistic regression, and Poisson regression (with standardized or unstandardized coefficients, with no covariates or covariate adjusted), (8) testing an indirect effect (with standardized or unstandardized coefficients, with no covariates or covariate adjusted) in the mediation analysis (Sobel, Joint, and Monte Carlo tests), (9) testing an R-squared against zero in linear regression, (10) testing an R-squared difference against zero in hierarchical regression, (11) testing an eta-squared or f-squared (for main and interaction effects) against zero in analysis of variance (could be one-way, two-way, and three-way), (12) testing an eta-squared or f-squared (for main and interaction effects) against zero in analysis of covariance (could be one-way, two-way, and three-way), (13) testing an eta-squared or f-squared (for between, within, and interaction effects) against zero in one-way repeated measures analysis of variance (with non-sphericity correction and repeated measures correlation), and (14) testing goodness-of-fit or independence for contingency tables. Alternative hypothesis can be formulated as &amp;amp;amp;quot;not equal&amp;quot;, &quot;less&quot;, &quot;greater&quot;, &quot;non-inferior&quot;, &quot;superior&quot;, or &quot;equivalent&quot; in (1), (2), (3), and (4); as &quot;not equal&quot;, &quot;less&quot;, or &quot;greater&quot; in (5), (6), (7) and (8); but always as &quot;greater&quot; in (9), (10), (11), (12), (13), and (14).;,citation_author=Metin Bulus;,citation_fulltext_html_url=https://CRAN.R-project.org/package=pwrss;,citation_doi=10.32614/CRAN.package.pwrss;">
<meta name="citation_reference" content="citation_title=tidygeocoder: An R package for geocoding;,citation_abstract=Tidygeocoder is a package for the R programming language that allows researchers and analysts to easily perform geocoding. Geocoding (also called “forward geocoding”) is the process of obtaining geographic coordinates (longitude and latitude) from an address or a place name, while reverse geocoding is the process of obtaining an address or place name from geographic coordinates.;,citation_author=Jesse Cambon;,citation_author=Diego Hernangómez;,citation_author=Christopher Belanger;,citation_author=Daniel Possenriede;,citation_publication_date=2021-09-09;,citation_cover_date=2021-09-09;,citation_year=2021;,citation_fulltext_html_url=https://joss.theoj.org/papers/10.21105/joss.03544;,citation_issue=65;,citation_doi=10.21105/joss.03544;,citation_issn=2475-9066;,citation_volume=6;,citation_language=en-US;,citation_journal_title=Journal of Open Source Software;,citation_journal_abbrev=JOSS;">
<meta name="citation_reference" content="citation_title=Cambridge dictionary;,citation_abstract=The most popular dictionary and thesaurus for learners of English. Meanings and definitions of words with pronunciations and translations.;,citation_author=Cambridge University Press;,citation_fulltext_html_url=https://dictionary.cambridge.org/;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Regression analysis by example;,citation_abstract=This fifth edition has been expanded and thoroughly updated to reflect recent advances in the field. The emphasis continues to be on exploratory data analysis rather than statistical theory. The coverage offers in-depth treatment of regression diagnostics, transformation, multicollinearity, logistic regression, and robust regression. Methods of regression analysis are clearly demonstrated, and examples containing the types of irregularities commonly encountered in the real world are provided. Each example isolates one or two techniques and features detailed discussions of the techniques themselves, the required assumptions, and the evaluated success of each technique.;,citation_author=Samprit Chatterjee;,citation_author=Ali S. Hadi;,citation_publication_date=2012;,citation_cover_date=2012;,citation_year=2012;,citation_isbn=978-0-470-90584-5;,citation_language=en-US;,citation_series_title=Wiley Series in Probability and Statistics;">
<meta name="citation_reference" content="citation_title=Statistical power analysis for the behavioral sciences;,citation_author=Jacob Cohen;,citation_publication_date=1988;,citation_cover_date=1988;,citation_year=1988;,citation_isbn=978-0-8058-0283-2;">
<meta name="citation_reference" content="citation_title=Things I have learned (so far);,citation_abstract=This reprinted article originally appeared in (American Psychologist, 1990, 45[12], 1304-1312). (The following abstract of the original article appeared in record 1991-11596-001.) This is an account of what I have learned (so far) about the application of statistics to psychology and the other sociobiomedical sciences. It includes the principles &amp;amp;amp;quot;less is more&amp;quot; (fewer variables, more highly targeted issues, sharp rounding off), &quot;simple is better&quot; (graphic representation, unit weighting for linear composites), and &quot;some things you learn aren’t so.&quot; I have learned to avoid the many misconceptions that surround Fisherian null hypothesis testing. I have also learned the importance of power analysis and the determination of just how big (rather than how statistically significant) are the effects that we study. Finally, I have learned that there is no royal road to statistical induction, that the informed judgment of the investigator is the crucial element in the interpretation of data, and that things take time.;,citation_author=Jacob Cohen;,citation_publication_date=1990-12;,citation_cover_date=1990-12;,citation_year=1990;,citation_issue=12;,citation_doi=10.1037/10109-028;,citation_volume=45;,citation_language=en-US;,citation_journal_title=American Psychologist;">
<meta name="citation_reference" content="citation_title=A power primer;,citation_author=Jacob Cohen;,citation_publication_date=1992;,citation_cover_date=1992;,citation_year=1992;,citation_fulltext_html_url=http://doi.apa.org/getdoi.cfm?doi=10.1037/0033-2909.112.1.155;,citation_issue=1;,citation_doi=10.1037/0033-2909.112.1.155;,citation_issn=1939-1455, 0033-2909;,citation_volume=112;,citation_language=en-US;,citation_journal_title=Psychological Bulletin;,citation_journal_abbrev=Psychological Bulletin;">
<meta name="citation_reference" content="citation_title=The earth is round (p$&amp;amp;amp;lt;$.05);,citation_abstract=After 4 decades of severe criticism, the ritual of null hypothesis significance testing (mechanical dichotomous decisions around a sacred .05 criterion) still persists. This article reviews the problems with this practice, including near universal misinterpretation of p as the probability that H₀ is false, the misinterpretation that its complement is the probability of successful replication, and the mistaken assumption that if one rejects H₀ one thereby affirms the theory that led to the test. Exploratory data analysis and the use of graphic methods, a steady improvement in and a movement toward standardization in measurement, an emphasis on estimating effect sizes using confidence intervals, and the informed use of available statistical methods are suggested. For generalization, psychologists must finally rely, as has been done in all the older sciences, on replication. (PsycINFO Database Record (c) 2016 APA, all rights reserved);,citation_author=Jacob Cohen;,citation_publication_date=1994;,citation_cover_date=1994;,citation_year=1994;,citation_issue=12;,citation_doi=10.1037/0003-066X.49.12.997;,citation_issn=1935-990X;,citation_volume=49;,citation_language=en-US;,citation_journal_title=American Psychologist;,citation_publisher=American Psychological Association;">
<meta name="citation_reference" content="citation_title=Applied multiple regression/correlation analysis for the behavioral sciences;,citation_abstract=This classic text on multiple regression is noted for its nonmathematical, applied, and data-analytic approach. Readers profit from its verbal-conceptual exposition and frequent use of examples. The applied emphasis provides clear illustrations of the principles and provides worked examples of the types of applications that are possible. Researchers learn how to specify regression models that directly address their research questions. An overview of the fundamental ideas of multiple regression and a review of bivariate correlation and regression and other elementary statistical concepts provide a strong foundation for understanding the rest of the text. The third edition features an increased emphasis on graphics and the use of confidence intervals and effect size measures, and an accompanying website with data for most of the numerical examples along with the computer code for SPSS, SAS, and SYSTAT, at www.psypress.com/9780805822236 . Applied Multiple Regression serves as both a textbook for graduate students and as a reference tool for researchers in psychology, education, health sciences, communications, business, sociology, political science, anthropology, and economics. An introductory knowledge of statistics is required. Self-standing chapters minimize the need for researchers to refer to previous chapters.;,citation_author=Jacob Cohen;,citation_author=Patricia Cohen;,citation_author=Stephen G. West;,citation_author=Leona S. Aiken;,citation_publication_date=2002-08;,citation_cover_date=2002-08;,citation_year=2002;,citation_isbn=0-8058-2223-2;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=1960: Biological clocks, vol. XXV;,citation_author=Cold Spring Harbor Laboratory;,citation_fulltext_html_url=https://symposium.cshlp.org/site/misc/topic25.xhtml;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Cours de philosophie positive: Tome premier: Les préliminaires généraux et la philosophie mathématique;,citation_author=Auguste Comte;,citation_publication_date=1892;,citation_cover_date=1892;,citation_year=1892;,citation_fulltext_html_url=http://archive.org/details/coursdephilosoph01comt;,citation_volume=1;,citation_language=fr-FR;">
<meta name="citation_reference" content="citation_title=Detection of influential observation in linear regression;,citation_author=R. Dennis Cook;,citation_publication_date=1977-02;,citation_cover_date=1977-02;,citation_year=1977;,citation_fulltext_html_url=https://www.tandfonline.com/doi/abs/10.1080/00401706.1977.10489493;,citation_issue=1;,citation_doi=10.1080/00401706.1977.10489493;,citation_issn=0040-1706;,citation_volume=19;,citation_language=en-US;,citation_journal_title=Technometrics;,citation_publisher=ASA Website;">
<meta name="citation_reference" content="citation_title=Influential observations in linear regression;,citation_abstract=Characteristics of observations which cause them to be influential in a least squares analysis are investigated and related to residual variances, residual correlations, and the convex hull of the observed values of the independent variables. It is shown how deleting an observation can substantially alter an analysis by changing the partial F-tests, the studentized residuals, the residual variances, the convex hull of the independent variables, and the estimated parameter vector. Outliers are discussed briefly, and an example is presented.;,citation_author=R. Dennis Cook;,citation_publication_date=1979-03-01;,citation_cover_date=1979-03-01;,citation_year=1979;,citation_fulltext_html_url=https://www.tandfonline.com/doi/abs/10.1080/01621459.1979.10481634;,citation_issue=365;,citation_doi=10.1080/01621459.1979.10481634;,citation_issn=0162-1459;,citation_volume=74;,citation_language=en-US;,citation_journal_title=Journal of the American Statistical Association;,citation_publisher=ASA Website;">
<meta name="citation_reference" content="citation_title=On the composition of elementary errors: First paper: Mathematical deductions;,citation_author=Harald Cramér;,citation_publication_date=1928-01-01;,citation_cover_date=1928-01-01;,citation_year=1928;,citation_fulltext_html_url=https://doi.org/10.1080/03461238.1928.10416862;,citation_issue=1;,citation_doi=10.1080/03461238.1928.10416862;,citation_issn=0346-1238;,citation_volume=1928;,citation_language=en-US;,citation_journal_title=Scandinavian Actuarial Journal;,citation_publisher=Taylor &amp;amp;amp; Francis;">
<meta name="citation_reference" content="citation_title=An omnibus test of normality for moderate and large size samples;,citation_abstract=We present a test of normality based on a statistic D which is up to a constant the ratio of Downton’s linear unbiased estimator of the population standard deviation to the sample standard deviation. For the usual levels of significance Monte Carlo simulations indicate that Cornish-Fisher expansions adequately approximate the null distribution of D if the sample size is 50 or more. The test is an omnibus test, being appropriate to detect deviations from normality due either to skewness or kurtosis. Simulation results of powers for various alternatives when the sample size is 50 indicate that the test compares favourably with the Shapiro-Wilk W test,√1, b2 and the ratio of range to standard deviation.;,citation_author=Ralph B. D’Agostino;,citation_publication_date=1971-08-01;,citation_cover_date=1971-08-01;,citation_year=1971;,citation_fulltext_html_url=https://doi.org/10.1093/biomet/58.2.341;,citation_issue=2;,citation_doi=10.1093/biomet/58.2.341;,citation_issn=0006-3444;,citation_volume=58;,citation_language=en-US;,citation_journal_title=Biometrika;,citation_journal_abbrev=Biometrika;">
<meta name="citation_reference" content="citation_title=Tests for departure from normality. Empirical results for the distributions of b2 and √b1;,citation_author=Ralph B. D’Agostino;,citation_author=E. S. Pearson;,citation_publication_date=1973;,citation_cover_date=1973;,citation_year=1973;,citation_fulltext_html_url=https://academic.oup.com/biomet/article-lookup/doi/10.1093/biomet/60.3.613;,citation_issue=3;,citation_doi=10.1093/biomet/60.3.613;,citation_issn=0006-3444, 1464-3510;,citation_volume=60;,citation_language=en-US;,citation_journal_title=Biometrika;,citation_journal_abbrev=Biometrika;">
<meta name="citation_reference" content="citation_title=A suggestion for using powerful and informative tests of normality;,citation_abstract=For testing that an underlying population is normally distributed the skewness and kurtosis statistics, $\sqrt{b_1}$ and b2, and the D’Agostino-Pearson K2 statistic that combines these two statistics have been shown to be powerful and informative tests. Their use, however, has not been as prevalent as their usefulness. We review these tests and show how readily available and popular statistical software can be used to implement them. Their relationship to deviations from linearity in normal probability plotting is also presented.;,citation_author=Ralph B. D’Agostino;,citation_author=Albert Belanger;,citation_publication_date=1990;,citation_cover_date=1990;,citation_year=1990;,citation_fulltext_html_url=https://www.jstor.org/stable/2684359;,citation_issue=4;,citation_doi=10.2307/2684359;,citation_issn=0003-1305;,citation_volume=44;,citation_journal_title=The American Statistician;,citation_publisher=[American Statistical Association, Taylor &amp;amp;amp; Francis, Ltd.];">
<meta name="citation_reference" content="citation_title=An analytic approximation to the distribution of Lilliefors’s test statistic for normality;,citation_author=Gerard E. Dallal;,citation_author=Leland Wilkinson;,citation_publication_date=1986-11;,citation_cover_date=1986-11;,citation_year=1986;,citation_fulltext_html_url=http://www.tandfonline.com/doi/abs/10.1080/00031305.1986.10475419;,citation_issue=4;,citation_doi=10.1080/00031305.1986.10475419;,citation_issn=0003-1305, 1537-2731;,citation_volume=40;,citation_language=en-US;,citation_journal_title=The American Statistician;,citation_journal_abbrev=The American Statistician;">
<meta name="citation_reference" content="citation_title=Probability and statistics;,citation_author=Morris H. DeGroot;,citation_author=Mark J. Schervish;,citation_publication_date=2012;,citation_cover_date=2012;,citation_year=2012;,citation_isbn=978-0-321-50046-5;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Sex difference in the near-24-hour intrinsic period of the human circadian timing system;,citation_abstract=The circadian rhythms of melatonin and body temperature are set to an earlier hour in women than in men, even when the women and men maintain nearly identical and consistent bedtimes and wake times. Moreover, women tend to wake up earlier than men and exhibit a greater preference for morning activities than men. Although the neurobiological mechanism underlying this sex difference in circadian alignment is unknown, multiple studies in nonhuman animals have demonstrated a sex difference in circadian period that could account for such a difference in circadian alignment between women and men. Whether a sex difference in intrinsic circadian period in humans underlies the difference in circadian alignment between men and women is unknown. We analyzed precise estimates of intrinsic circadian period collected from 157 individuals (52 women, 105 men; aged 18–74 y) studied in a month-long inpatient protocol designed to minimize confounding influences on circadian period estimation. Overall, the average intrinsic period of the melatonin and temperature rhythms in this population was very close to 24 h [24.15 ± 0.2 h (24 h 9 min ± 12 min)]. We further found that the intrinsic circadian period was significantly shorter in women [24.09 ± 0.2 h (24 h 5 min ± 12 min)] than in men [24.19 ± 0.2 h (24 h 11 min ± 12 min); P $&amp;amp;amp;lt;$ 0.01] and that a significantly greater proportion of women have intrinsic circadian periods shorter than 24.0 h (35% vs. 14%; P $&amp;lt;$ 0.01). The shorter average intrinsic circadian period observed in women may have implications for understanding sex differences in habitual sleep duration and insomnia prevalence.;,citation_author=Jeanne F. Duffy;,citation_author=Sean W. Cain;,citation_author=Anne-Marie Chang;,citation_author=Andrew J. K. Phillips;,citation_author=Mirjam Y. Münch;,citation_author=Claude Gronfier;,citation_author=James K. Wyatt;,citation_author=Derk-Jan Dijk;,citation_author=Kenneth P. Wright;,citation_author=Charles A. Czeisler;,citation_publication_date=2011-09-13;,citation_cover_date=2011-09-13;,citation_year=2011;,citation_fulltext_html_url=https://pnas.org/doi/full/10.1073/pnas.1010666108;,citation_doi=10.1073/pnas.1010666108;,citation_issn=0027-8424, 1091-6490;,citation_volume=108;,citation_language=en-US;,citation_journal_title=Proceedings of the National Academy of Sciences;,citation_journal_abbrev=Proc. Natl. Acad. Sci. U.S.A.;">
<meta name="citation_reference" content="citation_title=Testing for serial correlation in least squares regression. I;,citation_author=J. Durbin;,citation_author=G. S. Watson;,citation_publication_date=1950-12-01;,citation_cover_date=1950-12-01;,citation_year=1950;,citation_fulltext_html_url=https://doi.org/10.1093/biomet/37.3-4.409;,citation_issue=3–4;,citation_doi=10.1093/biomet/37.3-4.409;,citation_issn=0006-3444;,citation_volume=37;,citation_language=en-US;,citation_journal_title=Biometrika;,citation_journal_abbrev=Biometrika;">
<meta name="citation_reference" content="citation_title=Testing for serial correlation in least squares regression. II;,citation_author=J. Durbin;,citation_author=G. S. Watson;,citation_publication_date=1951-06-01;,citation_cover_date=1951-06-01;,citation_year=1951;,citation_fulltext_html_url=https://doi.org/10.1093/biomet/38.1-2.159;,citation_issue=1–2;,citation_doi=10.1093/biomet/38.1-2.159;,citation_issn=0006-3444;,citation_volume=38;,citation_language=en-US;,citation_journal_title=Biometrika;,citation_journal_abbrev=Biometrika;">
<meta name="citation_reference" content="citation_title=Testing for serial correlation in least squares regression. III;,citation_abstract=The paper considers a number of problems arising from the test of serial correlation based on the d statistic proposed earlier by the authors (Durbin &amp;amp;amp;amp; Watson, 1950, 1951). Methods of computing the exact distribution of d are investigated and the exact distribution is compared with six approximations to it for four sets of published data. It is found that approximations suggested by Theil and Nagar and by Hannan are too inaccurate for practical use but that the beta approximation proposed in the 1950 and 1951 papers and a new approximation, called by us the a + bdu approximation and based, like the beta approximation, on the exact first two moments of d, both perform well.The power of the d test is compared with that of certain exact tests proposed by Theil, Durbin, Koerts and Abrahamse from the standpoint of invariance theory. It is shown that the d test is locally most powerful invariant but that the other tests are not.There are three appendices. The first gives an account of the exact distribution of d. The second derives the mean and variance to a second order of approximation of a modified maximum likelihood statistic closely related to d. The third sets out details of the computations required for the a + hdu approximation.;,citation_author=J. Durbin;,citation_author=G. S. Watson;,citation_publication_date=1971-04-01;,citation_cover_date=1971-04-01;,citation_year=1971;,citation_fulltext_html_url=https://doi.org/10.1093/biomet/58.1.1;,citation_issue=1;,citation_doi=10.1093/biomet/58.1.1;,citation_issn=0006-3444;,citation_volume=58;,citation_language=en-US;,citation_journal_title=Biometrika;,citation_journal_abbrev=Biometrika;">
<meta name="citation_reference" content="citation_title=Evidence that the woman’s ovarian cycle is driven by an internal circamonthly timing system;,citation_abstract=The ovarian cycle has a well-established circa-monthly rhythm, but the mechanisms involved in its regularity are unknown. Is the rhythmicity driven by an endogenous clock-like timer or by other internal or external processes? Here, using two large epidemiological datasets (26,912 cycles from 2303 European women and 4786 cycles from 721 North American women), analyzed with time series and circular statistics, we find evidence that the rhythmic characteristics of the menstrual cycle are more likely to be explained by an endogenous clock-like driving mechanism than by any other internal or external process. We also show that the menstrual cycle is weakly but significantly influenced by the 29.5-day lunar cycle and that the phase alignment between the two cycles differs between the European and the North American populations. Given the need to find efficient treatments of subfertility in women, our results should be confirmed in larger populations, and chronobiological approaches to optimize the ovulatory cycle should be evaluated.;,citation_author=René Ecochard;,citation_author=Joseph B. Stanford;,citation_author=Richard J. Fehring;,citation_author=Marie Schneider;,citation_author=Shahpar Najmabadi;,citation_author=Claude Gronfier;,citation_publication_date=2024-04-10;,citation_cover_date=2024-04-10;,citation_year=2024;,citation_fulltext_html_url=https://www.science.org/doi/10.1126/sciadv.adg9646;,citation_issue=15;,citation_doi=10.1126/sciadv.adg9646;,citation_volume=10;,citation_journal_title=Science Advances;,citation_publisher=American Association for the Advancement of Science;">
<meta name="citation_reference" content="citation_title=The sense of time: Evidence for its molecular basis in the eukaryotic gene-action system;,citation_author=Charles F. Ehret;,citation_publication_date=1974;,citation_cover_date=1974;,citation_year=1974;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/B9780120052158500097;,citation_doi=10.1016/B978-0-12-005215-8.50009-7;,citation_isbn=978-0-12-005215-8;,citation_volume=15;,citation_language=en-US;,citation_inbook_title=Advances in Biological and Medical Physics;">
<meta name="citation_reference" content="citation_title=The essential guide to effect sizes: Statistical power, meta-analysis, and the interpretation of research results;,citation_abstract=This succinct and jargon-free introduction to effect sizes gives students and researchers the tools they need to interpret the practical significance of their results. Using a class-tested approach that includes numerous examples and step-by-step exercises, it introduces and explains three of the most important issues relating to the practical significance of research results: the reporting and interpretation of effect sizes (Part I), the analysis of statistical power (Part II), and the meta-analytic pooling of effect size estimates drawn from different studies (Part III). The book concludes with a handy list of recommendations for those actively engaged in or currently preparing research projects.;,citation_editor=Paul D. Ellis;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_isbn=978-0-521-19423-5;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Agent-based computational models and generative social science;,citation_abstract=This article argues that the agent-based computational model permits a distinctive approach to social science for which the term &amp;amp;amp;quot;generative&amp;quot; is suitable. In defending this terminology, features distinguishing the approach from both &quot;inductive&quot; and &quot;deductive&quot; science are given. Then, the following specific contributions to social science are discussed: The agent-based computational model is a new tool for empirical research. It offers a natural environment for the study of connectionist phenomena in social science. Agent-based modeling pro- vides a powerful way to address certain enduring-and especially interdisciplinary questions. It allows one to subject certain core theories-such as neoclassical microeconomics-to important types of stress (e.g., the effect of evolving preferences). It permits one to study how rules of individual behavior give rise-or &quot;map up&quot;-to macroscopic regularities and organizations. In turn, one can employ laboratory behavioral research findings to select among competing agent-based (&quot;bottom up&quot;) models. The agent-based approach may well have the important effect of decoupling individual rationality from macroscopic equilibrium and of separating decision science from social science more generally. Agent-based modeling offers powerful new forms of hybrid theo- retical-computational work; these are particularly relevant to the study of non-equilibrium systems. The agent- based approach invites the interpretation of society as a distributed computational device, and in turn the interpretation of social dynamics as a type of computation. This interpretation raises important foundational issues in social science-some related to intractability, and some to undecidability proper. Finally, since &quot;emer- gence&quot; figures prominently in this literature, I take up the connection between agent-based modeling and classical emergentism, criticizing the latter and arguing that the two are incompatible.;,citation_author=Joshua M. Epstein;,citation_publication_date=1999;,citation_cover_date=1999;,citation_year=1999;,citation_issue=5;,citation_doi=10.1002/(SICI)1099-0526(199905/06)4:5&amp;amp;lt;41::AID-CPLX9&amp;gt;3.0.CO;2-F;,citation_volume=4;,citation_language=en-US;,citation_journal_title=Complexity;">
<meta name="citation_reference" content="citation_title=Chrono-nutrition: From molecular and neuronal mechanisms to human epidemiology and timed feeding patterns;,citation_abstract=The circadian timing system governs daily biological rhythms, synchronising physiology and behaviour to the temporal world. External time cues, including the light-dark cycle and timing of food intake, provide daily signals for entrainment of the central, master circadian clock in the hypothalamic suprachiasmatic nuclei (SCN), and of metabolic rhythms in peripheral tissues, respectively. Chrono-nutrition is an emerging field building on the relationship between temporal eating patterns, circadian rhythms, and metabolic health. Evidence from both animal and human research demonstrates adverse metabolic consequences of circadian disruption. Conversely, a growing body of evidence indicates that aligning food intake to periods of the day when circadian rhythms in metabolic processes are optimised for nutrition may be effective for improving metabolic health. Circadian rhythms in glucose and lipid homeostasis, insulin responsiveness and sensitivity, energy expenditure, and postprandial metabolism, may favour eating patterns characterised by earlier temporal distribution of energy. This review details the molecular basis for metabolic clocks, the regulation of feeding behaviour, and the evidence for meal timing as an entraining signal for the circadian system in animal models. The epidemiology of temporal eating patterns in humans is examined, together with evidence from human intervention studies investigating the metabolic effects of morning compared to evening energy intake, and emerging chrono-nutrition interventions such as time-restricted feeding. Chrono-nutrition may have therapeutic application for individuals with and at-risk of metabolic disease and convey health benefits within the general population.;,citation_author=Alan Flanagan;,citation_author=David A. Bechtold;,citation_author=Gerda K. Pot;,citation_author=Jonathan D. Johnston;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/jnc.15246;,citation_issue=1;,citation_doi=10.1111/jnc.15246;,citation_issn=1471-4159;,citation_volume=157;,citation_language=en-US;,citation_journal_title=Journal of Neurochemistry;">
<meta name="citation_reference" content="citation_title=Rhythms of life: The biological clocks that control the daily lives of every living thing;,citation_author=Russell G. Foster;,citation_author=Leon Kreitzman;,citation_publication_date=2005;,citation_cover_date=2005;,citation_year=2005;,citation_isbn=978-1-86197-571-3;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Sleep, circadian rhythms and health;,citation_abstract=At the core of human thought, for the majority of individuals in the developed nations at least, there is the tacit assumption that as a species we are unfettered by the demands imposed by our biology and that we can do what we want, at whatever time we choose, whereas in reality every aspect of our physiology and behaviour is constrained by a 24 h beat arising from deep within our evolution. Our daily circadian rhythms and sleep/wake cycle allow us to function optimally in a dynamic world, adjusting our biology to the demands imposed by the day/night cycle. The themes developed in this review focus upon the growing realization that we ignore the circadian and sleep systems at our peril, and this paper considers the mechanisms that generate and regulate circadian and sleep systems; what happens mechanistically when these systems collapse as a result of societal pressures and disease; how sleep disruption and stress are linked; why sleep disruption and mental illness invariably occur together; and how individuals and employers can attempt to mitigate some of the problems associated with working against our internal temporal biology. While some of the health costs of sleep disruption can be reduced, in the short-term at least, there will always be significant negative consequences associated with shift work and sleep loss. With this in mind, society needs to address this issue and decide when the consequences of sleep disruption are justified in the workplace.;,citation_author=Russell G. Foster;,citation_publication_date=2020-04-17;,citation_cover_date=2020-04-17;,citation_year=2020;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rsfs.2019.0098;,citation_issue=3;,citation_doi=10.1098/rsfs.2019.0098;,citation_volume=10;,citation_language=en-US;,citation_journal_title=Interface Focus;,citation_publisher=Royal Society;">
<meta name="citation_reference" content="citation_title=Fundamentals of circadian entrainment by light;,citation_abstract=Light at dawn and dusk is the key signal for the entrainment of the circadian clock. Light at dusk delays the clock. Light at dawn advances the clock. The threshold for human entrainment requires relatively bright light for a long duration, but the precise irradiance/duration relationships for photoentrainment have yet to be fully defined. Photoentrainment is achieved by a network of photosensitive retinal ganglion cells (pRGCs) which utilise the short-wavelength light-sensitive photopigment, melanopsin. Although rods and cones are not required, they do play a role in photoentrainment, by projecting to and modulating the endogenous photosensitivity of the pRGCs, but in a manner that remains poorly understood. It is also important to emphasise that the age and prior light exposure of an individual will modify the efficacy of entrainment stimuli. Because of the complexity of photoreceptor interactions, attempts to develop evidence-based human centric lighting are not straightforward. We need to study how humans respond to dynamic light exposure in the “real world” where light intensity, duration, spectral quality and the time of exposure vary greatly. Defining these parameters will allow the development of electric lighting systems that will enhance human circadian entrainment.;,citation_author=Russell G. Foster;,citation_publication_date=2021-08-01;,citation_cover_date=2021-08-01;,citation_year=2021;,citation_fulltext_html_url=https://doi.org/10.1177/14771535211014792;,citation_issue=5;,citation_doi=10.1177/14771535211014792;,citation_issn=1477-1535;,citation_volume=53;,citation_language=en-US;,citation_journal_title=Lighting Research &amp;amp;amp; Technology;,citation_publisher=SAGE Publications Ltd STM;">
<meta name="citation_reference" content="citation_title=Applied regression analysis and generalized linear models;,citation_author=John Fox;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_isbn=978-1-4522-0566-3;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=The SAGE encyclopedia of research design;,citation_editor=Bruce B. Frey;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://sk.sagepub.com/reference/the-sage-encyclopedia-of-research-design-2e;,citation_doi=10.4135/9781071812082;,citation_isbn=978-1-0718-1212-9 978-1-0718-1208-2;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Phenotype and genotype: The search for influential genes;,citation_author=Florian Frommlet;,citation_author=Małgorzata Bogdan;,citation_author=David Ramsey;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_fulltext_html_url=http://link.springer.com/10.1007/978-1-4471-5310-8;,citation_doi=10.1007/978-1-4471-5310-8;,citation_isbn=978-1-4471-5309-2;,citation_volume=18;,citation_language=en-US;,citation_series_title=Computational Biology;">
<meta name="citation_reference" content="citation_title=Mindless statistics;,citation_abstract=Statistical rituals largely eliminate statistical thinking in the social sciences. Rituals are indispensable for identification with social groups, but they should be the subject rather than the procedure of science. What I call the “null ritual” consists of three steps: (1) set up a statistical null hypothesis, but do not specify your own hypothesis nor any alternative hypothesis, (2) use the 5% significance level for rejecting the null and accepting your hypothesis, and (3) always perform this procedure. I report evidence of the resulting collective confusion and fears about sanctions on the part of students and teachers, researchers and editors, as well as textbook writers.;,citation_author=Gerd Gigerenzer;,citation_publication_date=2004-11-01;,citation_cover_date=2004-11-01;,citation_year=2004;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S1053535704000927;,citation_issue=5;,citation_doi=10.1016/j.socec.2004.09.033;,citation_issn=1053-5357;,citation_volume=33;,citation_language=en-US;,citation_journal_title=The Journal of Socio-Economics;,citation_journal_abbrev=The Journal of Socio-Economics;,citation_series_title=Statistical Significance;">
<meta name="citation_reference" content="citation_title=Google Geocoding API;,citation_author=Google;,citation_fulltext_html_url=https://developers.google.com/maps/documentation/geocoding;,citation_publisher=Google;">
<meta name="citation_reference" content="citation_title=Stop testing for normality;,citation_abstract=Normality tests are misleading and a waste of your time!;,citation_author=Robert Greener;,citation_publication_date=2020-08-04;,citation_cover_date=2020-08-04;,citation_year=2020;,citation_fulltext_html_url=https://towardsdatascience.com/stop-testing-for-normality-dba96bb73f90;,citation_language=en-US;,citation_publisher=Medium;">
<meta name="citation_reference" content="citation_title=Multivariate data analysis;,citation_author=Joseph F. Hair;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_isbn=978-1-4737-5654-0;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Circannual rhythm in level and timing of serum corticosterone in standardized inbred mature C-mice;,citation_abstract=Circannual variations in serum corticosterone levels in Balbc mice are demonstrated in pooled data of profiles involving at least 6 time points of sampling per 24-hour span. High values are found during the winter months and low values in late spring and summer. Cosinor analysis of the circadian rhythm of serum corticosterone during different times of the year shows a change in the circadian acrophase (crest) from about 43° in February to 95° in May. This change is evident after a seven-day standardization span at relatively constant temperatures and a regimen of LD12:12 with light from 0600 to 1800 (for mice kept previously under largely natural lighting conditions). This circannual variation in circadian acrophase is compared with the time required for a phase shift after an abrupt change in lighting regimen: after a 180° shift of the lighting regimen, the change in the serum corticosterone acrophase reaches 45° in less than three days, over 100° in 4 days and almost a full 180° in 7 days. Therefore, the circannual variation in circadian system phase of the adrenal cycle shown in this study cannot be regarded as an incomplete phase shift such as occurs shortly after an abrupt shift of the lighting regimen. The underlying mechanisms of this presumably partly intrinsic circannual biorhythm await further study.;,citation_author=E. Haus;,citation_author=F. Halberg;,citation_publication_date=1970-03-01;,citation_cover_date=1970-03-01;,citation_year=1970;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/0013935170900083;,citation_issue=2;,citation_doi=10.1016/0013-9351(70)90008-3;,citation_issn=0013-9351;,citation_volume=3;,citation_language=en-US;,citation_journal_title=Environmental Research;,citation_journal_abbrev=Environmental Research;">
<meta name="citation_reference" content="citation_title=Complex adaptive systems;,citation_author=John H. Holland;,citation_publication_date=1992;,citation_cover_date=1992;,citation_year=1992;,citation_fulltext_html_url=https://www.jstor.org/stable/20025416;,citation_issue=1;,citation_issn=0011-5266;,citation_volume=121;,citation_journal_title=Daedalus;,citation_publisher=The MIT Press;">
<meta name="citation_reference" content="citation_title=Studying complex adaptive systems;,citation_abstract=Complex adaptive systems (cas) – systems that involve many components that adapt or learn as they interact – are at the heart of important contemporary problems. The study of cas poses unique challenges: Some of our most powerful mathematical tools, particularly methods involving fixed points, attractors, and the like, are of limited help in understanding the development of cas. This paper suggests ways to modify research methods and tools, with an emphasis on the role of computer-based models, to increase our understanding of cas.;,citation_author=John H. Holland;,citation_publication_date=2006-03-01;,citation_cover_date=2006-03-01;,citation_year=2006;,citation_fulltext_html_url=https://doi.org/10.1007/s11424-006-0001-z;,citation_issue=1;,citation_doi=10.1007/s11424-006-0001-z;,citation_issn=1559-7067;,citation_volume=19;,citation_language=en-US;,citation_journal_title=Journal of Systems Science and Complexity;,citation_journal_abbrev=Jrl Syst Sci &amp;amp;amp; Complex;">
<meta name="citation_reference" content="citation_title=Signals and boundaries: Building blocks for complex adaptive systems;,citation_author=John H. Holland;,citation_publication_date=2012;,citation_cover_date=2012;,citation_year=2012;,citation_doi=10.7551/mitpress/9412.001.0001;,citation_isbn=978-0-262-01783-1;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Complexity: A very short introduction;,citation_abstract=In this Very Short Introduction, John H. Holland, one of the leading figures in the field, introduces the key elements and conceptual framework of complexity. From complex physical systems such as fluid flow and the difficulties of predicting weather, to complex adaptive systems such as the diverse and interdependent ecosystems of rainforests, he uses simple examples to illustrate the approach and insights provided by complexity theory.;,citation_author=John H. Holland;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_isbn=978-0-19-966254-8;,citation_language=en-US;,citation_series_title=Very short introductions;">
<meta name="citation_reference" content="citation_title=A self-assessment questionnaire to determine morningness-eveningness in human circadian rhythms;,citation_abstract=An English language self-assessment Morningness-Eveningness questionnaire is presented and evaluated against individual differences in the circadian vatiation of oral temperature. 48 subjects falling into Morning, Evening and Intermediate type categories regularly took their temperature. Circadian peak time were identified from the smoothed temperature curves of each subject. Results showed that Morning types and a significantly earlier peak time than Evening types and tended to have a higher daytime temperature and lower post peak temperature. The Intermediate type had temperatures between those of the other groups. Although no significant differences in sleep lengths were found between the three types, Morning types retired and arose significantly earlier than Evening types. Whilst these time significatly correlated with peak time, the questionnaire showed a higher peak time correlation. Although sleep habits are an important déterminant of peak time there are other contibutory factors, and these appear to be partly covered by the questionnaire. Although the questionnaire appears to be valid, further evaluation using a wider subject population is required.;,citation_author=J. A. Horne;,citation_author=O. Östberg;,citation_publication_date=1976;,citation_cover_date=1976;,citation_year=1976;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pubmed/1027738;,citation_issue=2;,citation_issn=0300-9998;,citation_volume=4;,citation_language=en-US;,citation_journal_title=International Journal of Chronobiology;,citation_journal_abbrev=Int J Chronobiol;">
<meta name="citation_reference" content="citation_title=Morningness–eveningness and the environment hypothesis–a cross-cultural comparison of Turkish and German adolescents;,citation_author=Mehmet Barış Horzum;,citation_author=Christoph Randler;,citation_author=Ercan Masal;,citation_author=Şenol Beşoluk;,citation_author=İsmail Önder;,citation_author=Christian Vollmer;,citation_publication_date=2015-07-03;,citation_cover_date=2015-07-03;,citation_year=2015;,citation_fulltext_html_url=http://www.tandfonline.com/doi/full/10.3109/07420528.2015.1041598;,citation_issue=6;,citation_doi=10.3109/07420528.2015.1041598;,citation_issn=0742-0528, 1525-6073;,citation_volume=32;,citation_language=en-US;,citation_journal_title=Chronobiology International;,citation_journal_abbrev=Chronobiology International;">
<meta name="citation_reference" content="citation_title=Latitudinal clines: An evolutionary view on biological rhythms;,citation_abstract=Properties of the circadian and annual timing systems are expected to vary systematically with latitude on the basis of different annual light and temperature patterns at higher latitudes, creating specific selection pressures. We review literature with respect to latitudinal clines in circadian phenotypes as well as in polymorphisms of circadian clock genes and their possible association with annual timing. The use of latitudinal (and altitudinal) clines in identifying selective forces acting on biological rhythms is discussed, and we evaluate how these studies can reveal novel molecular and physiological components of these rhythms.;,citation_author=Roelof A. Hut;,citation_author=Silvia Paolucci;,citation_author=Roi Dor;,citation_author=Charalambos P. Kyriacou;,citation_author=Serge Daan;,citation_publication_date=2013-08-22;,citation_cover_date=2013-08-22;,citation_year=2013;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rspb.2013.0433;,citation_issue=1765;,citation_doi=10.1098/rspb.2013.0433;,citation_issn=0962-8452, 1471-2954;,citation_volume=280;,citation_language=en-US;,citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;,citation_journal_abbrev=Proc. R. Soc. B.;">
<meta name="citation_reference" content="citation_title=Pesquisa Nacional por Amostra de Domicílios Contínua: Acesso à internet e à televisão e posse de telefone móvel celular para uso pessoal 2019;,citation_author=Instituto Brasileiro de Geografia e Estatística;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://biblioteca.ibge.gov.br/visualizacao/livros/liv101794_informativo.pdf;,citation_isbn=978-65-87201-56-6;,citation_language=pt-BR;,citation_technical_report_institution=Instituto Brasileiro de Geografia e Estatística;">
<meta name="citation_reference" content="citation_title=Tabela 6407: População residente, por sexo e grupos de idade;,citation_author=Instituto Brasileiro de Geografia e Estatística;,citation_fulltext_html_url=https://sidra.ibge.gov.br/tabela/6407;,citation_publisher=SIDRA;">
<meta name="citation_reference" content="citation_title=Tabela 6579: População residente estimada;,citation_author=Instituto Brasileiro de Geografia e Estatística;,citation_fulltext_html_url=https://sidra.ibge.gov.br/Tabela/6579;,citation_language=pt-BR;,citation_publisher=SIDRA;">
<meta name="citation_reference" content="citation_title=Efficient tests for normality, homoscedasticity and serial independence of regression residuals;,citation_abstract=We use the Lagrange multiplier procedure to derive efficient joint tests for residual normality, homoscedasticity and serial independence. The tests are simple to compute and asymptotically distributed as χ2.;,citation_author=Carlos M. Jarque;,citation_author=Anil K. Bera;,citation_publication_date=1980-01-01;,citation_cover_date=1980-01-01;,citation_year=1980;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/0165176580900245;,citation_issue=3;,citation_doi=10.1016/0165-1765(80)90024-5;,citation_issn=0165-1765;,citation_volume=6;,citation_language=en-US;,citation_journal_title=Economics Letters;,citation_journal_abbrev=Economics Letters;">
<meta name="citation_reference" content="citation_title=A test for normality of observations and regression residuals;,citation_abstract=Using the Lagrange multiplier procedure or score test on the Pearson family of distributions we obtain tests for normality of observations and regression disturbances. The tests suggested have optimum asymptotic power properties and good finite sample performance. Due to their simplicity they should prove to be useful tools in statistical analysis.;,citation_author=Carlos M. Jarque;,citation_author=Anil K. Bera;,citation_publication_date=1987;,citation_cover_date=1987;,citation_year=1987;,citation_fulltext_html_url=https://www.jstor.org/stable/1403192;,citation_issue=2;,citation_doi=10.2307/1403192;,citation_issn=0306-7734;,citation_volume=55;,citation_language=en-US;,citation_journal_title=International Statistical Review;,citation_publisher=[Wiley, International Statistical Institute (ISI)];">
<meta name="citation_reference" content="citation_title=Weighting methods;,citation_abstract=Weighting adjustments are commonly applied in surveys to compensate for nonresponse and noncoverage, and to make weighted sample estimates conform to external values. Recent years have seen theoretical developments and increased use of methods that take account of substantial amounts of auxiliary information in making these adjustments. The article uses a simple example to describe such methods as cell weighting, raking, generalised regression estimation, logistic regression weighting, mixtures of methods, and methods for restricting the range of the resultant adjustments. It also discusses how auxiliary variables may be chosen for use in the adjustments and describes some applications.;,citation_author=Graham Kalton;,citation_author=Ismael Flores-Cervantes;,citation_publication_date=2003;,citation_cover_date=2003;,citation_year=2003;,citation_issue=2;,citation_issn=0282-423X;,citation_volume=19;,citation_language=en-US;,citation_journal_title=Journal of Official Statistics;">
<meta name="citation_reference" content="citation_title=A phase response curve to single bright light pulses in human subjects;,citation_author=Sat Bir S. Khalsa;,citation_author=Megan E. Jewett;,citation_author=Christian Cajochen;,citation_author=Charles A. Czeisler;,citation_publication_date=2003-06;,citation_cover_date=2003-06;,citation_year=2003;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1113/jphysiol.2003.040477;,citation_issue=3;,citation_doi=10.1113/jphysiol.2003.040477;,citation_issn=0022-3751, 1469-7793;,citation_volume=549;,citation_language=en-US;,citation_journal_title=The Journal of Physiology;,citation_journal_abbrev=The Journal of Physiology;">
<meta name="citation_reference" content="citation_title=A note on studentizing a test for heteroscedasticity;,citation_abstract=Breusch and Pagan (1979) have recently proposed a convenient test for heteroscedasticity in general linear models. This note derives the asymptotic distribution of their test under sequences of contiguous alternatives to the null hypothesis of homoscedasticity. The test is shown to possess asymptotically incorrect size (nominal significance level) except in the case of strictly Gaussian disturbances. A slight modification of the test is proposed which corrects this defect.;,citation_author=Roger Koenker;,citation_publication_date=1981-09-01;,citation_cover_date=1981-09-01;,citation_year=1981;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/0304407681900622;,citation_issue=1;,citation_doi=10.1016/0304-4076(81)90062-2;,citation_issn=0304-4076;,citation_volume=17;,citation_language=en-US;,citation_journal_title=Journal of Econometrics;,citation_journal_abbrev=Journal of Econometrics;">
<meta name="citation_reference" content="citation_title=Sulla determinazione empirica di una legge di distribuzione;,citation_author=A. Kolmogorov;,citation_publication_date=1933;,citation_cover_date=1933;,citation_year=1933;,citation_volume=4;,citation_language=it-IT;,citation_journal_title=Giornale dell’Istituto Italiano degli Attuari;">
<meta name="citation_reference" content="citation_title=What’s normal anyway? Residual plots are more telling than significance tests when checking ANOVA assumptions;,citation_abstract=We consider two questions important for applying analysis of variance (ANOVA): Should normality be checked on the raw data or on the residuals (or is it immaterial which of the two approaches we take)? Should normality and homogeneity of variance be checked using significance tests or diagnostic plots (or both)? Based on two examples, we show that residuals should be used for model checking and that residual plots are better for checking ANOVA assumptions than statistical tests. We also discuss why one should be very cautious when using statistical tests to check the assumptions.;,citation_author=M. Kozak;,citation_author=H.-P. Piepho;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/jac.12220;,citation_issue=1;,citation_doi=10.1111/jac.12220;,citation_issn=1439-037X;,citation_volume=204;,citation_language=en-US;,citation_journal_title=Journal of Agronomy and Crop Science;">
<meta name="citation_reference" content="citation_title=What is complexity?;,citation_publication_date=2023-02-17;,citation_cover_date=2023-02-17;,citation_year=2023;,citation_fulltext_html_url=https://www.youtube.com/watch?v=JR93X7xK05o;,citation_language=en-US;,citation_publisher=Santa Fe Institute;">
<meta name="citation_reference" content="citation_title=The reality ouroboros;,citation_abstract=Toward a new understanding of the nature of reality.;,citation_author=David Krakauer;,citation_author=David Wolpert;,citation_publication_date=2024-09-04;,citation_cover_date=2024-09-04;,citation_year=2024;,citation_fulltext_html_url=https://nautil.us/the-reality-ouroboros-809153/;,citation_language=en-US;,citation_journal_title=Nautilus;">
<meta name="citation_reference" content="citation_title=Chronobiology of interspecific interactions in a changing world;,citation_abstract=Animals should time activities, such as foraging, migration and reproduction, as well as seasonal physiological adaptation, in a way that maximizes fitness. The fitness outcome of such activities depends largely on their interspecific interactions; the temporal overlap with other species determines when they should be active in order to maximize their encounters with food and to minimize their encounters with predators, competitors and parasites. To cope with the constantly changing, but predictable structure of the environment, organisms have evolved internal biological clocks, which are synchronized mainly by light, the most predictable and reliable environmental cue (but which can be masked by other variables), which enable them to anticipate and prepare for predicted changes in the timing of the species they interact with, on top of responding to them directly. Here, we review examples where the internal timing system is used to predict interspecific interactions, and how these interactions affect the internal timing system and activity patterns. We then ask how plastic these mechanisms are, how this plasticity differs between and within species and how this variability in plasticity affects interspecific interactions in a changing world, in which light, the major synchronizer of the biological clock, is no longer a reliable cue owing to the rapidly changing climate, the use of artificial light and urbanization. This article is part of the themed issue “Wild clocks: integrating chronobiology and ecology to understand timekeeping in free-living animals”.;,citation_author=Noga Kronfeld-Schor;,citation_author=Marcel E. Visser;,citation_author=Lucia Salis;,citation_author=Jan A. Gils;,citation_publication_date=2017-10-09;,citation_cover_date=2017-10-09;,citation_year=2017;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/full/10.1098/rstb.2016.0248;,citation_issue=1734;,citation_doi=10.1098/rstb.2016.0248;,citation_volume=372;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;,citation_publisher=Royal Society;">
<meta name="citation_reference" content="citation_title=Introduction to chronobiology;,citation_author=Sandra J. Kuhlman;,citation_author=L. Michon Craig;,citation_author=Jeanne F. Duffy;,citation_publication_date=2018-09;,citation_cover_date=2018-09;,citation_year=2018;,citation_fulltext_html_url=http://cshperspectives.cshlp.org/lookup/doi/10.1101/cshperspect.a033613;,citation_issue=9;,citation_doi=10.1101/cshperspect.a033613;,citation_issn=1943-0264;,citation_volume=10;,citation_language=en-US;,citation_journal_title=Cold Spring Harbor Perspectives in Biology;,citation_journal_abbrev=Cold Spring Harb Perspect Biol;">
<meta name="citation_reference" content="citation_title=Using case weights with tidymodels;,citation_abstract=Support for case weights is now available across many tidymodels packages.;,citation_author=Max Kuhn;,citation_publication_date=2022-05-02;,citation_cover_date=2022-05-02;,citation_year=2022;,citation_fulltext_html_url=https://www.tidyverse.org/blog/2022/05/case-weights/;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=The targets R package: A dynamic make-like function-oriented pipeline toolkit for reproducibility and high-performance computing;,citation_author=William Landau;,citation_publication_date=2021-01-15;,citation_cover_date=2021-01-15;,citation_year=2021;,citation_fulltext_html_url=https://joss.theoj.org/papers/10.21105/joss.02959;,citation_issue=57;,citation_doi=10.21105/joss.02959;,citation_issn=2475-9066;,citation_volume=6;,citation_language=en-US;,citation_journal_title=Journal of Open Source Software;,citation_journal_abbrev=JOSS;">
<meta name="citation_reference" content="citation_title=Latin dictionaries;,citation_author=Latinitium;,citation_fulltext_html_url=https://latinitium.com/latin-dictionaries/;,citation_language=en-US;,citation_publisher=Latinitium;">
<meta name="citation_reference" content="citation_title=Detecting chronotype differences associated to latitude: A comparison between Horne–Östberg and Munich Chronotype questionnaires;,citation_author=Mario André Leocadio-Miguel;,citation_author=Valéria Clarisse De Oliveira;,citation_author=Danyella Pereira;,citation_author=Mario Pedrazzoli;,citation_publication_date=2014-03;,citation_cover_date=2014-03;,citation_year=2014;,citation_fulltext_html_url=http://www.tandfonline.com/doi/full/10.3109/03014460.2013.832795;,citation_issue=2;,citation_doi=10.3109/03014460.2013.832795;,citation_issn=0301-4460, 1464-5033;,citation_volume=41;,citation_language=en-US;,citation_journal_title=Annals of Human Biology;,citation_journal_abbrev=Annals of Human Biology;">
<meta name="citation_reference" content="citation_title=Latitudinal cline of chronotype;,citation_abstract=Abstract The rotation of the Earth around its own axis and around the sun determines the characteristics of the light/dark cycle, the most stable and ancient 24 h temporal cue for all organisms. Due to the tilt in the earth’s axis in relation to the plane of the earth’s orbit around the sun, sunlight reaches the Earth differentially depending on the latitude. The timing of circadian rhythms varies among individuals of a given population and biological and environmental factors underlie this variability. In the present study, we tested the hypothesis that latitude is associated to the regulation of circadian rhythm in humans. We have studied chronotype profiles across latitudinal cline from around 0° to 32° South in Brazil in a sample of 12,884 volunteers living in the same time zone. The analysis of the results revealed that humans are sensitive to the different sunlight signals tied to differences in latitude, resulting in a morning to evening latitudinal cline of chronotypes towards higher latitudes.;,citation_author=Mario André Leocadio-Miguel;,citation_author=Fernando Mazzili Louzada;,citation_author=Leandro Lourenção Duarte;,citation_author=Roberta Peixoto Areas;,citation_author=Marilene Alam;,citation_author=Marcelo Ventura Freire;,citation_author=John Fontenele-Araujo;,citation_author=Luiz Menna-Barreto;,citation_author=Mario Pedrazzoli;,citation_publication_date=2017-07-14;,citation_cover_date=2017-07-14;,citation_year=2017;,citation_fulltext_html_url=https://www.nature.com/articles/s41598-017-05797-w;,citation_issue=1;,citation_doi=10.1038/s41598-017-05797-w;,citation_issn=2045-2322;,citation_volume=7;,citation_language=en-US;,citation_journal_title=Scientific Reports;,citation_journal_abbrev=Sci Rep;">
<meta name="citation_reference" content="citation_title=Complexity: Life at the edge of chaos;,citation_author=Roger Lewin;,citation_publication_date=1993;,citation_cover_date=1993;,citation_year=1993;,citation_isbn=0-02-014795-3;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=On the Kolmogorov-Smirnov test for normality with mean and variance unknown;,citation_author=Hubert W. Lilliefors;,citation_publication_date=1967-06;,citation_cover_date=1967-06;,citation_year=1967;,citation_fulltext_html_url=http://www.tandfonline.com/doi/abs/10.1080/01621459.1967.10482916;,citation_issue=318;,citation_doi=10.1080/01621459.1967.10482916;,citation_issn=0162-1459, 1537-274X;,citation_volume=62;,citation_language=en-US;,citation_journal_title=Journal of the American Statistical Association;,citation_journal_abbrev=Journal of the American Statistical Association;">
<meta name="citation_reference" content="citation_title=Research commentary—Too big to fail: Large samples and the p-value problem;,citation_abstract=The Internet has provided IS researchers with the opportunity to conduct studies with extremely large samples, frequently well over 10,000 observations. There are many advantages to large samples, but researchers using statistical inference must be aware of the p-value problem associated with them. In very large samples, p-values go quickly to zero, and solely relying on p-values can lead the researcher to claim support for results of no practical significance. In a survey of large sample IS research, we found that a significant number of papers rely on a low p-value and the sign of a regression coefficient alone to support their hypotheses. This research commentary recommends a series of actions the researcher can take to mitigate the p-value problem in large samples and illustrates them with an example of over 300,000 camera sales on eBay. We believe that addressing the p-value problem will increase the credibility of large sample IS research as well as provide more insights for readers.;,citation_author=Mingfeng Lin;,citation_author=Henry C. Lucas;,citation_author=Galit Shmueli;,citation_publication_date=2013-12;,citation_cover_date=2013-12;,citation_year=2013;,citation_fulltext_html_url=https://pubsonline.informs.org/doi/10.1287/isre.2013.0480;,citation_issue=4;,citation_doi=10.1287/isre.2013.0480;,citation_issn=1047-7047;,citation_volume=24;,citation_language=en-US;,citation_journal_title=Information Systems Research;,citation_publisher=INFORMS;">
<meta name="citation_reference" content="citation_title=On a measure of lack of fit in time series models;,citation_abstract=The overall test for lack of fit in autoregressive-moving average models proposed by Box &amp;amp;amp; Pierce (1970) is considered. It is shown that a substantially improved approximation results from a simple modification of this test. Some consideration is given to the power of such tests and their robustness when the innovations are nonnormal. Similar modifications in the overall tests used for transfer function-noise models are proposed;,citation_author=G. M. Ljung;,citation_author=G. E. P. Box;,citation_publication_date=1978-08-01;,citation_cover_date=1978-08-01;,citation_year=1978;,citation_fulltext_html_url=https://doi.org/10.1093/biomet/65.2.297;,citation_issue=2;,citation_doi=10.1093/biomet/65.2.297;,citation_issn=0006-3444;,citation_volume=65;,citation_language=en-US;,citation_journal_title=Biometrika;,citation_journal_abbrev=Biometrika;">
<meta name="citation_reference" content="citation_title=Observation botanique;,citation_abstract=En une page, à partir d’observations très simples sur l’ouverture et la fermeture de fleurs, le polymathe Mairan ouvre le champ de la chronobiologie, ou étude des rythmes biologiques du vivant (végétal ou animal).;,citation_author=Jean-Jacques Mairan;,citation_publication_date=1729;,citation_cover_date=1729;,citation_year=1729;,citation_fulltext_html_url=https://gallica.bnf.fr/ark:/12148/bpt6k3527h/f43.item;,citation_language=fr-FR;,citation_inbook_title=Histoire de l’Académie Royale des Sciences: Avec les mémoires de mathématique et de physique, pour la même année: Tirés des registres de cette académie;">
<meta name="citation_reference" content="citation_title=Use of the p-values as a size-dependent function to address practical differences when analyzing large datasets;,citation_abstract=Biomedical research has come to rely on p-values as a deterministic measure for data-driven decision-making. In the largely extended null hypothesis significance testing for identifying statistically significant differences among groups of observations, a single p-value is computed from sample data. Then, it is routinely compared with a threshold, commonly set to 0.05, to assess the evidence against the hypothesis of having non-significant differences among groups, or the null hypothesis. Because the estimated p-value tends to decrease when the sample size is increased, applying this methodology to datasets with large sample sizes results in the rejection of the null hypothesis, making it not meaningful in this specific situation. We propose a new approach to detect differences based on the dependence of the p-value on the sample size. We introduce new descriptive parameters that overcome the effect of the size in the p-value interpretation in the framework of datasets with large sample sizes, reducing the uncertainty in the decision about the existence of biological differences between the compared experiments. The methodology enables the graphical and quantitative characterization of the differences between the compared experiments guiding the researchers in the decision process. An in-depth study of the methodology is carried out on simulated and experimental data. Code availability at https://github.com/BIIG-UC3M/pMoSS.;,citation_author=Estibaliz Gómez-de-Mariscal;,citation_author=Vanesa Guerrero;,citation_author=Alexandra Sneider;,citation_author=Hasini Jayatilaka;,citation_author=Jude M. Phillip;,citation_author=Denis Wirtz;,citation_author=Arrate Muñoz-Barrutia;,citation_publication_date=2021-10-22;,citation_cover_date=2021-10-22;,citation_year=2021;,citation_fulltext_html_url=https://www.nature.com/articles/s41598-021-00199-5;,citation_issue=1, 1;,citation_doi=10.1038/s41598-021-00199-5;,citation_issn=2045-2322;,citation_volume=11;,citation_language=en-US;,citation_journal_title=Scientific Reports;,citation_journal_abbrev=Sci Rep;,citation_publisher=Nature Publishing Group;">
<meta name="citation_reference" content="citation_title=The Kolmogorov-Smirnov test for goodness of fit;,citation_author=Frank J. Massey;,citation_publication_date=1951-03;,citation_cover_date=1951-03;,citation_year=1951;,citation_fulltext_html_url=http://www.tandfonline.com/doi/abs/10.1080/01621459.1951.10500769;,citation_issue=253;,citation_doi=10.1080/01621459.1951.10500769;,citation_issn=0162-1459, 1537-274X;,citation_volume=46;,citation_language=en-US;,citation_journal_title=Journal of the American Statistical Association;,citation_journal_abbrev=Journal of the American Statistical Association;">
<meta name="citation_reference" content="citation_title={actverse}: An R package for actigraphy data analysis [Poster];,citation_author=Vinicius Alves Matias;,citation_author=Cassio Serrano;,citation_author=Daniel Vartanian;,citation_author=Mario Pedrazzoli;,citation_author=Ana Amélia Benedito-Silva;,citation_publication_date=2022-09-03;,citation_cover_date=2022-09-03;,citation_year=2022;,citation_fulltext_html_url=http://dx.doi.org/10.13140/RG.2.2.12760.16643;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Designing experiments and analyzing data: A model comparison perspective;,citation_author=Scott E. Maxwell;,citation_author=Harold D. Delaney;,citation_author=Ken Kelley;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_isbn=978-1-315-16978-1;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Astronomical algorithms;,citation_author=Jean Meeus;,citation_publication_date=1991;,citation_cover_date=1991;,citation_year=1991;,citation_isbn=0-943396-35-2;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=História e perspectivas da cronobiologia no Brasil e na América Latina;,citation_abstract=Este volume representa o resultado de duas motivações: a primeira corresponde à necessidade de registrar um esforço de cerca de cinquenta anos para a implantação de pesquisas em uma nova área de conhecimento em nosso país e em nossa região, a cronobiologia; a segunda equivale a uma tentativa de delinear futuros possíveis e eventualmente desejáveis para essa área.Reunindo essas motivações num formato peculiar, esta obra apresenta depoimentos de estudantes que passaram pelo Grupo Multidisciplinar de Desenvolvimento e Ritmos Biológicos (gmdrb) e colegas latino-americanos que abraçaram a causa da criação de um objeto de diálogo entre nossos países. A opção de manter o idioma original nos depoimentos é uma forma de preservar as identidades temáticas e culturais em nossas pesquisas.Este não é um livro somente de biologia nem somente de história; é uma obra que emerge do entendimento de que a tanto a biologia como a história devem estar sempre presentes e conectadas na observação e na compreensão do mundo.História e Perspectivas da Cronobiologia no Brasil e na América Latina, de Luiz Menna-Barreto, é uma obra original e relevante que apresenta o histórico do Grupo Multidisciplinar de Desenvolvimento e Ritmos Biológicos (gmdrb), iniciado entre os anos de 1980 e 1981 por docentes da Universidade de São Paulo vinculados a diversas unidades. É uma contribuição valiosa para a história da ciência, em especial para o conhecimento do campo da cronobiologia e para o seu desenvolvimento no Brasil.;,citation_editor=Luiz Menna-Barreto;,citation_editor=Nelson Marques;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_isbn=978-65-5785-154-8;,citation_language=pt-BR;,citation_series_title=Campi;">
<meta name="citation_reference" content="citation_title=Merriam-Webster.com dictionary;,citation_abstract=The most popular dictionary and thesaurus for learners of English. Meanings and definitions of words with pronunciations and translations.;,citation_author=Merriam-Webster;,citation_fulltext_html_url=https://www.merriam-webster.com/dictionary;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=The circadian cycle: Daily rhythms from behaviour to genes;,citation_abstract=The daily recurrence of activity and rest are so common as to seem trivial. However, they reflect a ubiquitous temporal programme called the circadian clock. In the absence of either anatomical clock structures or clock genes, the timing of sleep and wakefulness is disrupted. The complex nature of circadian behaviour is evident in the fact that phasing of the cycle during the day varies widely for individuals, resulting in extremes colloquially called “larks” and “owls”. These behavioural oscillations are mirrored in the levels of physiology and gene expression. Deciphering the underlying mechanisms will provide important insights into how the circadian clock affects health and disease.;,citation_author=Martha Merrow;,citation_author=Kamiel Spoelstra;,citation_author=Till Roenneberg;,citation_publication_date=2005-10;,citation_cover_date=2005-10;,citation_year=2005;,citation_fulltext_html_url=https://www.embopress.org/doi/full/10.1038/sj.embor.7400541;,citation_issue=10;,citation_doi=10.1038/sj.embor.7400541;,citation_issn=1469-221X;,citation_volume=6;,citation_language=en-US;,citation_journal_title=EMBO reports;,citation_publisher=John Wiley &amp;amp;amp; Sons, Ltd;">
<meta name="citation_reference" content="citation_title=Circadian clocks: How rhythms structure life;,citation_author=Martha Merrow;,citation_author=Till Roenneberg;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://www.coursera.org/learn/circadian-clocks;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=A human phase-response curve to light;,citation_author=David S. Minors;,citation_author=James M. Waterhouse;,citation_author=Anna Wirz-Justice;,citation_publication_date=1991-11;,citation_cover_date=1991-11;,citation_year=1991;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/030439409190051T;,citation_issue=1;,citation_doi=10.1016/0304-3940(91)90051-T;,citation_issn=03043940;,citation_volume=133;,citation_language=en-US;,citation_journal_title=Neuroscience Letters;,citation_journal_abbrev=Neuroscience Letters;">
<meta name="citation_reference" content="citation_title=Complexity: A guided tour;,citation_author=Melanie Mitchell;,citation_publication_date=2009;,citation_cover_date=2009;,citation_year=2009;,citation_isbn=978-0-19-512441-5;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Introduction to regression methods for public health using R;,citation_abstract=This text was written to be used in a second biostatistics course for Master of Public Health students; however, students in any field will find it useful. Students in many disciplines take an introductory statistics course, providing foundational competencies but perhaps not enough to use more advanced methods without additional training. There are a plethora of textbooks covering topics such as linear regression, logistic regression, and survival analysis aimed at those with a background in mathematical statistics and/or without a focus specifically on public health and/or without a focus on using R statistical software. The goal of this text is to provide a gentle introduction to regression methods, using R, that covers all the basics and a bit more, with examples drawn from public health data.;,citation_author=Ramzi W. Nahhas;,citation_publication_date=2024-10-13;,citation_cover_date=2024-10-13;,citation_year=2024;,citation_fulltext_html_url=https://www.bookdown.org/rwnahhas/RMPH/;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Data.GISS: Time and date of vernal equinox;,citation_abstract=For the Model E AR5 simulations, all years have exactly 365 days and vernal equinox always occurs on March 21, hour 0. Future versions of Model E will use the more precise time of vernal equinox as described on this web page.;,citation_author=National Aeronautics and Space Administration;,citation_author=Goddard Institute for Space Studies;,citation_fulltext_html_url=https://data.giss.nasa.gov/modelE/ar5plots/srvernal.html;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=A simple, positive semi-definite, heteroskedasticity and autocorrelation consistent covariance matrix;,citation_author=Whitney K. Newey;,citation_author=Kenneth D. West;,citation_publication_date=1987;,citation_cover_date=1987;,citation_year=1987;,citation_fulltext_html_url=https://www.jstor.org/stable/1913610;,citation_issue=3;,citation_doi=10.2307/1913610;,citation_issn=0012-9682;,citation_volume=55;,citation_language=en-US;,citation_journal_title=Econometrica;,citation_publisher=[Wiley, Econometric Society];">
<meta name="citation_reference" content="citation_title=Automatic lag selection in covariance matrix estimation;,citation_abstract=We propose a nonparametric method for automatically selecting the number of autocovariances to use in computing a heteroskedasticity and autocorrelation consistent covariance matrix. For a given kernel for weighting the autocovariances, we prove that our procedure is asymptotically equivalent to one that is optimal under a mean-squared error loss function. Monte Carlo simulations suggest that our procedure performs tolerably well, although it does result in size distortions.;,citation_author=Whitney K. Newey;,citation_author=Kenneth D. West;,citation_publication_date=1994-10-01;,citation_cover_date=1994-10-01;,citation_year=1994;,citation_fulltext_html_url=https://doi.org/10.2307/2297912;,citation_issue=4;,citation_doi=10.2307/2297912;,citation_issn=0034-6527;,citation_volume=61;,citation_journal_title=The Review of Economic Studies;,citation_journal_abbrev=The Review of Economic Studies;">
<meta name="citation_reference" content="citation_title=On the use and interpretation of certain test criteria for purposes of statistical inference: Part I;,citation_author=J. Neyman;,citation_author=E. S. Pearson;,citation_publication_date=1928;,citation_cover_date=1928;,citation_year=1928;,citation_fulltext_html_url=https://www.jstor.org/stable/2331945;,citation_issue=1/2;,citation_doi=10.2307/2331945;,citation_issn=0006-3444;,citation_volume=20A;,citation_language=en-US;,citation_journal_title=Biometrika;,citation_publisher=[Oxford University Press, Biometrika Trust];">
<meta name="citation_reference" content="citation_title=On the use and interpretation of certain test criteria for purposes of statistical inference: Part II;,citation_author=J. Neyman;,citation_author=E. S. Pearson;,citation_publication_date=1928;,citation_cover_date=1928;,citation_year=1928;,citation_fulltext_html_url=https://www.jstor.org/stable/2332112;,citation_issue=3/4;,citation_doi=10.2307/2332112;,citation_issn=0006-3444;,citation_volume=20A;,citation_language=en-US;,citation_journal_title=Biometrika;,citation_publisher=[Oxford University Press, Biometrika Trust];">
<meta name="citation_reference" content="citation_title=Foundations of complex systems: Emergence, information and prediction;,citation_author=Grégoire Nicolis;,citation_author=Catherine Nicolis;,citation_publication_date=2012;,citation_cover_date=2012;,citation_year=2012;,citation_isbn=978-981-4366-60-1;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Press release;,citation_author=Nobel Prize Outreach AB;,citation_fulltext_html_url=https://www.nobelprize.org/prizes/medicine/2017/press-release/;,citation_language=en-US;,citation_publisher=The Nobel Prize;">
<meta name="citation_reference" content="citation_title=OpenStreetMap;,citation_author=OpenStreetMap contributors;,citation_fulltext_html_url=https://www.openstreetmap.org;,citation_publisher=OpenStreetMap Foundation;">
<meta name="citation_reference" content="citation_title=Evolution of temporal order in living organisms;,citation_author=Dhanashree A Paranjpe;,citation_author=Vijay Kumar Sharma;,citation_publication_date=2005-05-04;,citation_cover_date=2005-05-04;,citation_year=2005;,citation_fulltext_html_url=https://www.jcircadianrhythms.com/article/10.1186/1740-3391-3-7/;,citation_doi=10.1186/1740-3391-3-7;,citation_issn=1740-3391;,citation_volume=3;,citation_journal_title=Journal of Circadian Rhythms;,citation_journal_abbrev=J Circadian Rhythms;">
<meta name="citation_reference" content="citation_title=Molecular architecture of the mammalian circadian clock;,citation_author=Carrie L. Partch;,citation_author=Carla B. Green;,citation_author=Joseph S. Takahashi;,citation_publication_date=2014-02;,citation_cover_date=2014-02;,citation_year=2014;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S096289241300113X;,citation_issue=2;,citation_doi=10.1016/j.tcb.2013.07.002;,citation_issn=09628924;,citation_volume=24;,citation_language=en-US;,citation_journal_title=Trends in Cell Biology;,citation_journal_abbrev=Trends in Cell Biology;">
<meta name="citation_reference" content="citation_title=X. On the criterion that a given system of deviations from the probable in the case of a correlated system of variables is such that it can be reasonably supposed to have arisen from random sampling;,citation_author=Karl Pearson;,citation_publication_date=1900-07;,citation_cover_date=1900-07;,citation_year=1900;,citation_fulltext_html_url=https://www.tandfonline.com/doi/abs/10.1080/14786440009463897;,citation_issue=302;,citation_doi=10.1080/14786440009463897;,citation_issn=1941-5982;,citation_volume=50;,citation_language=en-US;,citation_journal_title=The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science;,citation_publisher=Taylor &amp;amp;amp; Francis;">
<meta name="citation_reference" content="citation_title=geobr: Download official spatial data sets of Brazil;,citation_abstract=geobr is a computational package to download official spatial data sets of Brazil. The package includes a wide range of geospatial data in geopackage format (like shapefiles but better), available at various geographic scales and for various years with harmonized attributes, projection and topology.;,citation_author=Rafael H. M. Pereira;,citation_author=Caio Nogueira Goncalves;,citation_fulltext_html_url=https://ipeagit.github.io/geobr;,citation_doi=10.32614/CRAN.package.geobr;">
<meta name="citation_reference" content="citation_title=Atlas brasileiro de energia solar;,citation_abstract=A primeira edição do Atlas Brasileiro de Energia Solar foi lançada em 2006 com base em 10 anos de dados dos satélites da série GOES e no modelo físico de transferência radiativa BRASIL-SR, validado com dados observados em 98 estações meteorológicas operadas pelo INMET (Instituto Nacional de Meteorologia) e espalhadas por todo território nacional. Na época do lançamento, a rede SONDA (Sistema de Organização Nacional de Dados Ambientais), operada pelo INPE, havia recém entrado em operação e contribuiu no processo de validação com apenas três anos de dados solarimétricos das 3 componentes da irradiação solar na superfície: global horizontal, direta normal e difusa. Essa edição pioneira do Atlas constituiu um marco importante no histórico da energia solar no Brasil e é, ainda hoje, empregada por vários investigadores e empreendedores da área de energia solar. Após mais de 10 anos, o Centro de Ciência do Sistema Terrestre (CCST) do Instituto Nacional de Pesquisas Espaciais (INPE), através do seu Laboratório de Modelagem e Estudos de Recursos Renováveis de Energia (LABREN), tem a satisfação de publicar a segunda edição, ampliada e revisada, do Atlas Brasileiro de Energia Solar. Trata-se de um exemplo de trabalho cooperativo entre o INPE e pesquisadores de várias instituições no Brasil: a Universidade Federal de São Paulo (UNIFESP), a Universidade Federal de Santa Catarina (UFSC), a Universidade Tecnológica Federal do Paraná (UTFPR) e o Instituto Federal de Santa Catarina (IFSC). Para essa nova edição, foram empregados mais de 17 anos de dados satelitais e implementados vários avanços nas parametrizações do modelo de transferência radiativa BRASIL-SR, visando melhorar ainda mais a confiabilidade e acurácia da base de dados produzida e disponibilizada para acesso público. Além desses avanços, a nova versão contém análises sobre os níveis de confiança, sobre a variabilidade espacial e temporal do recurso solar, além de apresentar cenários de emprego de várias tecnologias solares. Embora o foco do Atlas seja a área de energia, os dados apresentados também atendem usuários em várias outras áreas de conhecimento, como a meteorologia, climatologia, agricultura, hidrologia e arquitetura. Este Atlas contou com a contribuição científica do Instituto Nacional de Ciência e Tecnologia para Mudanças Climáticas (INCT-MC), através dos processos CNPq 573797/2008-0 e FAPESP 2008/57719-9, o qual apoiou a fase de pesquisa, consolidação e de sua montagem final. Não podemos também deixar de agradecer e de partilhar esse momento com o Centro de Pesquisas da Petrobras (CENPES) que, através do Projeto ANEEL PD-0553-0013/2010 com o INPE, forneceu o importante suporte financeiro para o aprimoramento do modelo BRASIL-SR, assim como para a expansão, operação e manutenção da rede solarimétrica SONDA. Os créditos também são endereçados à Rede Brasileira de Pesquisas sobre Mudanças Climáticas Globais, através do convênio FINEP / Rede CLIMA 01.13.0353-00, pelo suporte na fase de rodadas do modelo, e aos demais colegas do INPE, particularmente do Laboratório de Instrumentação Meteorológica (LIM), do Centro de Previsão de Tempo e Estudos Climáticos (CPTEC), que forneceram suporte logístico a esse trabalho desde a primeira edição do Atlas. Agradecemos também o apoio institucional da Agência Nacional de Energia Elétrica (ANEEL) e da International Solar Energy Society (ISES) pelo reconhecimento do mérito científico dessa publicação como marco importante para a penetração da tecnologia solar no Brasil.Os autores e as entidades de suporte e apoio que possibilitaram mais essa edição do Atlas esperam que esse trabalho constitua mais um importante marco para o avanço da tecnologia solar no Brasil.;,citation_author=Enio Bueno Pereira;,citation_author=Fernando Ramos Martins;,citation_author=André Rodrigues Gonçalves;,citation_author=Rodrigo Costa;,citation_author=Francisco J. Lopes Lima;,citation_author=Ricardo Rüther;,citation_author=Samuel Luna Abreu;,citation_author=Gerson Máximo Tiepolo;,citation_author=Sílvia Vitorino Pereira;,citation_author=Jefferson Gonçalves Souza;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_fulltext_html_url=https://labren.ccst.inpe.br/atlas_2017.html;,citation_doi=10.34024/978851700089;,citation_isbn=978-85-17-00089-8;,citation_language=pt-BR;">
<meta name="citation_reference" content="citation_title=Fisher, Neyman-Pearson or NHST? A tutorial for teaching data testing;,citation_abstract=Despite frequent calls for the overhaul of null hypothesis significance testing (NHST), this controversial procedure remains ubiquitous in behavioral, social and biomedical teaching and research. Little change seems possible once the procedure becomes well ingrained in the minds and current practice of researchers; thus, the optimal opportunity for such change is at the time the procedure is taught, be this at undergraduate or at postgraduate levels. This paper presents a tutorial for the teaching of data testing procedures, often referred to as hypothesis testing theories. The first procedure introduced is Fisher’s approach to data testing—tests of significance; the second is Neyman-Pearson’s approach—tests of acceptance; the final procedure is the incongruent combination of the previous two theories into the current approach—NSHT. For those researchers sticking with the latter, two compromise solutions on how to improve NHST conclude the tutorial.;,citation_author=Jose D. Perezgonzalez;,citation_publication_date=2015-03-02;,citation_cover_date=2015-03-02;,citation_year=2015;,citation_fulltext_html_url=https://www.frontiersin.org/journals/psychology/articles/10.3389/fpsyg.2015.00223/full;,citation_doi=10.3389/fpsyg.2015.00223;,citation_issn=1664-1078;,citation_volume=6;,citation_language=en-US;,citation_journal_title=Frontiers in Psychology;,citation_journal_abbrev=Front. Psychol.;,citation_publisher=Frontiers;">
<meta name="citation_reference" content="citation_title=Circadian rhythms and the circadian organization of living systems;,citation_author=Colin S. Pittendrigh;,citation_publication_date=1960;,citation_cover_date=1960;,citation_year=1960;,citation_fulltext_html_url=http://symposium.cshlp.org/cgi/doi/10.1101/SQB.1960.025.01.015;,citation_doi=10.1101/SQB.1960.025.01.015;,citation_issn=0091-7451, 1943-4456;,citation_volume=25;,citation_language=en-US;,citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;,citation_journal_abbrev=Cold Spring Harbor Symposia on Quantitative Biology;">
<meta name="citation_reference" content="citation_title=Circadian systems: General perspective;,citation_author=Colin S. Pittendrigh;,citation_publication_date=1981;,citation_cover_date=1981;,citation_year=1981;,citation_fulltext_html_url=http://link.springer.com/10.1007/978-1-4615-6552-9;,citation_doi=10.1007/978-1-4615-6552-9;,citation_isbn=978-1-4615-6552-9;,citation_volume=4;,citation_language=en-US;,citation_inbook_title=Biological rhythms;,citation_series_title=Handbook of Behavioral Neurobiology;">
<meta name="citation_reference" content="citation_title=Latitudinal clines in the properties of a circadian pacemaker;,citation_abstract=The circadian rhythm of eclosion activity and its pacemaker were analyzed in a series of latitudinal races of Drosophila auraria ranging from 34.2° to 42.9°N in...;,citation_author=Colin S. Pittendrigh;,citation_author=Tsuguhiko Takamura;,citation_publication_date=1989-06-01;,citation_cover_date=1989-06-01;,citation_year=1989;,citation_fulltext_html_url=https://journals.sagepub.com/doi/abs/10.1177/074873048900400209;,citation_issue=2;,citation_doi=10.1177/074873048900400209;,citation_volume=4;,citation_language=en-US;,citation_journal_title=Journal of Biological Rhythms;,citation_publisher=Sage PublicationsSage CA: Thousand Oaks, CA;">
<meta name="citation_reference" content="citation_title=The amplitude of circadian oscillations: Temperature dependence, latitudinal clines, and the photoperiodic time measurement;,citation_abstract=This paper develops several propositions concerning the lability of the amplitude of Drosophila circadian pacemakers. The first is that the amplitude of the pacemaker’s motion, unlike its period, is markedly temperature-dependent. The second is that latitudinal variation in pacemaker amplitude (higher in the north) is responsible for two very different sets of observations on Drosophila circadian systems at successively higher latitudes. One of these is a cline in D. auraria’s phase-shifting response to light, which steadily weakens in a succession of more northerly strains. The other, concerning D. littoralis in the very far north, is a cline in the rate at which eclosion activity becomes arrhythmic (the circadian rhythm damps out) in constant darkness; damping is faster in the north. The third proposition concerns a plausible selection pressure for the cline in pacemaker amplitude that we propose underlies the two directly observed clines. Two points are emphasized: (1) The amplitude of the pacemaker’s daily oscillation declines as the duration of the entraining light pulse (photoperiod) is increased; and (2) the duration of the daily photoperiods throughout the breeding season is steadily increased as one moves toward the poles. Selection for conservation of pacemaker amplitude (during the breeding season) would produce the latitudinal cline we propose. The fourth, and final proposition is that since the amplitude of the pacemaker’s daily motion responds systematically to change in photoperiod, amplitude is clearly one way—and a temperature-dependent way—in which insect circadian systems may sense seasonal change. These propositions concerning the temperature and latitude dependence of pacemaker amplitude may be relevant to a wider array of circadian pacemakers than Drosophila.;,citation_author=Colin S. Pittendrigh;,citation_author=Walter T. Kyner;,citation_author=Tsuguhiko Takamura;,citation_publication_date=1991-12;,citation_cover_date=1991-12;,citation_year=1991;,citation_fulltext_html_url=http://journals.sagepub.com/doi/10.1177/074873049100600402;,citation_issue=4;,citation_doi=10.1177/074873049100600402;,citation_issn=0748-7304, 1552-4531;,citation_volume=6;,citation_language=en-US;,citation_journal_title=Journal of Biological Rhythms;,citation_journal_abbrev=J Biol Rhythms;">
<meta name="citation_reference" content="citation_title=Temporal organization: Reflections of a darwinian clock-watcher;,citation_author=Colin S. Pittendrigh;,citation_publication_date=1993-10;,citation_cover_date=1993-10;,citation_year=1993;,citation_fulltext_html_url=https://www.annualreviews.org/doi/10.1146/annurev.ph.55.030193.000313;,citation_issue=1;,citation_doi=10.1146/annurev.ph.55.030193.000313;,citation_issn=0066-4278, 1545-1585;,citation_volume=55;,citation_language=en-US;,citation_journal_title=Annual Review of Physiology;,citation_journal_abbrev=Annu. Rev. Physiol.;">
<meta name="citation_reference" content="citation_title=Objective knowledge: An evolutionary approach;,citation_author=Karl Raimund Popper;,citation_publication_date=1979;,citation_cover_date=1979;,citation_year=1979;,citation_isbn=0-19-875024-2;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Conjectures and refutations: The growth of scientific knowledge;,citation_author=Karl Raimund Popper;,citation_publication_date=2002;,citation_cover_date=2002;,citation_year=2002;,citation_isbn=978-0-415-28593-3;,citation_language=en-US;,citation_series_title=Routledge Classics;">
<meta name="citation_reference" content="citation_title=Unended quest: An intellectual autobiography;,citation_abstract=At the age of eight, Karl Popper was puzzling over the idea of infinity and by fifteen was beginning to take a keen interest in his father’s well-stocked library of books. Unended Quest recounts these moments and many others in the life of one of the most influential thinkers of the twentieth century, providing an indispensable account of the ideas that influenced him most. As an introduction to Popper’s philosophy, Unended Quest also shines. Popper lucidly explains the central ideas in his work, making this book ideal for anyone coming to Popper’s life and work for the first time.;,citation_author=Karl Raimund Popper;,citation_publication_date=2005;,citation_cover_date=2005;,citation_year=2005;,citation_isbn=0-203-99425-6;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=The logic of scientific discovery;,citation_abstract=Described by the philosopher A.J. Ayer as a work of ’great originality and power’, this book revolutionized contemporary thinking on science and knowledge. Ideas such as the now legendary doctrine of ’falsificationism’ electrified the scientific community, influencing even working scientists, as well as post-war philosophy. This astonishing work ranks alongside The Open Society and Its Enemies as one of Popper’s most enduring books and contains insights and arguments that demand to be read to this day.;,citation_author=Karl Raimund Popper;,citation_publication_date=2005;,citation_cover_date=2005;,citation_year=2005;,citation_isbn=978-0-203-99462-7;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Banco de CEP e código IBGE;,citation_abstract=São 1.382.926 CEPs, 11.309 municipios/cidades e 71.352 bairros. Tenha uma base de CEP dos Correios e códigos IBGE de todo o Brasil para utilizar em consultas diárias, ou no desenvolvimento de suas aplicações pessoais ou corporativas.;,citation_author=Qual o CEP;,citation_publication_date=2024-11;,citation_cover_date=2024-11;,citation_year=2024;,citation_fulltext_html_url=https://www.qualocep.com/;,citation_language=pt-BR;">
<meta name="citation_reference" content="citation_title=Nature, nurture, or chance: Stochastic gene expression and its consequences;,citation_abstract=Gene expression is a fundamentally stochastic process, with randomness in transcription and translation leading to cell-to-cell variations in mRNA and protein levels. This variation appears in organisms ranging from microbes to metazoans, and its characteristics depend both on the biophysical parameters governing gene expression and on gene network structure. Stochastic gene expression has important consequences for cellular function, being beneficial in some contexts and harmful in others. These situations include the stress response, metabolism, development, the cell cycle, circadian rhythms, and aging.;,citation_author=Arjun Raj;,citation_author=Alexander Oudenaarden;,citation_publication_date=2008-10-17;,citation_cover_date=2008-10-17;,citation_year=2008;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0092867408012439;,citation_issue=2;,citation_doi=10.1016/j.cell.2008.09.050;,citation_issn=0092-8674;,citation_volume=135;,citation_journal_title=Cell;,citation_journal_abbrev=Cell;">
<meta name="citation_reference" content="citation_title=Tests for specification errors in classical linear least-squares regression analysis;,citation_abstract=The effects on the distribution of least-squares residuals of a series of model mis-specifications are considered. It is shown that for a variety of specification errors the distributions of the least-squares residuals are normal, but with non-zero means. An alternative predictor of the disturbance vector is used in developing four procedures for testing for the presence of specification error. The specification errors considered are omitted variables, incorrect functional form, simultaneous equation problems and heteroskedasticity.;,citation_author=J. B. Ramsey;,citation_publication_date=1969;,citation_cover_date=1969;,citation_year=1969;,citation_fulltext_html_url=https://academic.oup.com/jrsssb/article-abstract/31/2/350/7027014;,citation_issue=2;,citation_doi=10.1111/j.2517-6161.1969.tb00796.x;,citation_issn=0035-9246;,citation_volume=31;,citation_language=en-US;,citation_journal_title=Journal of the Royal Statistical Society. Series B (Methodological);,citation_publisher=[Royal Statistical Society, Oxford University Press];">
<meta name="citation_reference" content="citation_title=Morningness-eveningness comparison in adolescents from different countries around the world;,citation_author=Christoph Randler;,citation_publication_date=2008-01;,citation_cover_date=2008-01;,citation_year=2008;,citation_fulltext_html_url=http://www.tandfonline.com/doi/full/10.1080/07420520802551519;,citation_issue=6;,citation_doi=10.1080/07420520802551519;,citation_issn=0742-0528, 1525-6073;,citation_volume=25;,citation_language=en-US;,citation_journal_title=Chronobiology International;,citation_journal_abbrev=Chronobiology International;">
<meta name="citation_reference" content="citation_title=Latitude affects morningness-eveningness: Evidence for the environment hypothesis based on a systematic review;,citation_abstract=Morningness-eveningness (M/E) is an individual trait related to a person’s sleep-wake cycle and preference for morning or evening hours. The “environment hypothesis” suggests that M/E is dependent on environmental factors, such as latitude, mean average temperature and photoperiod. We here analyzed a large number of datasets to assess this effect based on a systematic review. Data were from a total of 87 datasets and 35,589 individuals based on 28 countries. Partial correlations correcting for age revealed significant relationships between M/E and latitude, mean yearly temperature, photoperiod and sunset. Evening orientation was related to higher latitude, longer days and later sunset. Morning orientation was related to higher average temperatures. Percentage of females and sunrise time had no significant influence. These variables (sunset, temperature, photoperiod) were then input in a general linear model. The full model showed an influence of age and of sunset on CSM scores, but not of photoperiod and average temperature. Sunset, therefore, seems to be the most important statistical predictor for the observed latitudinal gradient.;,citation_author=Christoph Randler;,citation_author=Arash Rahafar;,citation_publication_date=2017-01-03;,citation_cover_date=2017-01-03;,citation_year=2017;,citation_fulltext_html_url=https://www.nature.com/articles/srep39976;,citation_issue=1;,citation_doi=10.1038/srep39976;,citation_issn=2045-2322;,citation_volume=7;,citation_language=en-US;,citation_journal_title=Scientific Reports;,citation_journal_abbrev=Sci Rep;">
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;,citation_author=R Core Team;,citation_fulltext_html_url=https://www.R-project.org;,citation_publisher=R Foundation for Statistical Computing;">
<meta name="citation_reference" content="citation_title=Metade da população se sente mal no horário de verão, revela pesquisa;,citation_abstract=Efeitos da mudança do relógio na saúde foram pesquisados pela 1ª vez. Cientistas defendem que cada um de nós tem um tempo próprio.;,citation_publication_date=2017-10-15;,citation_cover_date=2017-10-15;,citation_year=2017;,citation_fulltext_html_url=https://g1.globo.com/fantastico/noticia/2017/10/metade-da-populacao-se-sente-mal-no-horario-de-verao-revela-pesquisa.html;,citation_language=pt-BR;,citation_publisher=Rede Globo;">
<meta name="citation_reference" content="citation_title=Validation of the Portuguese variant of the Munich Chronotype Questionnaire (MCTQ-PT);,citation_author=Cátia Reis;,citation_author=Sara Gamboa Madeira;,citation_author=Luísa V. Lopes;,citation_author=Teresa Paiva;,citation_author=Till Roenneberg;,citation_publication_date=2020-07-14;,citation_cover_date=2020-07-14;,citation_year=2020;,citation_fulltext_html_url=https://www.frontiersin.org/article/10.3389/fphys.2020.00795/full;,citation_doi=10.3389/fphys.2020.00795;,citation_issn=1664-042X;,citation_volume=11;,citation_journal_title=Frontiers in Physiology;,citation_journal_abbrev=Front. Physiol.;">
<meta name="citation_reference" content="citation_title=Sleep patterns in Portugal;,citation_abstract=Sleep represents an essential element for health and well-being, including cognitive performance, physiological processes, emotional regulation, physical development, and quality of life and is the best observed circadian rhythm in humans. Humans show large differences in the preferred timing of their sleep and activity, so-called chronotypes. Ranging from extreme early types to extreme late chronotypes, the majority falls between these extremes. While chronotype and sleep necessity appear to be independent characteristics, the difference in sleep duration between workdays and work-free days is nonetheless chronotype-dependent because of our social schedules. Extremely earlier chronotype experience shorter sleep duration on free-days and longer on workdays. On the other hand, the later chronotypes have shorter sleep duration on workdays and longer on free days. Its described in literature that chronotypes become later from eastern to western border of time zones. The social time is equal, whereas solar time is progressively delayed, producing a later phase of entrainment and increased discrepancies between individuals’ social and biological circadian times, that can lead to a higher circadian disruption with potential health detriments. Portugal is in Greenwich Meridian Time (GMT) time zone, the second most western and the most southern country, so it is expected that the Portuguese might be especially late chronotypes. Therefore, to explore their sleep/chronotype is of major interest. The studies present in this thesis aimed to characterize the sleep/wake phases of the Portuguese general population as well as in sleep medicine patients and to compare them with other countries/samples. The characterization of a clinical sample of extreme late chronotypes and to evaluate how much the social clocks influence the sleep quality in sleep disorder patients were also objectives of this thesis. In order to accomplish these objectives five different studies were performed: Study 1) Aiming to evaluate self-reported average sleep duration (n= 5,436 adults; EpiDoC3 cohort) and its possible associations with demographic features and chronic medical disorders. Cross-sectional, populational based study. The prevalence for Short Sleep Duration (SSD) was high (20.7%) and for Long Sleep Duration (LSD) (5.9%) was relatively low. Being older, with lower education, retired and unemployed were associated to SSD and LSD (p$&amp;amp;amp;lt;$0.01). Being obese was associated to SSD as well as hypertension, gastrointestinal disease and hypercholesterolemia (p$&amp;lt;$0.01). SSD and LSD, were associated with diabetes (p$<$0.01 and p=0.03, respectively) and with depression (p$<$0.01 and p=0.02, respectively). Cardiovascular disease (p$<$0.01) was associated to LSD. Multimorbidity was associated to SSD (p$<$0.01). Worse quality of life and bad physical function were associated to SSD and LSD, as well as being hospitalized in the previous 12 months (p$<$0.01). Study 2) Aimed at understanding if the reported “usual” sleep quality assessed by the Pittsburg Sleep Quality Index (PSQI) reflects workdays sleep of sleep disorder patients and to find a social influence on sleep quality detriment of these patients. Cross-sectional, case- control study. 431 sleep disorder patients and 338 subjects from the general population answered three variants of the PSQI (usual (u), work (w), work-free (f) days) and the Munich Chronotype Questionnaire (MCTQ). PSQI scores differed between groups (p$<$0.001). Post-hoc analysis revealed a significant difference between PSQIu vs. PSQIf and PSQIw vs. PSQIf with PSQIf presenting lower scores, while PSQIu vs. PSQIw did not differ in any group. In line with previous findings, Social Jetlag (SJL) was associated to PSQIdiff in sleep disorder patients. Study 3) Aimed at validating the Portuguese variant of the MCTQ. Cross-sectional observational study, including the, 80 Portuguese individuals from the general population with 4 weeks of actimetry and MCTQ; The median mid-point of sleep, chronotype (MSFsc) was 4:33 (abs range: 2:04 – 8:45). The 1st quartile was earlier than 3:43, and the 4th quartile later than 5:15. Extreme chronotypes (2.5% at each end of the distribution) were earlier than 2:20 and later than 8:36. Comparing the average chronotype of Portuguese subjects with other countries, the Korean’s were the latest (MSFsc = 5:07) followed by the Portuguese (MSFsc = 4:37), the German (MSFsc = 4:22) and the Japanese (MSFsc = 4:18). For the extreme late chronotypes the Portuguese (8:36) was later than the late 2.5% in the Korean sample (8:34). High correlations between MCTQPT and actimetry derived data were found showing that the MCTQ was a reliable questionnaire to access sleep timings and phase of entrainment for the Portuguese adult population. Study 4) Aimed at characterizing a clinical population of delayed sleep-wake phase disorder (DSWPD) patients. In 5 years of registries 245 patients with DSWPD were identified and 162 patients had performed at least one objective sleep/wake/phase measure (actimetry, DLMO, PSG). Gender differences for the different age groups (p=0.028) were observed. Men were more likely to be single and women more likely to be married (p=0.034). In students, school failure was higher for women (p$<$0.001); for workers, absenteeism was higher in women (p=0.001). In the circadian aligned (compared to misaligned group), DLMO was later (p$<$0.001), sleep onset time (p=0.046) was later, total sleep time (p=0.035), and number of sleep cycles (p=0.018) were lower, as measured using PSG T1. Study 5) Aimed at comparing 9 European countries in relation to chronotype, average sleep duration and social jetlag. Cross-sectional observational study. 164,234 individuals from the MCTQ-database (8 countries) and a sample of Icelanders (total n=165,378). The average chronotype among the studied countries ranged between 3.97 ± 1.16 for Belgium and 5.15 ± 0.25 for Spain, for the Central European Greenwich Meridian Time zone (GMT+1) and between 4.21 ± 1.41 for the United Kingdom and 4.76 ± 1.46 for Portugal among the GMT countries. The average Sleep Duration ranged between 7.36 ± 0.21 for Spain and 7.67 ± 0.95 for Belgium, for the GMT+1 and between 7.33 ± 1.05 for Ireland and 8.12 ± 2.42 for Portugal, among the GMT countries. The average SJL ranged between 1.32 ± 0.92 for Netherland and 1.65 ± 0.16, for Spain for the GMT+1 and between 1.24 ± 0.93 for United Kingdom and 1.70 ± 0.86 for Iceland among the GMT countries. In conclusion, the work in this dissertation showed the associations to self-reported total sleep time and chronic diseases and poor health quality to the Portuguese population. A negative social influence on reported sleep quality of Portuguese sleep disorder patients was present. Social jetlag was a predictor for poor sleep quality, mediated by chronotype. We suggest that treatment strategies addressing social aspects affecting sleep quality on workdays and minimizing actions for circadian strain would be of major importance on sleep medicine clinical practice. Late type seems to be common among the Portuguese and this was shown not only for subjective data as well as by objective data. Recommendations is something that needs to be considered in our society since it can cause a circadian misalignment, when leaving against our biological clocks. The causes of circadian misalignment and health detriments were well shown in this thesis and have an economic impact measured by the absenteeism and hospitalizations of the EpiReuma study, which should more precisely evaluated in future studies. Sleep education and awareness campaigns are needed for Portugal (general population, physicians, employers and policy makers) in order to advertise to this lateness predisposition, find countermeasures strategies and advertise for health consequences of leaving against the clock.;,citation_author=Cátia Reis;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=http://hdl.handle.net/10451/54147;,citation_language=pt-PT;,citation_dissertation_institution=Universidade de Lisboa;">
<meta name="citation_reference" content="citation_title=Life between clocks: Daily temporal patterns of human chronotypes;,citation_abstract=Human behavior shows large interindividual variation in temporal organization. Extreme “larks” wake up when extreme “owls” fall asleep. These chronotypes are attributed to differences in the circadian clock, and in animals, the genetic basis of similar phenotypic differences is well established. To better understand the genetic basis of temporal organization in humans, the authors developed a questionnaire to document individual sleep times, self-reported light exposure, and self-assessed chronotype, considering work and free days separately. This report summarizes the results of 500 questionnaires completed in a pilot study. Individual sleep times show large differences between work and free days, except for extreme early types. During the workweek, late chronotypes accumulate considerable sleep debt, for which they compensate on free days by lengthening their sleep by several hours. For all chronotypes, the amount of time spent outdoors in broad daylight significantly affects the timing of sleep: Increased self-reported light exposure advances sleep. The timing of self-selected sleep is multifactorial, including genetic disposition, sleep debt accumulated on workdays, and light exposure. Thus, accurate assessment of genetic chronotypes has to incorporate all of these parameters. The dependence of human chronotype on light, that is, on the amplitude of the light:dark signal, follows the known characteristics of circadian systems in all other experimental organisms. Our results predict that the timing of sleep has changed during industrialization and that a majority of humans are sleep deprived during the workweek. The implications are far ranging concerning learning, memory, vigilance, performance, and quality of life.;,citation_author=Till Roenneberg;,citation_author=Anna Wirz-Justice;,citation_author=Martha Merrow;,citation_publication_date=2003-02;,citation_cover_date=2003-02;,citation_year=2003;,citation_fulltext_html_url=http://journals.sagepub.com/doi/10.1177/0748730402239679;,citation_issue=1;,citation_doi=10.1177/0748730402239679;,citation_issn=0748-7304, 1552-4531;,citation_volume=18;,citation_language=en-US;,citation_journal_title=Journal of Biological Rhythms;,citation_journal_abbrev=J Biol Rhythms;">
<meta name="citation_reference" content="citation_title=Epidemiology of the human circadian clock;,citation_author=Till Roenneberg;,citation_author=Tim Kuehnle;,citation_author=Myriam Juda;,citation_author=Thomas Kantermann;,citation_author=Karla Allebrandt;,citation_author=Marijke Gordijn;,citation_author=Martha Merrow;,citation_publication_date=2007-12;,citation_cover_date=2007-12;,citation_year=2007;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S1087079207000895;,citation_issue=6;,citation_doi=10.1016/j.smrv.2007.07.005;,citation_issn=10870792;,citation_volume=11;,citation_language=en-US;,citation_journal_title=Sleep Medicine Reviews;,citation_journal_abbrev=Sleep Medicine Reviews;">
<meta name="citation_reference" content="citation_title=The human circadian clock entrains to sun time;,citation_author=Till Roenneberg;,citation_author=C. Jairaj Kumar;,citation_author=Martha Merrow;,citation_publication_date=2007-01;,citation_cover_date=2007-01;,citation_year=2007;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S0960982206026091;,citation_issue=2;,citation_doi=10.1016/j.cub.2006.12.011;,citation_issn=09609822;,citation_volume=17;,citation_language=en-US;,citation_journal_title=Current Biology;,citation_journal_abbrev=Current Biology;">
<meta name="citation_reference" content="citation_title=What is chronotype?;,citation_author=Till Roenneberg;,citation_publication_date=2012-04;,citation_cover_date=2012-04;,citation_year=2012;,citation_fulltext_html_url=http://doi.wiley.com/10.1111/j.1479-8425.2012.00541.x;,citation_issue=2;,citation_doi=10.1111/j.1479-8425.2012.00541.x;,citation_issn=14469235;,citation_volume=10;,citation_language=en-US;,citation_journal_title=Sleep and Biological Rhythms;">
<meta name="citation_reference" content="citation_title=Social jetlag and obesity;,citation_author=Till Roenneberg;,citation_author=Karla V. Allebrandt;,citation_author=Martha Merrow;,citation_author=Céline Vetter;,citation_publication_date=2012-05;,citation_cover_date=2012-05;,citation_year=2012;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S0960982212003259;,citation_issue=10;,citation_doi=10.1016/j.cub.2012.03.038;,citation_issn=09609822;,citation_volume=22;,citation_language=en-US;,citation_journal_title=Current Biology;,citation_journal_abbrev=Current Biology;">
<meta name="citation_reference" content="citation_title=The circadian clock and human health;,citation_author=Till Roenneberg;,citation_author=Martha Merrow;,citation_publication_date=2016-05;,citation_cover_date=2016-05;,citation_year=2016;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S0960982216303335;,citation_issue=10;,citation_doi=10.1016/j.cub.2016.04.011;,citation_issn=09609822;,citation_volume=26;,citation_language=en-US;,citation_journal_title=Current Biology;,citation_journal_abbrev=Current Biology;">
<meta name="citation_reference" content="citation_title=Why should we abolish daylight saving time?;,citation_abstract=Local and national governments around the world are currently considering the elimination of the annual switch to and from Daylight Saving Time (DST). As an international organization of scientists dedicated to studying circadian and other biological rhythms, the Society for Research on Biological Rhythms (SRBR) engaged experts in the field to write a Position Paper on the consequences of choosing to live on DST or Standard Time (ST). The authors take the position that, based on comparisons of large populations living in DST or ST or on western versus eastern edges of time zones, the advantages of permanent ST outweigh switching to DST annually or permanently. Four peer reviewers provided expert critiques of the initial submission, and the SRBR Executive Board approved the revised manuscript as a Position Paper to help educate the public in their evaluation of current legislative actions to end DST.;,citation_author=Till Roenneberg;,citation_author=Anna Wirz-Justice;,citation_author=Debra J. Skene;,citation_author=Sonia Ancoli-Israel;,citation_author=Kenneth P. Wright;,citation_author=Derk-Jan Dijk;,citation_author=Phyllis Zee;,citation_author=Michael R. Gorman;,citation_author=Eva C. Winnebeck;,citation_author=Elizabeth B. Klerman;,citation_publication_date=2019-06;,citation_cover_date=2019-06;,citation_year=2019;,citation_fulltext_html_url=http://journals.sagepub.com/doi/10.1177/0748730419854197;,citation_issue=3;,citation_doi=10.1177/0748730419854197;,citation_issn=0748-7304, 1552-4531;,citation_volume=34;,citation_language=en-US;,citation_journal_title=Journal of Biological Rhythms;,citation_journal_abbrev=J Biol Rhythms;">
<meta name="citation_reference" content="citation_title=Chronotype and social jetlag: A (self-) critical review;,citation_abstract=The Munich ChronoType Questionnaire (MCTQ) has now been available for more than 15 years and its original publication has been cited 1240 times (Google Scholar, May 2019). Additionally, its online version, which was available until July 2017, produced almost 300,000 entries from all over the world (MCTQ database). The MCTQ has gone through several versions, has been translated into 13 languages, and has been validated against other more objective measures of daily timing in several independent studies. Besides being used as a method to correlate circadian features of human biology with other factors—ranging from health issues to geographical factors—the MCTQ gave rise to the quantification of old wisdoms, like “teenagers are late”, and has produced new concepts, like social jetlag. Some like the MCTQ’s simplicity and some view it critically. Therefore, it is time to present a self-critical view on the MCTQ, to address some misunderstandings, and give some definitions of the MCTQ-derived chronotype and the concept of social jetlag.;,citation_author=Till Roenneberg;,citation_author=Luísa K. Pilz;,citation_author=Giulia Zerbini;,citation_author=Eva C. Winnebeck;,citation_publication_date=2019-07-12;,citation_cover_date=2019-07-12;,citation_year=2019;,citation_fulltext_html_url=https://www.mdpi.com/2079-7737/8/3/54;,citation_issue=3;,citation_doi=10.3390/biology8030054;,citation_issn=2079-7737;,citation_volume=8;,citation_language=en-US;,citation_journal_title=Biology;,citation_journal_abbrev=Biology;">
<meta name="citation_reference" content="citation_title=Early chronotype with advanced activity rhythms and dim light melatonin onset in a rural population;,citation_author=Francieli S. Ruiz;,citation_author=Felipe Beijamini;,citation_author=Andrew D. Beale;,citation_author=Bruno Da Silva B. Gonçalves;,citation_author=Daniel Vartanian;,citation_author=Tâmara P. Taporoski;,citation_author=Benita Middleton;,citation_author=José E. Krieger;,citation_author=Homero Vallada;,citation_author=Josephine Arendt;,citation_author=Alexandre C. Pereira;,citation_author=Kristen L. Knutson;,citation_author=Mario Pedrazzoli;,citation_author=Malcolm Von Schantz;,citation_publication_date=2020-10;,citation_cover_date=2020-10;,citation_year=2020;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/jpi.12675;,citation_issue=3;,citation_doi=10.1111/jpi.12675;,citation_issn=0742-3098, 1600-079X;,citation_volume=69;,citation_language=en-US;,citation_journal_title=Journal of Pineal Research;,citation_journal_abbrev=J. Pineal. Res.;">
<meta name="citation_reference" content="citation_title=The circadian clock of the bacterium B. Subtilis evokes properties of complex, multicellular circadian systems;,citation_abstract=Circadian clocks are pervasive throughout nature, yet only recently has this adaptive regulatory program been described in nonphotosynthetic bacteria. Here, we describe an inherent complexity in the Bacillus subtilis circadian clock. We find that B. subtilis entrains to blue and red light and that circadian entrainment is separable from masking through fluence titration and frequency demultiplication protocols. We identify circadian rhythmicity in constant light, consistent with the Aschoff’s rule, and entrainment aftereffects, both of which are properties described for eukaryotic circadian clocks. We report that circadian rhythms occur in wild isolates of this prokaryote, thus establishing them as a general property of this species, and that its circadian system responds to the environment in a complex fashion that is consistent with multicellular eukaryotic circadian systems.;,citation_author=Francesca Sartor;,citation_author=Xinming Xu;,citation_author=Tanja Popp;,citation_author=Antony N. Dodd;,citation_author=Ákos T. Kovács;,citation_author=Martha Merrow;,citation_publication_date=2023-08-04;,citation_cover_date=2023-08-04;,citation_year=2023;,citation_fulltext_html_url=https://www.science.org/doi/full/10.1126/sciadv.adh1308;,citation_issue=31;,citation_doi=10.1126/sciadv.adh1308;,citation_volume=9;,citation_language=en-US;,citation_journal_title=Science Advances;,citation_publisher=American Association for the Advancement of Science;">
<meta name="citation_reference" content="citation_title=Introduction to the modeling and analysis of complex systems;,citation_author=Hiroki Sayama;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_isbn=978-1-942341-09-3;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Preliminary goodness-of-fit tests for normality do not validate the one-sample Student t;,citation_abstract=One of the most basic topics in many introductory statistical methods texts is inference for a population mean, μ. The primary tool for confidence intervals and tests is the Student t sampling dist...;,citation_author=William R. Schucany;,citation_author=H. K. Tony Ng;,citation_publication_date=2006-12-01;,citation_cover_date=2006-12-01;,citation_year=2006;,citation_fulltext_html_url=https://www.tandfonline.com/doi/abs/10.1080/03610920600853308;,citation_issue=12;,citation_doi=10.1080/03610920600853308;,citation_volume=35;,citation_language=en-US;,citation_journal_title=Communications in Statistics - Theory and Methods;,citation_publisher=Taylor &amp;amp;amp; Francis Group;">
<meta name="citation_reference" content="citation_title=An introduction to the history of chronobiology: Biological rhythms emerge as a subject of scientific research;,citation_abstract=In three volumes, historian Jole Shackelford delineates the history of the study of biological rhythms—now widely known as chronobiology—from antiquity into the twentieth century. Perhaps the most well-known biological rhythm is the circadian rhythm, tied to the cycles of day and night and often referred to as the “body clock.” But there are many other biological rhythms, and although scientists and the natural philosophers who preceded them have long known about them, only in the past thirty years have a handful of pioneering scientists begun to study such rhythms in plants and animals seriously. Tracing the intellectual and institutional development of biological rhythm studies, Shackelford offers a meaningful, evidence-based account of a field that today holds great promise for applications in agriculture, health care, and public health. Volume 1 follows early biological observations and research, chiefly on plants; volume 2 turns to animal and human rhythms and the disciplinary contexts for chronobiological investigation; and volume 3 focuses primarily on twentieth-century researchers who modeled biological clocks and sought them out, including three molecular biologists whose work in determining clock mechanisms earned them a Nobel Prize in 2017.;,citation_author=Jole Shackelford;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_isbn=978-0-8229-4732-5;,citation_volume=1;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=An analysis of variance test for normality (complete samples)†;,citation_author=S. S. Shapiro;,citation_author=M. B. Wilk;,citation_publication_date=1965-12-01;,citation_cover_date=1965-12-01;,citation_year=1965;,citation_fulltext_html_url=https://doi.org/10.1093/biomet/52.3-4.591;,citation_issue=3–4;,citation_doi=10.1093/biomet/52.3-4.591;,citation_issn=0006-3444;,citation_volume=52;,citation_language=en-US;,citation_journal_title=Biometrika;,citation_journal_abbrev=Biometrika;">
<meta name="citation_reference" content="citation_title=An approximate analysis of variance test for normality;,citation_abstract=This article presents a modification of the Shapiro-Wilk W statistic for testing normality which can be used with large samples. Shapiro and Wilk gave coefficients and percentage points for sample sizes up to 50. These coefficients required obtaining an approximation to the covariance matrix of the normal order statistics. The proposed test uses coefficients which depend only on the expected values of the normal order statistics which are generally available. Results of an empirical sampling study to compare the sensitivity of the test statistic to the W test statistic are briefly discussed.;,citation_author=S. S. Shapiro;,citation_author=R. S. Francia;,citation_publication_date=1972-03-01;,citation_cover_date=1972-03-01;,citation_year=1972;,citation_fulltext_html_url=https://www.tandfonline.com/doi/abs/10.1080/01621459.1972.10481232;,citation_issue=337;,citation_doi=10.1080/01621459.1972.10481232;,citation_issn=0162-1459;,citation_volume=67;,citation_language=en-US;,citation_journal_title=Journal of the American Statistical Association;,citation_publisher=ASA Website;">
<meta name="citation_reference" content="citation_title=Assumption-checking rather than (just) testing: The importance of visualization and effect size in statistical diagnostics;,citation_abstract=Statistical methods generally have assumptions (e.g., normality in linear regression models). Violations of these assumptions can cause various issues, like statistical errors and biased estimates, whose impact can range from inconsequential to critical. Accordingly, it is important to check these assumptions, but this is often done in a flawed way. Here, I first present a prevalent but problematic approach to diagnostics—testing assumptions using null hypothesis significance tests (e.g., the Shapiro–Wilk test of normality). Then, I consolidate and illustrate the issues with this approach, primarily using simulations. These issues include statistical errors (i.e., false positives, especially with large samples, and false negatives, especially with small samples), false binarity, limited descriptiveness, misinterpretation (e.g., of p-value as an&nbsp;effect size), and potential testing failure due to unmet test assumptions. Finally, I synthesize the implications of these issues for statistical diagnostics, and provide practical recommendations for improving such diagnostics. Key recommendations include maintaining awareness of the issues with assumption tests (while recognizing they can be useful), using appropriate combinations of diagnostic methods (including visualization and effect sizes) while recognizing their limitations, and distinguishing between testing and checking assumptions. Additional recommendations include judging assumption violations as a complex spectrum (rather than a simplistic binary), using programmatic tools that increase replicability and decrease researcher degrees of freedom, and sharing the material and rationale involved in the diagnostics.;,citation_author=Itamar Shatz;,citation_publication_date=2024-02-01;,citation_cover_date=2024-02-01;,citation_year=2024;,citation_fulltext_html_url=https://doi.org/10.3758/s13428-023-02072-x;,citation_issue=2;,citation_doi=10.3758/s13428-023-02072-x;,citation_issn=1554-3528;,citation_volume=56;,citation_language=en-US;,citation_journal_title=Behavior Research Methods;,citation_journal_abbrev=Behav Res;">
<meta name="citation_reference" content="citation_title=Changes in daily activity patterns throughout the year in a free-living South American subterranean rodent (Ctenomys coludo);,citation_abstract=Many species of small mammals inhabit subterranean galleries, where daily environmental cycles such as light intensity and temperature, otherwise conspicuous aboveground, are either absent or attenuated. The presence of biological rhythms was investigated in many species of these subterranean habitats. Activity patterns of the subterranean rodent tuco-tuco (Ctenomys coludo), have been studied in laboratory and semi-natural enclosures. Here we extended the description in free-living field conditions. Data from 21 tuco-tucos were collected using animal-borne accelerometers and light-loggers in 4 field campaigns across a year, covering all seasons. Accelerometer data confirmed diurnality in free-living conditions and showed that diurnality does not change throughout the year. However, a significant reduction in general activity was observed during winter. To extract more temporal information from accelerometer data, we used hidden Markov models (HMMs) classifying the data into three distinct activity states roughly corresponding to low-, medium-, and high-intensity activity. While high-intensity activities were day-concentrated and low-intensity was concentrated at night, medium-intensity activity bouts were spread throughout day and night. Furthermore, tuco-tucos displayed significant changes in the time spent in different activity states throughout the year, clarifying that the observed changes in activity levels in winter were due to reduced high activity, allocating more time to medium activities, not rest. Light-logger data confirmed the seasonal variation of daily surface emergence, concentrated in twilight hours during summer and around mid-day in winter, in contrast to general activity, which peaked around mid-day throughout the year. In addition, the HMM analysis clarified that tuco-tucos were mainly in a high-intensity activity state on the surface, which did not change throughout the year. The combination of biologgers and new analytical tools enabled more detailed characterization of activity patterns in free-living animals, expanding the scope of biological clock studies in natural conditions.;,citation_author=Jefferson T. Silvério;,citation_author=Patricia Tachinardi;,citation_author=Roland Langrock;,citation_author=Arne Kramer-Sunderbrink;,citation_author=Gisele A. Oda;,citation_author=Veronica S. Valentinuzzi;,citation_publication_date=2024-12-03;,citation_cover_date=2024-12-03;,citation_year=2024;,citation_fulltext_html_url=https://doi.org/10.1007/s42991-024-00470-y;,citation_doi=10.1007/s42991-024-00470-y;,citation_issn=1618-1476;,citation_language=en-US;,citation_journal_title=Mammalian Biology;,citation_journal_abbrev=Mamm Biol;">
<meta name="citation_reference" content="citation_title=Weekly and seasonal variation in the circadian melatonin rhythm in humans: Entrained to local clock time, social time, light exposure or sun time?;,citation_author=Anne C. Skeldon;,citation_author=Derk-Jan Dijk;,citation_publication_date=2021-08;,citation_cover_date=2021-08;,citation_year=2021;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/jpi.12746;,citation_issue=1;,citation_doi=10.1111/jpi.12746;,citation_issn=0742-3098, 1600-079X;,citation_volume=71;,citation_language=en-US;,citation_journal_title=Journal of Pineal Research;,citation_journal_abbrev=Journal of Pineal Research;">
<meta name="citation_reference" content="citation_title=Table for estimating the goodness of fit of empirical distributions;,citation_author=N. Smirnov;,citation_publication_date=1948;,citation_cover_date=1948;,citation_year=1948;,citation_volume=19;,citation_journal_title=Annals of Mathematical Statistics;">
<meta name="citation_reference" content="citation_title=Regression Diagnostics with R;,citation_abstract=This book uses R. A Stata version of this book is available at Regression Diagnostics with Stata. Regression diagnostics are a critical step in the modeling process. Diagnostics for regression models are tools that assess a model’s compliance to its assumptions and investigate if there is a single observation or group of observations that are not well represented by the model. These tools allow researchers to evaluate if a model appropriately represents the data of their study. In this book we separate diagnostics from the other parts of model selection to provide a focus on this important topic. This separation is not meant to imply that these tools are used separately from other regression modeling tools.;,citation_author=Jason Struck;,citation_publication_date=2024-06;,citation_cover_date=2024-06;,citation_year=2024;,citation_fulltext_html_url=https://sscc.wisc.edu/sscc/pubs/RegDiag-R/;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=An action spectrum for melatonin suppression: Evidence for a novel non-rod, non-cone photoreceptor system in humans;,citation_abstract=1 Non-image forming, irradiance-dependent responses mediated by the human eye include synchronisation of the circadian axis and suppression of pineal melatonin production. The retinal photopigment(s) transducing these light responses in humans have not been characterised. 2 Using the ability of light to suppress nocturnal melatonin production, we aimed to investigate its spectral sensitivity and produce an action spectrum. Melatonin suppression was quantified in 22 volunteers in 215 light exposure trials using monochromatic light (30 min pulse administered at circadian time (CT) 16-18) of different wavelengths (λmax 424, 456, 472, 496, 520 and 548 nm) and irradiances (0.7-65.0 μW cm−2). 3 At each wavelength, suppression of plasma melatonin increased with increasing irradiance. Irradiance-response curves (IRCs) were fitted and the generated half-maximal responses (IR50) were corrected for lens filtering and used to construct an action spectrum. 4 The resulting action spectrum showed unique short-wavelength sensitivity very different from the classical scotopic and photopic visual systems. The lack of fit (r2 $&amp;amp;amp;lt;$ 0.1) of our action spectrum with the published rod and cone absorption spectra precluded these photoreceptors from having a major role. Cryptochromes 1 and 2 also had a poor fit to the data. Fitting a series of Dartnall nomograms generated for rhodopsin-based photopigments over the λmax range 420-480 nm showed that rhodopsin templates between λmax 457 and 462 nm fitted the data well (r2≥ 0.73). Of these, the best fit was to the rhodopsin template with λmax 459 nm (r2= 0.74). 5 Our data strongly support a primary role for a novel short-wavelength photopigment in light-induced melatonin suppression and provide the first direct evidence of a non-rod, non-cone photoreceptive system in humans.;,citation_author=Kavita Thapan;,citation_author=Josephine Arendt;,citation_author=Debra J. Skene;,citation_publication_date=2001;,citation_cover_date=2001;,citation_year=2001;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1469-7793.2001.t01-1-00261.x;,citation_issue=1;,citation_doi=10.1111/j.1469-7793.2001.t01-1-00261.x;,citation_issn=1469-7793;,citation_volume=535;,citation_language=en-US;,citation_journal_title=The Journal of Physiology;">
<meta name="citation_reference" content="citation_title=MCTQ;,citation_author=The Worldwide Experimental Platform;,citation_fulltext_html_url=https://www.thewep.org/documentations/mctq;,citation_language=en-US;,citation_publisher=The Worldwide Experimental Platform;">
<meta name="citation_reference" content="citation_title=Testing for normality;,citation_author=Henry C. Thode;,citation_publication_date=2002;,citation_cover_date=2002;,citation_year=2002;,citation_isbn=978-0-8247-9613-6;,citation_language=en-US;,citation_series_title=Statistics, textbooks and monographs;">
<meta name="citation_reference" content="citation_title=Solstices &amp;amp;amp; equinoxes for UTC (2000–2049);,citation_abstract=Exact time and date of the June solstice and other seasons in UTC, Time Zone.;,citation_author=Time and Date AS;,citation_fulltext_html_url=https://www.timeanddate.com/calendar/seasons.html?year=2000&amp;amp;n=1440;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=renv: Project environments;,citation_abstract=A dependency management toolkit for R. Using ’renv’, you can create and manage project-local R libraries, save the state of these libraries to a ’lockfile’, and later restore your library as required. Together, these tools can help make your projects more isolated, portable, and reproducible.;,citation_author=Kevin Ushey;,citation_author=Hadley Wickham;,citation_fulltext_html_url=https://doi.org/10.32614/CRAN.package.renv;">
<meta name="citation_reference" content="citation_title=The stringdist package for approximate string matching;,citation_abstract=Implements an approximate string matching version of R’s native ’match’ function. Also offers fuzzy text search based on various string distance measures. Can calculate various string distances based on edits (Damerau-Levenshtein, Hamming, Levenshtein, optimal sting alignment), qgrams (q- gram, cosine, jaccard distance) or heuristic metrics (Jaro, Jaro-Winkler). An implementation of soundex is provided as well. Distances can be computed between character vectors while taking proper care of encoding or between integer vectors representing generic sequences. This package is built for speed and runs in parallel by using ’openMP’. An API for C or C++ is exposed as well.;,citation_author=Mark P. J. Loo;,citation_fulltext_html_url=https://CRAN.R-project.org/package=stringdist;">
<meta name="citation_reference" content="citation_title=Ecology of sleep and circadian phenotypes of the Brazilian population [Poster];,citation_author=Daniel Vartanian;,citation_publication_date=2022-11-19;,citation_cover_date=2022-11-19;,citation_year=2022;,citation_fulltext_html_url=https://doi.org/10.13140/RG.2.2.25343.07840;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Plano de ensino: ACH0042 - Resolução de Problemas II;,citation_author=Daniel Vartanian;,citation_publication_date=2023-07-12;,citation_cover_date=2023-07-12;,citation_year=2023;,citation_fulltext_html_url=https://doi.org/10.13140/RG.2.2.33335.50086;,citation_language=pt-BR;">
<meta name="citation_reference" content="citation_title=Is latitude associated with chronotype? [Data Management Plan];,citation_author=Daniel Vartanian;,citation_publication_date=2024-11-26;,citation_cover_date=2024-11-26;,citation_year=2024;,citation_fulltext_html_url=https://doi.org/10.48321/D1B5C8068D;,citation_language=en-US;,citation_publisher=DMPHub;">
<meta name="citation_reference" content="citation_title=Is latitude associated with chronotype? [Research compendium];,citation_abstract=Research compendium of the Master’s Thesis: Is Latitude Associated with Chronotype?;,citation_author=Daniel Vartanian;,citation_publication_date=2025;,citation_cover_date=2025;,citation_year=2025;,citation_fulltext_html_url=https://doi.org/10.17605/OSF.IO/YGKTS;,citation_doi=10.17605/OSF.IO/YGKTS;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title={prettycheck}: Pretty assertive programming;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://github.com/danielvartan/prettycheck;">
<meta name="citation_reference" content="citation_title={lubritime}: An extension for the lubridate package;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://github.com/danielvartan/lubritime;">
<meta name="citation_reference" content="citation_title={lockr}: Easily encrypt/decrypt files;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://github.com/danielvartan/lockr;">
<meta name="citation_reference" content="citation_title={entrainment}: A rule-based model of the 24h light/dark cycle entrainment phenomenon;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://github.com/danielvartan/entrainment;">
<meta name="citation_reference" content="citation_title={abnt}: Quarto format designed for theses and dissertations that adhere to the standards of the Brazilian Association of Technical Standards (ABNT);,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://github.com/danielvartan/abnt;">
<meta name="citation_reference" content="citation_title={groomr}: Tidy tools;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://github.com/danielvartan/groomr;">
<meta name="citation_reference" content="citation_title={mctq}: Munich ChronoType Questionnaire tools;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://docs.ropensci.org/mctq/;">
<meta name="citation_reference" content="citation_title=Illustration of the SIR model;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://danielvartan.github.io/sir;">
<meta name="citation_reference" content="citation_title=Illustration of the Lorenz system;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://danielvartan.github.io/lorenz-system;">
<meta name="citation_reference" content="citation_title=A demonstration of general linear models using penguins;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://danielvartan.github.io/linear-models;">
<meta name="citation_reference" content="citation_title=Demonstrations of the central limit theorem;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://danielvartan.github.io/central-limit-theorem;">
<meta name="citation_reference" content="citation_title=Illustration of the Lotka–Volterra’s predator–prey model;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://danielvartan.github.io/lotka-volterra;">
<meta name="citation_reference" content="citation_title=HTML color codes in hex and RGB formats for Espanso;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://hub.espanso.org/html-colors;">
<meta name="citation_reference" content="citation_title={actverse}: Tools for actigraphy data analysis;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://danielvartan.github.io/actverse/;">
<meta name="citation_reference" content="citation_title={quartor}: Tools for the Quarto publishing system;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://danielvartan.github.io/quartor/;">
<meta name="citation_reference" content="citation_title={orbis}: Spatial data analysis tools;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://danielvartan.github.io/orbis/;">
<meta name="citation_reference" content="citation_title={brandr}: Brand identity management using brand.yml standard;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://doi.org/10.32614/CRAN.package.brandr;">
<meta name="citation_reference" content="citation_title={plotr}: Plots made easy;,citation_author=Daniel Vartanian;,citation_fulltext_html_url=https://danielvartan.github.io/plotr/;">
<meta name="citation_reference" content="citation_title=ViaCEP API: Consulte CEPs de todo o Brasil;,citation_abstract=Webservice gratuito de alto desempenho para consulta de Código de Endereçamento Postal (CEP) do Brasil.;,citation_author=ViaCEP;,citation_fulltext_html_url=https://viacep.com.br;">
<meta name="citation_reference" content="citation_title=Actigraphic characterization of sleep and circadian phenotypes of PER3 gene VNTR genotypes;,citation_author=Juliana Viana-Mendes;,citation_author=Ana Amélia Benedito-Silva;,citation_author=Maria Augusta Medeiros Andrade;,citation_author=Daniel Vartanian;,citation_author=Bruno da Silva Brandão Gonçalves;,citation_author=José Cipolla-Neto;,citation_author=Mario Pedrazzoli;,citation_publication_date=2023-09-10;,citation_cover_date=2023-09-10;,citation_year=2023;,citation_fulltext_html_url=https://www.tandfonline.com/doi/full/10.1080/07420528.2023.2256858;,citation_issue=9;,citation_doi=10.1080/07420528.2023.2256858;,citation_issn=0742-0528, 1525-6073;,citation_volume=40;,citation_language=en-US;,citation_journal_title=Chronobiology International;,citation_journal_abbrev=Chronobiology International;">
<meta name="citation_reference" content="citation_title=General system theory: Foundations, development, applications;,citation_author=Ludwig Bertalanffy;,citation_publication_date=1968;,citation_cover_date=1968;,citation_year=1968;,citation_isbn=978-0-8076-0453-3;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Consistency of chronotype measurements is affected by sleep quality, gender, longitude, and latitude;,citation_author=Haien Wang;,citation_author=Shuo Wang;,citation_author=Wenqing Yu;,citation_author=Xu Lei;,citation_publication_date=2023-07-03;,citation_cover_date=2023-07-03;,citation_year=2023;,citation_fulltext_html_url=https://www.tandfonline.com/doi/abs/10.1080/07420528.2023.2237118;,citation_issue=7;,citation_doi=10.1080/07420528.2023.2237118;,citation_issn=0742-0528;,citation_volume=40;,citation_journal_title=Chronobiology International;,citation_publisher=Taylor &amp;amp;amp; Francis;">
<meta name="citation_reference" content="citation_title=The ASA statement on p-values: Context, process, and purpose;,citation_author=Ronald L. Wasserstein;,citation_author=Nicole A. Lazar;,citation_publication_date=2016-04-02;,citation_cover_date=2016-04-02;,citation_year=2016;,citation_fulltext_html_url=https://www.tandfonline.com/doi/abs/10.1080/00031305.2016.1154108;,citation_issue=2;,citation_doi=10.1080/00031305.2016.1154108;,citation_issn=0003-1305;,citation_volume=70;,citation_language=en-US;,citation_journal_title=The American Statistician;,citation_publisher=Taylor &amp;amp;amp; Francis;">
<meta name="citation_reference" content="citation_title=Recommended amount of sleep for a healthy adult: A joint consensus statement of the American Academy of Sleep Medicine and Sleep Research Society;,citation_author=Nathaniel F. Watson;,citation_author=M. Safwan Badr;,citation_author=Gregory Belenky;,citation_author=Donald L. Bliwise;,citation_author=Orfeu M. Buxton;,citation_author=Daniel Buysse;,citation_author=David F. Dinges;,citation_author=James Gangwisch;,citation_author=Michael A. Grandner;,citation_author=Clete Kushida;,citation_author=Raman K. Malhotra;,citation_author=Jennifer L. Martin;,citation_author=Sanjay R. Patel;,citation_author=Stuart F. Quan;,citation_author=Esra Tasali;,citation_publication_date=2015-06-15;,citation_cover_date=2015-06-15;,citation_year=2015;,citation_fulltext_html_url=http://jcsm.aasm.org/doi/10.5664/jcsm.4758;,citation_issue=6;,citation_doi=10.5664/jcsm.4758;,citation_issn=1550-9389, 1550-9397;,citation_volume=11;,citation_language=en-US;,citation_journal_title=Journal of Clinical Sleep Medicine;,citation_journal_abbrev=Journal of Clinical Sleep Medicine;">
<meta name="citation_reference" content="citation_title=Linear regression diagnostics;,citation_author=Roy Welsch;,citation_author=Edwin Kuh;,citation_publication_date=1977-03;,citation_cover_date=1977-03;,citation_year=1977;,citation_fulltext_html_url=http://www.nber.org/papers/w0173.pdf;,citation_doi=10.3386/w0173;,citation_language=en-US;,citation_technical_report_institution=National Bureau of Economic Research;,citation_technical_report_number=0173;">
<meta name="citation_reference" content="citation_title=The tidy tools manifesto;,citation_abstract=tidyverse;,citation_author=Hadley Wickham;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://tidyverse.tidyverse.org/articles/manifesto.html;,citation_language=en-US;,citation_publisher=Tidyverse;">
<meta name="citation_reference" content="citation_title=R for data science: Import, tidy, transform, visualize, and model data;,citation_abstract=Use R to turn data into insight, knowledge, and understanding. With this practical book, aspiring data scientists will learn how to do data science with R and RStudio, along with the tidyverseâ??a collection of R packages designed to work together to make data science fast, fluent, and fun. Even if you have no programming experience, this updated edition will have you doing data science quickly. You’ll learn how to import, transform, and visualize your data and communicate the results. And you’ll get a complete, big-picture understanding of the data science cycle and the basic tools you need to manage the details. Updated for the latest tidyverse features and best practices, new chapters show you how to get data from spreadsheets, databases, and websites. Exercises help you practice what you’ve learned along the way.;,citation_author=Hadley Wickham;,citation_author=Mine Çetinkaya-Rundel;,citation_author=Garrett Grolemund;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://r4ds.hadley.nz;,citation_isbn=978-1-4920-9740-2;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=The FAIR guiding principles for scientific data management and stewardship;,citation_abstract=There is an urgent need to improve the infrastructure supporting the reuse of scholarly data. A diverse set of stakeholders—representing academia, industry, funding agencies, and scholarly publishers—have come together to design and jointly endorse a concise and measureable set of principles that we refer to as the FAIR Data Principles. The intent is that these may act as a guideline for those wishing to enhance the reusability of their data holdings. Distinct from peer initiatives that focus on the human scholar, the FAIR Principles put specific emphasis on enhancing the ability of machines to automatically find and use the data, in addition to supporting its reuse by individuals. This Comment is the first formal publication of the FAIR Principles, and includes the rationale behind them, and some exemplar implementations in the community.;,citation_author=Mark D. Wilkinson;,citation_author=Michel Dumontier;,citation_author=IJsbrand Jan Aalbersberg;,citation_author=Gabrielle Appleton;,citation_author=Myles Axton;,citation_author=Arie Baak;,citation_author=Niklas Blomberg;,citation_author=Jan-Willem Boiten;,citation_author=Luiz Bonino Da Silva Santos;,citation_author=Philip E. Bourne;,citation_author=Jildau Bouwman;,citation_author=Anthony J. Brookes;,citation_author=Tim Clark;,citation_author=Mercè Crosas;,citation_author=Ingrid Dillo;,citation_author=Olivier Dumon;,citation_author=Scott Edmunds;,citation_author=Chris T. Evelo;,citation_author=Richard Finkers;,citation_author=Alejandra Gonzalez-Beltran;,citation_author=Alasdair J. G. Gray;,citation_author=Paul Groth;,citation_author=Carole Goble;,citation_author=Jeffrey S. Grethe;,citation_author=Jaap Heringa;,citation_author=Peter A. C Hoen;,citation_author=Rob Hooft;,citation_author=Tobias Kuhn;,citation_author=Ruben Kok;,citation_author=Joost Kok;,citation_author=Scott J. Lusher;,citation_author=Maryann E. Martone;,citation_author=Albert Mons;,citation_author=Abel L. Packer;,citation_author=Bengt Persson;,citation_author=Philippe Rocca-Serra;,citation_author=Marco Roos;,citation_author=Rene Van Schaik;,citation_author=Susanna-Assunta Sansone;,citation_author=Erik Schultes;,citation_author=Thierry Sengstag;,citation_author=Ted Slater;,citation_author=George Strawn;,citation_author=Morris A. Swertz;,citation_author=Mark Thompson;,citation_author=Johan Van Der Lei;,citation_author=Erik Van Mulligen;,citation_author=Jan Velterop;,citation_author=Andra Waagmeester;,citation_author=Peter Wittenburg;,citation_author=Katherine Wolstencroft;,citation_author=Jun Zhao;,citation_author=Barend Mons;,citation_publication_date=2016-03-15;,citation_cover_date=2016-03-15;,citation_year=2016;,citation_fulltext_html_url=https://www.nature.com/articles/sdata201618;,citation_issue=1;,citation_doi=10.1038/sdata.2016.18;,citation_issn=2052-4463;,citation_volume=3;,citation_language=en-US;,citation_journal_title=Scientific Data;,citation_journal_abbrev=Sci Data;">
<meta name="citation_reference" content="citation_title=Emergence isn’t magic;,citation_abstract=Pondering the hard problem of consciousness, Simon Blackburn quite rightly describes “old-fashioned emergence” of mind from brain as “magic tissue secreting a magical effect” (11 September, p 42). But this does not mean emergence necessarily entails smoke and mirrors. It is a familiar phenomenon that science helps us to understand. As we learn in school, if you chemically combine the gases oxygen and hydrogen the result is liquid water. There is no magic involved in the fact that the properties of water are different from its constituent elements. Equally, the emergence of consciousness from certain arrangements of neurons is a natural process. The problem of how it comes about is hard because we have only just begun to investigate the complexity of the brain. While agreeing that neuroscientists can benefit from critical analysis of their ideas by philosophers, the latter tend to be led astray by metaphysical speculation. The Cartesian dualist tradition is a case in point. Modern dualists such as David Chalmers argue that the emergence of consciousness from the brain is not just remarkable, but logically impossible. And so it would be if Descartes’ claim that the conscious mind is separate from the mechanistic body were true. Assuming a logical division between the mental and the physical begs the mind-body question. Mental and physical properties do appear to be radically different but we simply don’t know whether this is a real difference. Bicester, Oxfordshire, UK;,citation_author=Richard Wilson;,citation_publication_date=2004-10-02;,citation_cover_date=2004-10-02;,citation_year=2004;,citation_fulltext_html_url=https://www.newscientist.com/letter/mg18424674-700-emergence-isnt-magic/;,citation_issue=2467;,citation_language=en-US;,citation_journal_title=New Scientist;">
</head>
<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top"><nav class="quarto-secondary-nav"><div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../qmd/supplementary-material-1.html">Appendices</a></li><li class="breadcrumb-item"><a href="../qmd/supplementary-material-8.html"><span class="chapter-number">H</span>&nbsp; <span class="chapter-title">Hypothesis Test B</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav></header><!-- content --><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto"><div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">Is Latitude Associated with Chronotype?</a> 
        <div class="sidebar-tools-main tools-wide">
    <a href="../../print/index.pdf" title="" class="quarto-navigation-tool px-1" aria-label=""><i class="bi bi-file-earmark-pdf-fill"></i></a>
    <a href="https://github.com/danielvartan/mastersthesis/" title="Source Code" class="quarto-navigation-tool px-1" aria-label="Source Code"><i class="bi bi-github"></i></a>
  <a href="" class="quarto-reader-toggle quarto-navigation-tool px-1" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
<li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Welcome</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Preliminary Sections</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">
<li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/acknowledgments.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Acknowledgments</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/epigraph.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Epigraph</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/vernacular-abstract.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Abstract</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/foreign-abstract.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Resumo</span></a>
  </div>
</li>
      </ul>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true">
 <span class="menu-text">Chapters</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">
<li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/chapter-1.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Introduction</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/chapter-2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">On Chronobiology</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/chapter-3.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">On Complexity Science</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/chapter-4.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">On the Latitude Hypothesis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/chapter-5.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Is Latitude Associated with Chronotype?</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/chapter-6.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Conclusion</span></span></a>
  </div>
</li>
      </ul>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true">
 <span class="menu-text">Appendices</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">
<li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-1.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">A</span>&nbsp; <span class="chapter-title">Question, Objective and Hypothesis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">B</span>&nbsp; <span class="chapter-title">Methods</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-3.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">C</span>&nbsp; <span class="chapter-title">Exploratory Data Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-4.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">D</span>&nbsp; <span class="chapter-title">Sample Balance</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-5.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">E</span>&nbsp; <span class="chapter-title">Power Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-6.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">F</span>&nbsp; <span class="chapter-title">Restricted Model</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-7.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">G</span>&nbsp; <span class="chapter-title">Hypothesis Test A</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-8.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">H</span>&nbsp; <span class="chapter-title">Hypothesis Test B</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-9.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">I</span>&nbsp; <span class="chapter-title">Comparative Assessment of QualoCEP and Google Geocoding API Performance</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-10.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">J</span>&nbsp; <span class="chapter-title">Overview of General Linear Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-11.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">K</span>&nbsp; <span class="chapter-title">Critique of Leocadio-Miguel et al.&nbsp;(2017)</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../qmd/supplementary-material-12.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">L</span>&nbsp; <span class="chapter-title">Works and Activities Developed During the Thesis</span></span></a>
  </div>
</li>
      </ul>
</li>
    </ul>
</div>
</nav><div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active"><h2 id="toc-title">Table of contents</h2>
   
  <ul>
<li><a href="#overview" id="toc-overview" class="nav-link active" data-scroll-target="#overview"><span class="header-section-number">H.1</span> Overview</a></li>
  <li><a href="#setting-the-environment" id="toc-setting-the-environment" class="nav-link" data-scroll-target="#setting-the-environment"><span class="header-section-number">H.2</span> Setting the Environment</a></li>
  <li><a href="#loading-and-processing-the-data" id="toc-loading-and-processing-the-data" class="nav-link" data-scroll-target="#loading-and-processing-the-data"><span class="header-section-number">H.3</span> Loading and Processing the Data</a></li>
  <li><a href="#building-the-restricted-model" id="toc-building-the-restricted-model" class="nav-link" data-scroll-target="#building-the-restricted-model"><span class="header-section-number">H.4</span> Building the Restricted Model</a></li>
  <li>
<a href="#building-the-full-model" id="toc-building-the-full-model" class="nav-link" data-scroll-target="#building-the-full-model"><span class="header-section-number">H.5</span> Building the Full Model</a>
  <ul class="collapse">
<li><a href="#fitting-the-model" id="toc-fitting-the-model" class="nav-link" data-scroll-target="#fitting-the-model"><span class="header-section-number">H.5.1</span> Fitting the Model</a></li>
  <li><a href="#evaluating-the-model-fit" id="toc-evaluating-the-model-fit" class="nav-link" data-scroll-target="#evaluating-the-model-fit"><span class="header-section-number">H.5.2</span> Evaluating the Model Fit</a></li>
  <li><a href="#conducting-model-diagnostics" id="toc-conducting-model-diagnostics" class="nav-link" data-scroll-target="#conducting-model-diagnostics"><span class="header-section-number">H.5.3</span> Conducting Model Diagnostics</a></li>
  </ul>
</li>
  <li>
<a href="#hypothesis-testing" id="toc-hypothesis-testing" class="nav-link" data-scroll-target="#hypothesis-testing"><span class="header-section-number">H.6</span> Hypothesis Testing</a>
  <ul class="collapse">
<li><a href="#f-test" id="toc-f-test" class="nav-link" data-scroll-target="#f-test"><span class="header-section-number">H.6.1</span> F-Test</a></li>
  <li><a href="#effect-size" id="toc-effect-size" class="nav-link" data-scroll-target="#effect-size"><span class="header-section-number">H.6.2</span> Effect Size</a></li>
  </ul>
</li>
  <li><a href="#conclusion" id="toc-conclusion" class="nav-link" data-scroll-target="#conclusion"><span class="header-section-number">H.7</span> Conclusion</a></li>
  </ul><div class="toc-actions"><ul><li><a href="https://github.com/danielvartan/mastersthesis/edit/main/qmd/supplementary-material-8.qmd" class="toc-action"><i class="bi bi-github"></i>Edit this page</a></li><li><a href="https://github.com/danielvartan/mastersthesis/issues/" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content"><header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../qmd/supplementary-material-1.html">Appendices</a></li><li class="breadcrumb-item"><a href="../qmd/supplementary-material-8.html"><span class="chapter-number">H</span>&nbsp; <span class="chapter-title">Hypothesis Test B</span></a></li></ol></nav><div class="quarto-title">
<h1 class="title"><span id="sec-sm-hypothesis-test-b" class="quarto-section-identifier">Appendix H — Hypothesis Test B</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header><!-- %:::% .common h1 begin %:::% --><!-- %:::% .common h1 end %:::% --><section id="overview" class="level2" data-number="H.1"><h2 data-number="H.1" class="anchored" data-anchor-id="overview">
<span class="header-section-number">H.1</span> Overview</h2>
<p>This document presents the analysis for testing the thesis hypothesis (<strong>Test B</strong>) using the methodology outlined in Supplementary Material <a href="supplementary-material-2.html" class="quarto-xref"><span>B</span></a>.</p>
<p>The assumptions evaluated here pertain to general linear models, with additional details provided in Supplementary Material <a href="supplementary-material-10.html" class="quarto-xref"><span>J</span></a>.</p>
</section><section id="setting-the-environment" class="level2" data-number="H.2"><h2 data-number="H.2" class="anchored" data-anchor-id="setting-the-environment">
<span class="header-section-number">H.2</span> Setting the Environment</h2>
<div class="cell">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb1"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://danielvartan.github.io/brandr/">brandr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://broom.tidymodels.org/">broom</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org">dplyr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://pkg.robjhyndman.com/forecast/">forecast</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://ggplot2.tidyverse.org">ggplot2</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/GuangchuangYu/ggplotify">ggplotify</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://here.r-lib.org/">here</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://hms.tidyverse.org/">hms</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/sfirke/janitor">janitor</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://www.stefanom.io/latex2exp/">latex2exp</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va">lmtest</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://danielvartan.github.io/lubritime/">lubritime</a></span><span class="op">)</span> <span class="co"># github.com/danielvartan/lubritime</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://magrittr.tidyverse.org">magrittr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va">methods</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://olsrr.rsquaredacademy.com/">olsrr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/tidymodels/parsnip">parsnip</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://patchwork.data-imaginist.com">patchwork</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://easystats.github.io/performance/">performance</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va">pwrss</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://danielvartan.github.io/quartor/">quartor</a></span><span class="op">)</span> <span class="co"># github.com/danielvartan/quartor</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://easystats.github.io/report/">report</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://rlang.r-lib.org">rlang</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://danielvartan.github.io/rutils/">rutils</a></span><span class="op">)</span> <span class="co"># github.com/danielvartan/rutils</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://easystats.github.io/see/">see</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/tjfarrar/skedastic">skedastic</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va">stats</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://stringr.tidyverse.org">stringr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://tidyr.tidyverse.org">tidyr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/tidymodels/workflows">workflows</a></span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section><section id="loading-and-processing-the-data" class="level2" data-number="H.3"><h2 data-number="H.3" class="anchored" data-anchor-id="loading-and-processing-the-data">
<span class="header-section-number">H.3</span> Loading and Processing the Data</h2>
<div class="callout callout-style-simple callout-tip">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-body-container">
<dl>
<dt>Assumption 1</dt>
<dd>
<strong>Predictor is known</strong>. Either the vectors <span class="math inline">\(z_{1}, \dots , z_{n}\)</span> are known ahead of time, or they are the observed values of random vectors <span class="math inline">\(Z_{1}, \dots , Z_{n}\)</span> on whose values we condition before computing the joint distribution of (<span class="math inline">\(Y_{1}, \dots , Y_{n}\)</span>) <span class="citation" data-cites="degroot2012a">(<a href="references.html#ref-degroot2012a" role="doc-biblioref">DeGroot &amp; Schervish, 2012, p. 736</a>)</span>.
</dd>
</dl>
</div>
</div>
</div>
<p><strong>Assumption 1</strong> is satisfied, as the predictors are known.</p>
<div class="cell">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb2"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">targets</span><span class="fu">::</span><span class="fu"><a href="https://docs.ropensci.org/targets/reference/tar_make.html">tar_make</a></span><span class="op">(</span>script <span class="op">=</span> <span class="fu">here</span><span class="fu">::</span><span class="fu"><a href="https://here.r-lib.org//reference/here.html">here</a></span><span class="op">(</span><span class="st">"_targets.R"</span><span class="op">)</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>This data processing is performed solely for the purpose of the analysis. The variables undergo numerical transformations to streamline the modeling process and minimize potential errors.</p>
<div class="cell">
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">data</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu">targets</span><span class="fu">::</span><span class="fu"><a href="https://docs.ropensci.org/targets/reference/tar_read.html">tar_read</a></span><span class="op">(</span><span class="st">"weighted_data"</span>, store <span class="op">=</span> <span class="fu">here</span><span class="fu">::</span><span class="fu"><a href="https://here.r-lib.org//reference/here.html">here</a></span><span class="op">(</span><span class="st">"_targets"</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html">select</a></span><span class="op">(</span></span>
<span>    <span class="va">msf_sc</span>, <span class="va">age</span>, <span class="va">sex</span>, <span class="va">latitude</span>, <span class="va">longitude</span>, <span class="va">ghi_month</span>, <span class="va">cell_weight</span></span>
<span>  <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html">mutate</a></span><span class="op">(</span></span>
<span>    msf_sc <span class="op">=</span></span>
<span>      <span class="va">msf_sc</span> <span class="op">|&gt;</span></span>
<span>      <span class="fu">lubritime</span><span class="fu">:::</span><span class="fu"><a href="https://danielvartan.github.io/lubritime/reference/link_to_timeline.html">link_to_timeline</a></span><span class="op">(</span>threshold <span class="op">=</span> <span class="fu">hms</span><span class="fu">::</span><span class="fu"><a href="https://hms.tidyverse.org/reference/parse_hms.html">parse_hms</a></span><span class="op">(</span><span class="st">"12:00:00"</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>      <span class="fu"><a href="https://rdrr.io/r/base/numeric.html">as.numeric</a></span><span class="op">(</span><span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">tidyr</span><span class="fu">::</span><span class="fu"><a href="https://tidyr.tidyverse.org/reference/drop_na.html">drop_na</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section><section id="building-the-restricted-model" class="level2" data-number="H.4"><h2 data-number="H.4" class="anchored" data-anchor-id="building-the-restricted-model">
<span class="header-section-number">H.4</span> Building the Restricted Model</h2>
<div class="callout callout-style-simple callout-tip">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-body-container">
<p>The development and analysis of the restricted model are detailed in Supplementary Material <a href="supplementary-material-6.html" class="quarto-xref"><span>F</span></a>.</p>
</div>
</div>
</div>
<div class="cell">
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">form</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/formula.html">formula</a></span><span class="op">(</span><span class="va">msf_sc</span> <span class="op">~</span> <span class="va">age</span> <span class="op">+</span> <span class="va">sex</span> <span class="op">+</span> <span class="va">longitude</span> <span class="op">+</span> <span class="va">ghi_month</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">model</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu">parsnip</span><span class="fu">::</span><span class="fu"><a href="https://parsnip.tidymodels.org/reference/linear_reg.html">linear_reg</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">parsnip</span><span class="fu">::</span><span class="fu"><a href="https://parsnip.tidymodels.org/reference/set_engine.html">set_engine</a></span><span class="op">(</span><span class="st">"lm"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">parsnip</span><span class="fu">::</span><span class="fu"><a href="https://parsnip.tidymodels.org/reference/set_args.html">set_mode</a></span><span class="op">(</span><span class="st">"regression"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">workflow</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu">workflows</span><span class="fu">::</span><span class="fu"><a href="https://workflows.tidymodels.org/reference/workflow.html">workflow</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">workflows</span><span class="fu">::</span><span class="fu"><a href="https://workflows.tidymodels.org/reference/add_case_weights.html">add_case_weights</a></span><span class="op">(</span><span class="va">cell_weight</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">workflows</span><span class="fu">::</span><span class="fu"><a href="https://workflows.tidymodels.org/reference/add_formula.html">add_formula</a></span><span class="op">(</span><span class="va">form</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">workflows</span><span class="fu">::</span><span class="fu"><a href="https://workflows.tidymodels.org/reference/add_model.html">add_model</a></span><span class="op">(</span><span class="va">model</span><span class="op">)</span></span>
<span></span>
<span><span class="va">workflow</span></span>
<span><span class="co">#&gt; ══ Workflow ═════════════════════════════════════════════════════════════════</span></span>
<span><span class="co">#&gt; Preprocessor: Formula</span></span>
<span><span class="co">#&gt; Model: linear_reg()</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; ── Preprocessor ─────────────────────────────────────────────────────────────</span></span>
<span><span class="co">#&gt; msf_sc ~ age + sex + longitude + ghi_month</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; ── Case Weights ─────────────────────────────────────────────────────────────</span></span>
<span><span class="co">#&gt; cell_weight</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; ── Model ────────────────────────────────────────────────────────────────────</span></span>
<span><span class="co">#&gt; Linear Regression Model Specification (regression)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Computational engine: lm</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit</span> <span class="op">&lt;-</span> <span class="va">workflow</span> <span class="op">|&gt;</span> <span class="fu">parsnip</span><span class="fu">::</span><span class="fu"><a href="https://generics.r-lib.org/reference/fit.html">fit</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span></span>
<span><span class="va">fit_restricted</span> <span class="op">&lt;-</span> <span class="va">fit</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb8"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_coefs</span> <span class="op">&lt;-</span> <span class="va">fit</span> <span class="op">|&gt;</span> <span class="fu">broom</span><span class="fu">::</span><span class="fu"><a href="https://generics.r-lib.org/reference/tidy.html">tidy</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">fit_coefs</span> <span class="op">|&gt;</span> <span class="fu">janitor</span><span class="fu">::</span><span class="fu"><a href="https://sfirke.github.io/janitor/reference/adorn_rounding.html">adorn_rounding</a></span><span class="op">(</span><span class="fl">5</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-rm-fit-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-rm-fit-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.1: Output from the model fitting process showing the estimated coefficients, standard errors, test statistics, and <span class="math inline">\(p\)</span>-values for the terms in the linear regression model.
</figcaption><div aria-describedby="tbl-sm-8-rm-fit-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["term"],"name":[1],"type":["chr"],"align":["left"]},{"label":["estimate"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["std.error"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["statistic"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["p.value"],"name":[5],"type":["dbl"],"align":["right"]}],"data":[{"1":"(Intercept)","2":"19426.69522","3":"389.03361","4":"49.93578","5":"0"},{"1":"age","2":"-124.87574","3":"1.72710","4":"-72.30353","5":"0"},{"1":"sexMale","2":"358.47473","3":"39.07147","4":"9.17485","5":"0"},{"1":"longitude","2":"-71.88511","3":"4.86858","4":"-14.76512","5":"0"},{"1":"ghi_month","2":"-0.52011","3":"0.03995","4":"-13.01860","5":"0"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb9"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_stats</span> <span class="op">&lt;-</span> <span class="va">fit</span> <span class="op">|&gt;</span> <span class="fu">broom</span><span class="fu">::</span><span class="fu"><a href="https://generics.r-lib.org/reference/glance.html">glance</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">fit_stats</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">tidyr</span><span class="fu">::</span><span class="fu"><a href="https://tidyr.tidyverse.org/reference/pivot_longer.html">pivot_longer</a></span><span class="op">(</span>cols <span class="op">=</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://tidyselect.r-lib.org/reference/everything.html">everything</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">janitor</span><span class="fu">::</span><span class="fu"><a href="https://sfirke.github.io/janitor/reference/adorn_rounding.html">adorn_rounding</a></span><span class="op">(</span><span class="fl">10</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-rm-fit-2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-rm-fit-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.2: Summary of model fit statistics showing key metrics including R-squared, adjusted R-squared, sigma, F-statistic, <span class="math inline">\(p\)</span>-value, degrees of freedom, log-likelihood, AIC, BIC, and deviance.
</figcaption><div aria-describedby="tbl-sm-8-rm-fit-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["name"],"name":[1],"type":["chr"],"align":["left"]},{"label":["value"],"name":[2],"type":["dbl"],"align":["right"]}],"data":[{"1":"r.squared","2":"0.0851662847"},{"1":"adj.r.squared","2":"0.0851106869"},{"1":"sigma","2":"4537.4426747882"},{"1":"statistic","2":"1531.8287996472"},{"1":"p.value","2":"0.0000000000"},{"1":"df","2":"4.0000000000"},{"1":"logLik","2":"-664455.8576196495"},{"1":"AIC","2":"1328923.7152392990"},{"1":"BIC","2":"1328978.2835869009"},{"1":"deviance","2":"1355086391524.3586425781"},{"1":"df.residual","2":"65818.0000000000"},{"1":"nobs","2":"65823.0000000000"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
<div class="cell">
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">&lt;-</span> <span class="va">fit</span> <span class="op">|&gt;</span> <span class="fu">parsnip</span><span class="fu">::</span><span class="fu"><a href="https://hardhat.tidymodels.org/reference/hardhat-extract.html">extract_fit_engine</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Call:</span></span>
<span><span class="co">#&gt; stats::lm(formula = ..y ~ ., data = data, weights = weights)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Weighted Residuals:</span></span>
<span><span class="co">#&gt;        Min         1Q     Median         3Q        Max </span></span>
<span><span class="co">#&gt; -44246.473  -2705.422   -315.873   2429.960  52016.987 </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Coefficients:</span></span>
<span><span class="co">#&gt;                   Estimate     Std. Error   t value   Pr(&gt;|t|)    </span></span>
<span><span class="co">#&gt; (Intercept) 19426.69521742   389.03361181  49.93578 &lt; 2.22e-16 ***</span></span>
<span><span class="co">#&gt; age          -124.87573578     1.72710425 -72.30353 &lt; 2.22e-16 ***</span></span>
<span><span class="co">#&gt; sexMale       358.47472586    39.07147223   9.17485 &lt; 2.22e-16 ***</span></span>
<span><span class="co">#&gt; longitude     -71.88511434     4.86857616 -14.76512 &lt; 2.22e-16 ***</span></span>
<span><span class="co">#&gt; ghi_month      -0.52011291     0.03995153 -13.01860 &lt; 2.22e-16 ***</span></span>
<span><span class="co">#&gt; ---</span></span>
<span><span class="co">#&gt; Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Residual standard error: 4537.443 on 65818 degrees of freedom</span></span>
<span><span class="co">#&gt; Multiple R-squared:  0.08516628, Adjusted R-squared:  0.08511069 </span></span>
<span><span class="co">#&gt; F-statistic: 1531.829 on 4 and 65818 DF,  p-value: &lt; 2.2204e-16</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb11"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># A jerry-rigged solution to fix issues related to modeling using the pipe.</span></span>
<span></span>
<span><span class="va">fit_engine_2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/lm.html">lm</a></span><span class="op">(</span><span class="va">form</span>, data <span class="op">=</span> <span class="va">data</span>, weights <span class="op">=</span> <span class="va">cell_weight</span><span class="op">)</span></span>
<span><span class="va">fit_engine_restricted</span> <span class="op">&lt;-</span> <span class="va">fit_engine_2</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">report</span><span class="fu">::</span><span class="fu"><a href="https://easystats.github.io/report/reference/report.html">report</a></span><span class="op">(</span><span class="va">fit_engine_2</span><span class="op">)</span></span>
<span><span class="co">#&gt; We fitted a linear model (estimated using OLS) to predict msf_sc with age,</span></span>
<span><span class="co">#&gt; sex, longitude and ghi_month (formula: msf_sc ~ age + sex + longitude +</span></span>
<span><span class="co">#&gt; ghi_month). The model explains a statistically significant and weak</span></span>
<span><span class="co">#&gt; proportion of variance (R2 = 0.09, F(4, 65818) = 1531.83, p &lt; .001, adj. R2</span></span>
<span><span class="co">#&gt; = 0.09). The model's intercept, corresponding to age = 0, sex = Female,</span></span>
<span><span class="co">#&gt; longitude = 0 and ghi_month = 0, is at 19426.70 (95% CI [18664.19,</span></span>
<span><span class="co">#&gt; 20189.20], t(65818) = 49.94, p &lt; .001). Within this model:</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;   - The effect of age is statistically significant and negative (beta =</span></span>
<span><span class="co">#&gt; -124.88, 95% CI [-128.26, -121.49], t(65818) = -72.30, p &lt; .001; Std. beta =</span></span>
<span><span class="co">#&gt; -0.27, 95% CI [-0.28, -0.26])</span></span>
<span><span class="co">#&gt;   - The effect of sex [Male] is statistically significant and positive (beta =</span></span>
<span><span class="co">#&gt; 358.47, 95% CI [281.89, 435.05], t(65818) = 9.17, p &lt; .001; Std. beta =</span></span>
<span><span class="co">#&gt; 0.07, 95% CI [0.05, 0.08])</span></span>
<span><span class="co">#&gt;   - The effect of longitude is statistically significant and negative (beta =</span></span>
<span><span class="co">#&gt; -71.89, 95% CI [-81.43, -62.34], t(65818) = -14.77, p &lt; .001; Std. beta =</span></span>
<span><span class="co">#&gt; -0.07, 95% CI [-0.08, -0.06])</span></span>
<span><span class="co">#&gt;   - The effect of ghi month is statistically significant and negative (beta =</span></span>
<span><span class="co">#&gt; -0.52, 95% CI [-0.60, -0.44], t(65818) = -13.02, p &lt; .001; Std. beta =</span></span>
<span><span class="co">#&gt; -0.06, 95% CI [-0.07, -0.05])</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Standardized parameters were obtained by fitting the model on a standardized</span></span>
<span><span class="co">#&gt; version of the dataset. 95% Confidence Intervals (CIs) and p-values were</span></span>
<span><span class="co">#&gt; computed using a Wald t-distribution approximation.</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section><section id="building-the-full-model" class="level2" data-number="H.5"><h2 data-number="H.5" class="anchored" data-anchor-id="building-the-full-model">
<span class="header-section-number">H.5</span> Building the Full Model</h2>
<section id="fitting-the-model" class="level3" data-number="H.5.1"><h3 data-number="H.5.1" class="anchored" data-anchor-id="fitting-the-model">
<span class="header-section-number">H.5.1</span> Fitting the Model</h3>
<div class="cell">
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">form</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/formula.html">as.formula</a></span><span class="op">(</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span></span>
<span>      <span class="st">"msf_sc ~ "</span>,</span>
<span>      <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span></span>
<span>        <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"age"</span>, <span class="st">"sex"</span>, <span class="st">"longitude"</span>, <span class="st">"ghi_month"</span>,  <span class="st">"latitude"</span><span class="op">)</span>,</span>
<span>        collapse <span class="op">=</span> <span class="st">" + "</span></span>
<span>      <span class="op">)</span></span>
<span>    <span class="op">)</span></span>
<span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/stats/lm.html">lm</a></span><span class="op">(</span><span class="va">form</span>, data <span class="op">=</span> <span class="va">data</span>, weights <span class="op">=</span> <span class="va">cell_weight</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Call:</span></span>
<span><span class="co">#&gt; lm(formula = form, data = data, weights = cell_weight)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Coefficients:</span></span>
<span><span class="co">#&gt;   (Intercept)            age        sexMale      longitude      ghi_month  </span></span>
<span><span class="co">#&gt; 18514.7078232   -125.2679118    359.1669617    -63.3360146     -0.3451286  </span></span>
<span><span class="co">#&gt;      latitude  </span></span>
<span><span class="co">#&gt;   -21.7291415</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">model</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu">parsnip</span><span class="fu">::</span><span class="fu"><a href="https://parsnip.tidymodels.org/reference/linear_reg.html">linear_reg</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">parsnip</span><span class="fu">::</span><span class="fu"><a href="https://parsnip.tidymodels.org/reference/set_engine.html">set_engine</a></span><span class="op">(</span><span class="st">"lm"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">parsnip</span><span class="fu">::</span><span class="fu"><a href="https://parsnip.tidymodels.org/reference/set_args.html">set_mode</a></span><span class="op">(</span><span class="st">"regression"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">workflow</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu">workflows</span><span class="fu">::</span><span class="fu"><a href="https://workflows.tidymodels.org/reference/workflow.html">workflow</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">workflows</span><span class="fu">::</span><span class="fu"><a href="https://workflows.tidymodels.org/reference/add_case_weights.html">add_case_weights</a></span><span class="op">(</span><span class="va">cell_weight</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">workflows</span><span class="fu">::</span><span class="fu"><a href="https://workflows.tidymodels.org/reference/add_formula.html">add_formula</a></span><span class="op">(</span><span class="va">form</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">workflows</span><span class="fu">::</span><span class="fu"><a href="https://workflows.tidymodels.org/reference/add_model.html">add_model</a></span><span class="op">(</span><span class="va">model</span><span class="op">)</span></span>
<span></span>
<span><span class="va">workflow</span></span>
<span><span class="co">#&gt; ══ Workflow ═════════════════════════════════════════════════════════════════</span></span>
<span><span class="co">#&gt; Preprocessor: Formula</span></span>
<span><span class="co">#&gt; Model: linear_reg()</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; ── Preprocessor ─────────────────────────────────────────────────────────────</span></span>
<span><span class="co">#&gt; msf_sc ~ age + sex + longitude + ghi_month + latitude</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; ── Case Weights ─────────────────────────────────────────────────────────────</span></span>
<span><span class="co">#&gt; cell_weight</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; ── Model ────────────────────────────────────────────────────────────────────</span></span>
<span><span class="co">#&gt; Linear Regression Model Specification (regression)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Computational engine: lm</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit</span> <span class="op">&lt;-</span> <span class="va">workflow</span> <span class="op">|&gt;</span> <span class="fu">parsnip</span><span class="fu">::</span><span class="fu"><a href="https://generics.r-lib.org/reference/fit.html">fit</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span></span>
<span><span class="va">fit_full</span> <span class="op">&lt;-</span> <span class="va">fit</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb18"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_coefs</span> <span class="op">&lt;-</span> <span class="va">fit</span> <span class="op">|&gt;</span> <span class="fu">broom</span><span class="fu">::</span><span class="fu"><a href="https://generics.r-lib.org/reference/tidy.html">tidy</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">fit_coefs</span> <span class="op">|&gt;</span> <span class="fu">janitor</span><span class="fu">::</span><span class="fu"><a href="https://sfirke.github.io/janitor/reference/adorn_rounding.html">adorn_rounding</a></span><span class="op">(</span><span class="fl">5</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-fit-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-fit-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.3: Output from the model fitting process showing the estimated coefficients, standard errors, test statistics, and <span class="math inline">\(p\)</span>-values for the terms in the linear regression model.
</figcaption><div aria-describedby="tbl-sm-8-fit-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["term"],"name":[1],"type":["chr"],"align":["left"]},{"label":["estimate"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["std.error"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["statistic"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["p.value"],"name":[5],"type":["dbl"],"align":["right"]}],"data":[{"1":"(Intercept)","2":"18514.70782","3":"416.59910","4":"44.44251","5":"0"},{"1":"age","2":"-125.26791","3":"1.72782","4":"-72.50050","5":"0"},{"1":"sexMale","2":"359.16696","3":"39.06086","4":"9.19506","5":"0"},{"1":"longitude","2":"-63.33601","3":"5.06446","4":"-12.50597","5":"0"},{"1":"ghi_month","2":"-0.34513","3":"0.04915","4":"-7.02171","5":"0"},{"1":"latitude","2":"-21.72914","3":"3.55729","4":"-6.10834","5":"0"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb19"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_stats</span> <span class="op">&lt;-</span> <span class="va">fit</span> <span class="op">|&gt;</span> <span class="fu">broom</span><span class="fu">::</span><span class="fu"><a href="https://generics.r-lib.org/reference/glance.html">glance</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">fit_stats</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">tidyr</span><span class="fu">::</span><span class="fu"><a href="https://tidyr.tidyverse.org/reference/pivot_longer.html">pivot_longer</a></span><span class="op">(</span>cols <span class="op">=</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://tidyselect.r-lib.org/reference/everything.html">everything</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">janitor</span><span class="fu">::</span><span class="fu"><a href="https://sfirke.github.io/janitor/reference/adorn_rounding.html">adorn_rounding</a></span><span class="op">(</span><span class="fl">10</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-fit-2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-fit-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.4: Summary of model fit statistics showing key metrics including R-squared, adjusted R-squared, sigma, F-statistic, <span class="math inline">\(p\)</span>-value, degrees of freedom, log-likelihood, AIC, BIC, and deviance.
</figcaption><div aria-describedby="tbl-sm-8-fit-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["name"],"name":[1],"type":["chr"],"align":["left"]},{"label":["value"],"name":[2],"type":["dbl"],"align":["right"]}],"data":[{"1":"r.squared","2":"0.0856846119"},{"1":"adj.r.squared","2":"0.0856151530"},{"1":"sigma","2":"4536.1915391061"},{"1":"statistic","2":"1233.6014844333"},{"1":"p.value","2":"0.0000000000"},{"1":"df","2":"5.0000000000"},{"1":"logLik","2":"-664437.2053132770"},{"1":"AIC","2":"1328888.4106265539"},{"1":"BIC","2":"1328952.0736987561"},{"1":"deviance","2":"1354318625680.8759765625"},{"1":"df.residual","2":"65817.0000000000"},{"1":"nobs","2":"65823.0000000000"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
<div class="cell">
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">&lt;-</span> <span class="va">fit</span> <span class="op">|&gt;</span> <span class="fu">parsnip</span><span class="fu">::</span><span class="fu"><a href="https://hardhat.tidymodels.org/reference/hardhat-extract.html">extract_fit_engine</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Call:</span></span>
<span><span class="co">#&gt; stats::lm(formula = ..y ~ ., data = data, weights = weights)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Weighted Residuals:</span></span>
<span><span class="co">#&gt;        Min         1Q     Median         3Q        Max </span></span>
<span><span class="co">#&gt; -44379.655  -2728.158   -339.719   2408.172  52050.417 </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Coefficients:</span></span>
<span><span class="co">#&gt;                   Estimate     Std. Error   t value           Pr(&gt;|t|)    </span></span>
<span><span class="co">#&gt; (Intercept) 18514.70782319   416.59909595  44.44251         &lt; 2.22e-16 ***</span></span>
<span><span class="co">#&gt; age          -125.26791175     1.72782129 -72.50050         &lt; 2.22e-16 ***</span></span>
<span><span class="co">#&gt; sexMale       359.16696172    39.06086322   9.19506         &lt; 2.22e-16 ***</span></span>
<span><span class="co">#&gt; longitude     -63.33601460     5.06446295 -12.50597         &lt; 2.22e-16 ***</span></span>
<span><span class="co">#&gt; ghi_month      -0.34512864     0.04915163  -7.02171 0.0000000000022128 ***</span></span>
<span><span class="co">#&gt; latitude      -21.72914149     3.55729235  -6.10834 0.0000000010123637 ***</span></span>
<span><span class="co">#&gt; ---</span></span>
<span><span class="co">#&gt; Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Residual standard error: 4536.192 on 65817 degrees of freedom</span></span>
<span><span class="co">#&gt; Multiple R-squared:  0.08568461, Adjusted R-squared:  0.08561515 </span></span>
<span><span class="co">#&gt; F-statistic: 1233.601 on 5 and 65817 DF,  p-value: &lt; 2.2204e-16</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb21"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># A jerry-rigged solution to fix issues related to modeling using the pipe.</span></span>
<span></span>
<span><span class="va">fit_engine_2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/lm.html">lm</a></span><span class="op">(</span><span class="va">form</span>, data <span class="op">=</span> <span class="va">data</span>, weights <span class="op">=</span> <span class="va">cell_weight</span><span class="op">)</span></span>
<span><span class="va">fit_engine_full</span> <span class="op">&lt;-</span> <span class="va">fit_engine_2</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<div class="sourceCode" id="cb22"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">report</span><span class="fu">::</span><span class="fu"><a href="https://easystats.github.io/report/reference/report.html">report</a></span><span class="op">(</span><span class="va">fit_engine_2</span><span class="op">)</span></span>
<span><span class="co">#&gt; We fitted a linear model (estimated using OLS) to predict msf_sc with age,</span></span>
<span><span class="co">#&gt; sex, longitude, ghi_month and latitude (formula: msf_sc ~ age + sex +</span></span>
<span><span class="co">#&gt; longitude + ghi_month + latitude). The model explains a statistically</span></span>
<span><span class="co">#&gt; significant and weak proportion of variance (R2 = 0.09, F(5, 65817) =</span></span>
<span><span class="co">#&gt; 1233.60, p &lt; .001, adj. R2 = 0.09). The model's intercept, corresponding to</span></span>
<span><span class="co">#&gt; age = 0, sex = Female, longitude = 0, ghi_month = 0 and latitude = 0, is at</span></span>
<span><span class="co">#&gt; 18514.71 (95% CI [17698.17, 19331.24], t(65817) = 44.44, p &lt; .001). Within</span></span>
<span><span class="co">#&gt; this model:</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;   - The effect of age is statistically significant and negative (beta =</span></span>
<span><span class="co">#&gt; -125.27, 95% CI [-128.65, -121.88], t(65817) = -72.50, p &lt; .001; Std. beta =</span></span>
<span><span class="co">#&gt; -0.27, 95% CI [-0.28, -0.26])</span></span>
<span><span class="co">#&gt;   - The effect of sex [Male] is statistically significant and positive (beta =</span></span>
<span><span class="co">#&gt; 359.17, 95% CI [282.61, 435.73], t(65817) = 9.20, p &lt; .001; Std. beta =</span></span>
<span><span class="co">#&gt; 0.07, 95% CI [0.05, 0.08])</span></span>
<span><span class="co">#&gt;   - The effect of longitude is statistically significant and negative (beta =</span></span>
<span><span class="co">#&gt; -63.34, 95% CI [-73.26, -53.41], t(65817) = -12.51, p &lt; .001; Std. beta =</span></span>
<span><span class="co">#&gt; -0.06, 95% CI [-0.07, -0.05])</span></span>
<span><span class="co">#&gt;   - The effect of ghi month is statistically significant and negative (beta =</span></span>
<span><span class="co">#&gt; -0.35, 95% CI [-0.44, -0.25], t(65817) = -7.02, p &lt; .001; Std. beta = -0.04,</span></span>
<span><span class="co">#&gt; 95% CI [-0.05, -0.03])</span></span>
<span><span class="co">#&gt;   - The effect of latitude is statistically significant and negative (beta =</span></span>
<span><span class="co">#&gt; -21.73, 95% CI [-28.70, -14.76], t(65817) = -6.11, p &lt; .001; Std. beta =</span></span>
<span><span class="co">#&gt; -0.03, 95% CI [-0.05, -0.02])</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Standardized parameters were obtained by fitting the model on a standardized</span></span>
<span><span class="co">#&gt; version of the dataset. 95% Confidence Intervals (CIs) and p-values were</span></span>
<span><span class="co">#&gt; computed using a Wald t-distribution approximation.</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section><section id="evaluating-the-model-fit" class="level3" data-number="H.5.2"><h3 data-number="H.5.2" class="anchored" data-anchor-id="evaluating-the-model-fit">
<span class="header-section-number">H.5.2</span> Evaluating the Model Fit</h3>
<section id="predictions" class="level4" data-number="H.5.2.1"><h4 data-number="H.5.2.1" class="anchored" data-anchor-id="predictions">
<span class="header-section-number">H.5.2.1</span> Predictions</h4>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb23"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">limits</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/predict.html">predict</a></span><span class="op">(</span><span class="va">fit_engine</span>, interval <span class="op">=</span> <span class="st">"prediction"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/as_tibble.html">as_tibble</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">rutils</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/rutils/reference/shush.html">shush</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">fit</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">broom</span><span class="fu">::</span><span class="fu"><a href="https://generics.r-lib.org/reference/augment.html">augment</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/bind_cols.html">bind_cols</a></span><span class="op">(</span><span class="va">limits</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html">ggplot</a></span><span class="op">(</span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span><span class="va">msf_sc</span>, <span class="va">.pred</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="co"># ggplot2::geom_ribbon(</span></span>
<span>  <span class="co">#   mapping = ggplot2::aes(ymin = lwr, ymax = upr),</span></span>
<span>  <span class="co">#   alpha = 0.2</span></span>
<span>  <span class="co"># ) +</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_ribbon.html">geom_ribbon</a></span><span class="op">(</span></span>
<span>    mapping <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span></span>
<span>      ymin <span class="op">=</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/predict.html">predict</a></span><span class="op">(</span><span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/loess.html">loess</a></span><span class="op">(</span><span class="va">lwr</span> <span class="op">~</span> <span class="va">msf_sc</span><span class="op">)</span><span class="op">)</span>,</span>
<span>      ymax <span class="op">=</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/predict.html">predict</a></span><span class="op">(</span><span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/loess.html">loess</a></span><span class="op">(</span><span class="va">upr</span> <span class="op">~</span> <span class="va">msf_sc</span><span class="op">)</span><span class="op">)</span>,</span>
<span>    <span class="op">)</span>,</span>
<span>    alpha <span class="op">=</span> <span class="fl">0.2</span></span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_smooth.html">geom_smooth</a></span><span class="op">(</span></span>
<span>    mapping <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span>y <span class="op">=</span> <span class="va">lwr</span><span class="op">)</span>,</span>
<span>    se <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>    method <span class="op">=</span> <span class="st">"loess"</span>,</span>
<span>    formula <span class="op">=</span> <span class="va">y</span> <span class="op">~</span> <span class="va">x</span>,</span>
<span>    linetype <span class="op">=</span> <span class="st">"dashed"</span>,</span>
<span>    linewidth <span class="op">=</span> <span class="fl">0.2</span>,</span>
<span>    color <span class="op">=</span> <span class="st">"black"</span></span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_smooth.html">geom_smooth</a></span><span class="op">(</span></span>
<span>    mapping <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span>y <span class="op">=</span> <span class="va">upr</span><span class="op">)</span>,</span>
<span>    se <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>    method <span class="op">=</span> <span class="st">"loess"</span>,</span>
<span>    formula <span class="op">=</span> <span class="va">y</span> <span class="op">~</span> <span class="va">x</span>,</span>
<span>    linetype <span class="op">=</span> <span class="st">"dashed"</span>,</span>
<span>    linewidth <span class="op">=</span> <span class="fl">0.2</span>,</span>
<span>    color <span class="op">=</span> <span class="st">"black"</span></span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_point.html">geom_point</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_abline</a></span><span class="op">(</span></span>
<span>    intercept <span class="op">=</span> <span class="fl">0</span>,</span>
<span>    slope <span class="op">=</span> <span class="fl">1</span>,</span>
<span>    color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span></span>
<span>    x <span class="op">=</span> <span class="st">"Observed"</span>,</span>
<span>    y <span class="op">=</span> <span class="st">"Predicted"</span></span>
<span>  <span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-fit" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-fit-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.1: Relation between observed and predicted values. The orange line is a 45-degree line originating from the plane’s origin and represents a perfect fit. The shaded area depicts a smoothed version of the 95% confidence of the <a href="http://www.sthda.com/english/articles/40-regression-analysis/166-predict-in-r-model-predictions-and-confidence-intervals/">prediction interval</a>.
</figcaption><div aria-describedby="fig-sm-8-fit-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-28-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</section><section id="posterior-predictive-checks" class="level4" data-number="H.5.2.2"><h4 data-number="H.5.2.2" class="anchored" data-anchor-id="posterior-predictive-checks">
<span class="header-section-number">H.5.2.2</span> Posterior Predictive Checks</h4>
<p>Posterior predictive checks are a Bayesian technique used to assess model fit by comparing observed data to data simulated from the <a href="https://en.wikipedia.org/wiki/Posterior_predictive_distribution">posterior predictive distribution</a> (i.e., the distribution of potential unobserved values given the observed data). These checks help identify systematic discrepancies between the observed and simulated data, providing insight into whether the chosen model (or distributional family) is appropriate. Ideally, the model-predicted lines should closely match the observed data patterns.</p>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb24"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">diag_sum_plots</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">fit_engine_2</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">performance</span><span class="fu">::</span><span class="fu"><a href="https://easystats.github.io/performance/reference/check_model.html">check_model</a></span><span class="op">(</span></span>
<span>    panel <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>    colors <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span></span>
<span>      <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span>,</span>
<span>      <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"black"</span><span class="op">)</span>,</span>
<span>      <span class="st">"black"</span></span>
<span>    <span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html">plot</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">rutils</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/rutils/reference/shush.html">shush</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">diag_sum_plots</span><span class="op">$</span><span class="va">PP_CHECK</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span></span>
<span>    title <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>    subtitle <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>    x <span class="op">=</span> <span class="st">"MSFsc (Chronotype proxy) (s)"</span>,</span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/theme_get.html">theme_get</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-fit-comparison" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-fit-comparison-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.2: Posterior predictive checks for the model. The orange line represents the observed data, while the black lines represent the model-predicted data.
</figcaption><div aria-describedby="fig-sm-8-fit-comparison-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-29-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</section></section><section id="conducting-model-diagnostics" class="level3" data-number="H.5.3"><h3 data-number="H.5.3" class="anchored" data-anchor-id="conducting-model-diagnostics">
<span class="header-section-number">H.5.3</span> Conducting Model Diagnostics</h3>
<div class="callout callout-style-simple callout-warning">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-body-container">
<p>It’s important to note that objective assumption tests (e.g., Anderson–Darling test) is not advisable for larger samples, since they can be overly sensitive to minor deviations. Additionally, they might overlook visual patterns that are not captured by a single metric <span class="citation" data-cites="shatz2024 kozak2018 schucany2006">(<a href="references.html#ref-kozak2018" role="doc-biblioref">Kozak &amp; Piepho, 2018</a>; <a href="references.html#ref-schucany2006" role="doc-biblioref">Schucany &amp; Ng, 2006</a>; <a href="references.html#ref-shatz2024" role="doc-biblioref">Shatz, 2024</a>)</span>.</p>
<p>These tests are included here <strong>for reference only</strong>. However, due to the reasons mentioned above, all assumptions were primarily diagnosed through <strong>visual inspection</strong>.</p>
<p>For a straightforward critique of normality tests specifically, refer to <a href="https://towardsdatascience.com/stop-testing-for-normality-dba96bb73f90">this</a> article by <span class="citation" data-cites="greener2020">Greener (<a href="references.html#ref-greener2020" role="doc-biblioref">2020</a>)</span>.</p>
</div>
</div>
</div>
<section id="normality" class="level4" data-number="H.5.3.1"><h4 data-number="H.5.3.1" class="anchored" data-anchor-id="normality">
<span class="header-section-number">H.5.3.1</span> Normality</h4>
<div class="callout callout-style-simple callout-tip">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-body-container">
<dl>
<dt>Assumption 2</dt>
<dd>
<strong>Normality</strong>. For <span class="math inline">\(i = 1, \dots, n\)</span>, the conditional distribution of <span class="math inline">\(Y_{i}\)</span> given the vectors <span class="math inline">\(z_{1}, \dots , z_{n}\)</span> is a normal distribution <span class="citation" data-cites="degroot2012a">(<a href="references.html#ref-degroot2012a" role="doc-biblioref">DeGroot &amp; Schervish, 2012, p. 737</a>)</span>.
</dd>
</dl>
<p>(Normality of the error term distribution <span class="citation" data-cites="hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019, p. 287</a>)</span>)</p>
</div>
</div>
</div>
<p><strong>Assumption 2</strong> is satisfied, as the residuals shown a fairly normal distribution by visual inspection.</p>
<section id="inspection" class="level5" data-number="H.5.3.1.1"><h5 data-number="H.5.3.1.1" class="anchored" data-anchor-id="inspection">
<span class="header-section-number">H.5.3.1.1</span> Inspection</h5>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb25"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/residuals.html">residuals</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/as_tibble.html">as_tibble</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">rutils</span><span class="fu">:::</span><span class="fu">stats_summary</span><span class="op">(</span>col <span class="op">=</span> <span class="st">"value"</span>, name <span class="op">=</span> <span class="st">"Residuals"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-residual-diag-normality-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-residual-diag-normality-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.5: Summary statistics of model residuals.
</figcaption><div aria-describedby="tbl-sm-8-residual-diag-normality-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["name"],"name":[1],"type":["chr"],"align":["left"]},{"label":["value"],"name":[2],"type":["chr"],"align":["left"]}],"data":[{"1":"name","2":"Residuals"},{"1":"n","2":"65823"},{"1":"n_rm_na","2":"65823"},{"1":"n_na","2":"0"},{"1":"mean","2":"-85.7718042074199"},{"1":"var","2":"25144713.5279995"},{"1":"sd","2":"5014.4504711882"},{"1":"min","2":"-16207.8476931081"},{"1":"q_1","2":"-3636.95737875514"},{"1":"median","2":"-469.493356793294"},{"1":"q_3","2":"3208.56695553457"},{"1":"max","2":"18069.694305669"},{"1":"iqr","2":"6845.5243342897"},{"1":"range","2":"34277.5419987771"},{"1":"skewness","2":"0.259860107105403"},{"1":"kurtosis","2":"2.82015304828676"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb26"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/residuals.html">residuals</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/as_tibble.html">as_tibble</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">rutils</span><span class="fu">:::</span><span class="fu">test_normality</span><span class="op">(</span>col <span class="op">=</span> <span class="st">"value"</span>, name <span class="op">=</span> <span class="st">"Residuals"</span><span class="op">)</span></span>
<span><span class="co">#&gt; Registered S3 method overwritten by 'quantmod':</span></span>
<span><span class="co">#&gt;   method            from</span></span>
<span><span class="co">#&gt;   as.zoo.data.frame zoo</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-residual-diag-normality-2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-residual-diag-normality-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.3: Histogram of the model residuals with a kernel density estimate, along with a quantile-quantile (Q-Q) plot between the residuals and the theoretical quantiles of the normal distribution.
</figcaption><div aria-describedby="fig-sm-8-residual-diag-normality-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-31-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb27"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">broom</span><span class="fu">::</span><span class="fu"><a href="https://generics.r-lib.org/reference/augment.html">augment</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html">select</a></span><span class="op">(</span><span class="va">.resid</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">tidyr</span><span class="fu">::</span><span class="fu"><a href="https://tidyr.tidyverse.org/reference/pivot_longer.html">pivot_longer</a></span><span class="op">(</span><span class="va">.resid</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html">ggplot</a></span><span class="op">(</span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">name</span>, y <span class="op">=</span> <span class="va">value</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_boxplot.html">geom_boxplot</a></span><span class="op">(</span></span>
<span>    outlier.colour <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span>,</span>
<span>    outlier.shape <span class="op">=</span> <span class="fl">1</span>,</span>
<span>    width <span class="op">=</span> <span class="fl">0.5</span></span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"Variable"</span>, y <span class="op">=</span> <span class="st">"Value"</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/coord_flip.html">coord_flip</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/theme.html">theme</a></span><span class="op">(</span></span>
<span>    axis.title.y <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>    axis.text.y <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>    axis.ticks.y <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span></span>
<span>  <span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-residual-diag-normality-3" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-residual-diag-normality-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.4: Boxplot of model residuals with outliers highlighted in orange.
</figcaption><div aria-describedby="fig-sm-8-residual-diag-normality-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-32-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</section><section id="tests" class="level5" data-number="H.5.3.1.2"><h5 data-number="H.5.3.1.2" class="anchored" data-anchor-id="tests">
<span class="header-section-number">H.5.3.1.2</span> Tests</h5>
<p>It’s important to note that the Kolmogorov-Smirnov and Pearson chi-square tests are included here just for reference, as many authors don’t recommend using them when testing for normality <span class="citation" data-cites="dagostino1990">(<a href="references.html#ref-dagostino1990" role="doc-biblioref">D’Agostino &amp; Belanger, 1990</a>)</span>.</p>
<p>To learn more about normality tests, refer to <span class="citation" data-cites="thode2002">Thode (<a href="references.html#ref-thode2002" role="doc-biblioref">2002</a>)</span> and the original papers for each test to understand their assumptions and limitations:</p>
<ul>
<li>
<a href="https://en.wikipedia.org/wiki/Anderson%E2%80%93Darling_test">Anderson-Darling test</a>: <span class="citation" data-cites="anderson1952">Anderson &amp; Darling (<a href="references.html#ref-anderson1952" role="doc-biblioref">1952</a>)</span>; <span class="citation" data-cites="anderson1954">Anderson &amp; Darling (<a href="references.html#ref-anderson1954" role="doc-biblioref">1954</a>)</span>.</li>
<li>Bonett-Seier test: <span class="citation" data-cites="bonett2002">Bonett &amp; Seier (<a href="references.html#ref-bonett2002" role="doc-biblioref">2002</a>)</span>.</li>
<li>
<a href="https://en.wikipedia.org/wiki/Cram%C3%A9r%E2%80%93von_Mises_criterion">Cramér-von Mises test</a>: <span class="citation" data-cites="cramer1928">Cramér (<a href="references.html#ref-cramer1928" role="doc-biblioref">1928</a>)</span>; <span class="citation" data-cites="anderson1962">Anderson (<a href="references.html#ref-anderson1962" role="doc-biblioref">1962</a>)</span>.</li>
<li>
<a href="https://en.wikipedia.org/wiki/D%27Agostino%27s_K-squared_test">D’Agostino test</a>: <span class="citation" data-cites="dagostino1971">D’Agostino (<a href="references.html#ref-dagostino1971" role="doc-biblioref">1971</a>)</span>; <span class="citation" data-cites="dagostino1973">D’Agostino &amp; Pearson (<a href="references.html#ref-dagostino1973" role="doc-biblioref">1973</a>)</span>.</li>
<li>
<a href="https://en.wikipedia.org/wiki/Jarque%E2%80%93Bera_test">Jarque–Bera test</a>: <span class="citation" data-cites="jarque1980">Jarque &amp; Bera (<a href="references.html#ref-jarque1980" role="doc-biblioref">1980</a>)</span>; <span class="citation" data-cites="bera1981">Bera &amp; Jarque (<a href="references.html#ref-bera1981" role="doc-biblioref">1981</a>)</span>; <span class="citation" data-cites="jarque1987">Jarque &amp; Bera (<a href="references.html#ref-jarque1987" role="doc-biblioref">1987</a>)</span>.</li>
<li>
<a href="https://en.wikipedia.org/wiki/Lilliefors_test">Lilliefors (K-S) test</a>: <span class="citation" data-cites="smirnov1948">Smirnov (<a href="references.html#ref-smirnov1948" role="doc-biblioref">1948</a>)</span>; <span class="citation" data-cites="kolmogorov1933">Kolmogorov (<a href="references.html#ref-kolmogorov1933" role="doc-biblioref">1933</a>)</span>; <span class="citation" data-cites="massey1951">Massey (<a href="references.html#ref-massey1951" role="doc-biblioref">1951</a>)</span>; <span class="citation" data-cites="lilliefors1967">Lilliefors (<a href="references.html#ref-lilliefors1967" role="doc-biblioref">1967</a>)</span>; <span class="citation" data-cites="dallal1986">Dallal &amp; Wilkinson (<a href="references.html#ref-dallal1986" role="doc-biblioref">1986</a>)</span>.</li>
<li>
<a href="https://en.wikipedia.org/wiki/Pearson%27s_chi-squared_test">Pearson chi-square test</a>: <span class="citation" data-cites="pearson1900">Pearson (<a href="references.html#ref-pearson1900" role="doc-biblioref">1900</a>)</span>.</li>
<li>
<a href="https://en.wikipedia.org/wiki/Shapiro%E2%80%93Francia_test">Shapiro-Francia test</a>: <span class="citation" data-cites="shapiro1972">Shapiro &amp; Francia (<a href="references.html#ref-shapiro1972" role="doc-biblioref">1972</a>)</span>.</li>
<li>
<a href="https://en.wikipedia.org/wiki/Shapiro%E2%80%93Wilk_test">Shapiro-Wilk test</a>: <span class="citation" data-cites="shapiro1965">Shapiro &amp; Wilk (<a href="references.html#ref-shapiro1965" role="doc-biblioref">1965</a>)</span>.</li>
</ul>
<p><span class="math display">\[
\begin{cases}
\text{H}_{0}: \text{The data is normally distributed} \\
\text{H}_{a}: \text{The data is not normally distributed}
\end{cases}
\]</span></p>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb28"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/residuals.html">residuals</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/as_tibble.html">as_tibble</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">rutils</span><span class="fu">:::</span><span class="fu">normality_summary</span><span class="op">(</span>col <span class="op">=</span> <span class="st">"value"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-residual-diag-normality-2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-residual-diag-normality-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.6: Summary of statistical tests conducted to assess the normality of the residuals.
</figcaption><div aria-describedby="tbl-sm-8-residual-diag-normality-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["test"],"name":[1],"type":["chr"],"align":["left"]},{"label":["statistic_1"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["statistic_2"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["p_value"],"name":[4],"type":["dbl"],"align":["right"]}],"data":[{"1":"Anderson-Darling","2":"126.65506574730","3":"NA","4":"3.700000000e-24"},{"1":"Bonett-Seier","2":"4033.74914716527","3":"-7.815075156","4":"5.551115123e-15"},{"1":"Cramer-von Mises","2":"21.37668357279","3":"NA","4":"7.370000000e-10"},{"1":"D'Agostino Omnibus Test","2":"NA","3":"NA","4":"NA"},{"1":"D'Agostino Skewness Test","2":"26.78125832236","3":"NA","4":"0.000000000e+00"},{"1":"D'Agostino Kurtosis Test","2":"NA","3":"NA","4":"NA"},{"1":"Jarque-Bera","2":"829.51797607478","3":"2.000000000","4":"0.000000000e+00"},{"1":"Lilliefors (K-S)","2":"0.03103016778","3":"NA","4":"3.958470809e-165"},{"1":"Pearson chi-square","2":"1645.50690488127","3":"NA","4":"4.002943788e-241"},{"1":"Shapiro-Francia","2":"NA","3":"NA","4":"NA"},{"1":"Shapiro-Wilk","2":"NA","3":"NA","4":"NA"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
</section></section><section id="linearity" class="level4" data-number="H.5.3.2"><h4 data-number="H.5.3.2" class="anchored" data-anchor-id="linearity">
<span class="header-section-number">H.5.3.2</span> Linearity</h4>
<div class="callout callout-style-simple callout-tip">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-body-container">
<dl>
<dt>Assumption 3</dt>
<dd>
<strong>Linear mean</strong>. There is a vector of parameters <span class="math inline">\(\beta = (\beta_{0}, \dots, \beta_{p - 1})\)</span> such that the conditional mean of <span class="math inline">\(Y_{i}\)</span> given the values <span class="math inline">\(z_{1}, \dots , z_{n}\)</span> has the form
</dd>
</dl>
<p><span class="math display">\[
z_{i0} \beta_{0} + z_{i1} \beta_{1} + \cdots + z_{ip - 1} \beta_{p - 1}
\]</span></p>
<p>for <span class="math inline">\(i = 1, \dots, n\)</span> <span class="citation" data-cites="degroot2012a">(<a href="references.html#ref-degroot2012a" role="doc-biblioref">DeGroot &amp; Schervish, 2012, p. 737</a>)</span>.</p>
<p>(Linearity of the phenomenon measured <span class="citation" data-cites="hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019, p. 287</a>)</span>)</p>
</div>
</div>
</div>
<p><strong>Assumption 3</strong> is satisfied, as the relationship between the variables is fairly linear. As shown in <a href="chapter-4.html" class="quarto-xref"><span>Chapter 4</span></a>, the hypothesis implies linearity and the distribution of the residuals also support this.</p>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb29"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plot</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">fit</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">broom</span><span class="fu">::</span><span class="fu"><a href="https://generics.r-lib.org/reference/augment.html">augment</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html">ggplot</a></span><span class="op">(</span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span><span class="va">.pred</span>, <span class="va">.resid</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_point.html">geom_point</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span></span>
<span>    yintercept <span class="op">=</span> <span class="fl">0</span>,</span>
<span>    color <span class="op">=</span> <span class="st">"black"</span>,</span>
<span>    linewidth <span class="op">=</span> <span class="fl">0.5</span>,</span>
<span>    linetype <span class="op">=</span> <span class="st">"dashed"</span></span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_smooth.html">geom_smooth</a></span><span class="op">(</span>color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"Fitted values"</span>, y <span class="op">=</span> <span class="st">"Residuals"</span><span class="op">)</span></span>
<span></span>
<span><span class="va">plot</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/base/print.html">print</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu">rutils</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/rutils/reference/shush.html">shush</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-residual-diag-fitted-values-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-residual-diag-fitted-values-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.5: Residual plot showing the relationship between fitted values and residuals.<code>&lt;br&gt;</code>{html}The dashed black line represent zero residuals, indicating an ideal model fit. The orange line indicate the conditional mean of residuals.
</figcaption><div aria-describedby="fig-sm-8-residual-diag-fitted-values-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-34-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb30"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plots</span> <span class="op">&lt;-</span> <span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">olsrr</span><span class="fu">::</span><span class="fu"><a href="https://olsrr.rsquaredacademy.com/reference/ols_plot_resid_fit_spread.html">ols_plot_resid_fit_spread</a></span><span class="op">(</span>print_plot <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span></span>
<span><span class="kw">for</span> <span class="op">(</span><span class="va">i</span> <span class="kw">in</span> <span class="fu"><a href="https://rdrr.io/r/base/seq.html">seq_along</a></span><span class="op">(</span><span class="va">plots</span><span class="op">)</span><span class="op">)</span> <span class="op">{</span></span>
<span>  <span class="va">q</span> <span class="op">&lt;-</span> <span class="va">plots</span><span class="op">[[</span><span class="va">i</span><span class="op">]</span><span class="op">]</span> <span class="op">+</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>title <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span>  <span class="va">q</span> <span class="op">&lt;-</span> <span class="va">q</span> <span class="op">|&gt;</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot_build.html">ggplot_build</a></span><span class="op">(</span><span class="op">)</span></span>
<span>  <span class="va">q</span><span class="op">$</span><span class="va">data</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span><span class="op">$</span><span class="va">colour</span> <span class="op">&lt;-</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span></span>
<span>  <span class="va">q</span><span class="op">$</span><span class="va">plot</span><span class="op">$</span><span class="va">layers</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span><span class="op">$</span><span class="va">constructor</span><span class="op">$</span><span class="va">color</span> <span class="op">&lt;-</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span></span>
<span></span>
<span>  <span class="va">plots</span><span class="op">[[</span><span class="va">i</span><span class="op">]</span><span class="op">]</span> <span class="op">&lt;-</span> <span class="va">q</span> <span class="op">|&gt;</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot_gtable.html">ggplot_gtable</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu">ggplotify</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/ggplotify/man/as.ggplot.html">as.ggplot</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="op">}</span></span>
<span></span>
<span><span class="fu">patchwork</span><span class="fu">::</span><span class="fu"><a href="https://patchwork.data-imaginist.com/reference/wrap_plots.html">wrap_plots</a></span><span class="op">(</span><span class="va">plots</span><span class="op">$</span><span class="va">fm_plot</span>, <span class="va">plots</span><span class="op">$</span><span class="va">rsd_plot</span>, ncol <span class="op">=</span> <span class="fl">2</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-residual-diag-linearity-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-residual-diag-linearity-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.6: Residual fit spread plots to detect non-linearity, influential observations, and outliers.<code>&lt;br&gt;</code>{html}The side-by-side plots show the centered fit and residuals, illustrating the variation explained by the model and what remains in the residuals. Inappropriately specified models often exhibit greater spread in the residuals than in the centered fit. “Proportion Less” indicates the cumulative distribution function, representing the proportion of observations below a specific value, facilitating an assessment of model performance.
</figcaption><div aria-describedby="fig-sm-8-residual-diag-linearity-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-35-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
<p>The <a href="https://en.wikipedia.org/wiki/Ramsey_RESET_test">Ramsey’s RESET test</a> checks if the model has no omitted variables. The test examines whether non-linear combinations of the fitted values can explain the response variable.</p>
<p>Learn more about the Ramsey’s RESET test in <span class="citation" data-cites="ramsey1969">Ramsey (<a href="references.html#ref-ramsey1969" role="doc-biblioref">1969</a>)</span>.</p>
<p><span class="math display">\[
\begin{cases}
\text{H}_{0}: \text{The model has no omitted variables} \\
\text{H}_{a}: \text{The model has omitted variables}
\end{cases}
\]</span></p>
<div class="cell">
<div class="sourceCode" id="cb31"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">lmtest</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/lmtest/man/resettest.html">resettest</a></span><span class="op">(</span>power <span class="op">=</span> <span class="fl">2</span><span class="op">:</span><span class="fl">3</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;  RESET test</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; data:  fit_engine</span></span>
<span><span class="co">#&gt; RESET = 157.25133, df1 = 2, df2 = 65815, p-value &lt; 2.2204e-16</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">lmtest</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/lmtest/man/resettest.html">resettest</a></span><span class="op">(</span>type <span class="op">=</span> <span class="st">"regressor"</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;  RESET test</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; data:  fit_engine</span></span>
<span><span class="co">#&gt; RESET = 42.46296, df1 = 12, df2 = 65805, p-value &lt; 2.2204e-16</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section><section id="homoscedasticity-common-variance" class="level4" data-number="H.5.3.3"><h4 data-number="H.5.3.3" class="anchored" data-anchor-id="homoscedasticity-common-variance">
<span class="header-section-number">H.5.3.3</span> Homoscedasticity (Common Variance)</h4>
<div class="callout callout-style-simple callout-tip">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-body-container">
<dl>
<dt>Assumption 4</dt>
<dd>
<strong>Common variance</strong> (homoscedasticity). There is as parameter <span class="math inline">\(\sigma^{2}\)</span> such the conditional variance of <span class="math inline">\(Y_{i}\)</span> given the values <span class="math inline">\(z_{1}, \dots , z_{n}\)</span> is <span class="math inline">\(\sigma^{2}\)</span> for <span class="math inline">\(i = 1, \dots, n\)</span>.
</dd>
</dl>
<p>(Constant variance of the error terms <span class="citation" data-cites="hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019, p. 287</a>)</span>)</p>
</div>
</div>
</div>
<p><strong>Assumption 4</strong> is satisfied. When comparing the standardized residuals (<span class="math inline">\(\sqrt{|\text{Standardized Residuals}|}\)</span>) spread to the fitted values, we can observe that the residuals are fairly constant across the range of values. This suggests that the residuals have a constant variance.</p>
<section id="visual-inspection" class="level5" data-number="H.5.3.3.1"><h5 data-number="H.5.3.3.1" class="anchored" data-anchor-id="visual-inspection">
<span class="header-section-number">H.5.3.3.1</span> Visual Inspection</h5>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb33"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plot</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">fit</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/predict.html">predict</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html">mutate</a></span><span class="op">(</span></span>
<span>    .sd_resid <span class="op">=</span></span>
<span>      <span class="va">fit_engine</span> <span class="op">|&gt;</span></span>
<span>      <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">rstandard</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>      <span class="fu"><a href="https://rdrr.io/r/base/MathFun.html">abs</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>      <span class="fu"><a href="https://rdrr.io/r/base/MathFun.html">sqrt</a></span><span class="op">(</span><span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html">ggplot</a></span><span class="op">(</span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span><span class="va">.pred</span>, <span class="va">.sd_resid</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_point.html">geom_point</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_smooth.html">geom_smooth</a></span><span class="op">(</span>color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span></span>
<span>    x <span class="op">=</span> <span class="st">"Fitted values"</span>,</span>
<span>    y <span class="op">=</span> <span class="fu">latex2exp</span><span class="fu">::</span><span class="fu"><a href="https://www.stefanom.io/latex2exp/reference/TeX.html">TeX</a></span><span class="op">(</span><span class="st">"$\\sqrt{|Standardized \\ Residuals|}$"</span><span class="op">)</span></span>
<span>  <span class="op">)</span></span>
<span></span>
<span><span class="va">plot</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/base/print.html">print</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu">rutils</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/rutils/reference/shush.html">shush</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-homoscedasticity-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-homoscedasticity-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.7: Relation between the fitted values of the model and its standardized residuals.
</figcaption><div aria-describedby="fig-sm-8-diag-homoscedasticity-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-39-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist">
<li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true" aria-current="page">MSF<sub>sc</sub> (Chronotype proxy) (Seconds)</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false">Age (Years)</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false">Latitude (Decimal degrees)</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-4" role="tab" aria-controls="tabset-1-4" aria-selected="false">Longitude (Decimal degrees)</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-5-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-5" role="tab" aria-controls="tabset-1-5" aria-selected="false">Monthly average global horizontal irradiance (Wh/m²)</a></li>
</ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb34"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">plotr</span><span class="fu">:::</span><span class="fu">plot_homoscedasticity</span><span class="op">(</span></span>
<span>    data <span class="op">=</span> <span class="va">data</span>,</span>
<span>    col <span class="op">=</span> <span class="st">"msf_sc"</span>,</span>
<span>    x_label <span class="op">=</span> <span class="st">"MSF~sc~ (Chronotype proxy) (Seconds)"</span></span>
<span>  <span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-6-fm-var-homoscedasticity-msf-sc" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-6-fm-var-homoscedasticity-msf-sc-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.7: Relation between <code>msf_sc</code> and the model standardized residuals.
</figcaption><div aria-describedby="tbl-sm-6-fm-var-homoscedasticity-msf-sc-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-41-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb35"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">plotr</span><span class="fu">:::</span><span class="fu">plot_homoscedasticity</span><span class="op">(</span></span>
<span>    data <span class="op">=</span> <span class="va">data</span>,</span>
<span>    col <span class="op">=</span> <span class="st">"age"</span>,</span>
<span>    x_label <span class="op">=</span> <span class="st">"Age (Years)"</span></span>
<span>  <span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-6-fm-var-homoscedasticity-age" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-6-fm-var-homoscedasticity-age-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.8: Relation between <code>age</code> and the model standardized residuals.
</figcaption><div aria-describedby="tbl-sm-6-fm-var-homoscedasticity-age-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-42-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb36"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">plotr</span><span class="fu">:::</span><span class="fu">plot_homoscedasticity</span><span class="op">(</span></span>
<span>    data <span class="op">=</span> <span class="va">data</span>,</span>
<span>    col <span class="op">=</span> <span class="st">"longitude"</span>,</span>
<span>    x_label <span class="op">=</span> <span class="st">"Latitude (Decimal degrees)"</span></span>
<span>  <span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-6-fm-var-homoscedasticity-longitude" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-6-fm-var-homoscedasticity-longitude-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.9: Relation between <code>longitude</code> and the model standardized residuals.
</figcaption><div aria-describedby="tbl-sm-6-fm-var-homoscedasticity-longitude-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-43-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
<div id="tabset-1-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-4-tab">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb37"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">plotr</span><span class="fu">:::</span><span class="fu">plot_homoscedasticity</span><span class="op">(</span></span>
<span>    data <span class="op">=</span> <span class="va">data</span>,</span>
<span>    col <span class="op">=</span> <span class="st">"ghi_month"</span>,</span>
<span>    x_label <span class="op">=</span> <span class="st">"Longitude (Decimal degrees)"</span></span>
<span>  <span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-6-fm-var-homoscedasticity-ghi-month" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-6-fm-var-homoscedasticity-ghi-month-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.10: Relation between <code>ghi_month</code> and the model standardized residuals.
</figcaption><div aria-describedby="tbl-sm-6-fm-var-homoscedasticity-ghi-month-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-44-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
<div id="tabset-1-5" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-5-tab">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb38"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">plotr</span><span class="fu">:::</span><span class="fu">plot_homoscedasticity</span><span class="op">(</span></span>
<span>    data <span class="op">=</span> <span class="va">data</span>,</span>
<span>    col <span class="op">=</span> <span class="st">"latitude"</span>,</span>
<span>    x_label <span class="op">=</span> <span class="st">"Monthly average global horizontal irradiance (Wh/m²)"</span></span>
<span>  <span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-6-fm-var-homoscedasticity-latitude" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-6-fm-var-homoscedasticity-latitude-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.11: Relation between <code>latitude</code> and the model standardized residuals.
</figcaption><div aria-describedby="tbl-sm-6-fm-var-homoscedasticity-latitude-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-45-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
</div>
</div>
</section><section id="breusch-pagan-test" class="level5" data-number="H.5.3.3.2"><h5 data-number="H.5.3.3.2" class="anchored" data-anchor-id="breusch-pagan-test">
<span class="header-section-number">H.5.3.3.2</span> Breusch-Pagan Test</h5>
<p>The <a href="https://en.wikipedia.org/wiki/Breusch%E2%80%93Pagan_test">Breusch-Pagan test</a> checks whether the residuals exhibit constant variance.</p>
<p>Learn more about the Breusch-Pagan test in <span class="citation" data-cites="breusch1979">Breusch &amp; Pagan (<a href="references.html#ref-breusch1979" role="doc-biblioref">1979</a>)</span> and <span class="citation" data-cites="koenker1981">Koenker (<a href="references.html#ref-koenker1981" role="doc-biblioref">1981</a>)</span>.</p>
<p><span class="math display">\[
\begin{cases}
\text{H}_{0}: \text{The variance is constant} \\
\text{H}_{a}: \text{The variance is not constant}
\end{cases}
\]</span></p>
<div class="cell">
<div class="sourceCode" id="cb39"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># With studentising modification of Koenker</span></span>
<span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">lmtest</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/lmtest/man/bptest.html">bptest</a></span><span class="op">(</span>studentize <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;  studentized Breusch-Pagan test</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; data:  fit_engine</span></span>
<span><span class="co">#&gt; BP = 1131.6884, df = 5, p-value &lt; 2.2204e-16</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb40"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">lmtest</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/lmtest/man/bptest.html">bptest</a></span><span class="op">(</span>studentize <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;  Breusch-Pagan test</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; data:  fit_engine</span></span>
<span><span class="co">#&gt; BP = 1552.1088, df = 5, p-value &lt; 2.2204e-16</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb41"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="co"># Using the studentized modification of Koenker.</span></span>
<span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">skedastic</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/skedastic/man/breusch_pagan.html">breusch_pagan</a></span><span class="op">(</span>koenker <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["statistic"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["p.value"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["parameter"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["method"],"name":[4],"type":["chr"],"align":["left"]},{"label":["alternative"],"name":[5],"type":["chr"],"align":["left"]}],"data":[{"1":"193.3620477","2":"7.465535148e-40","3":"5","4":"Koenker (studentised)","5":"greater"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
<div class="cell">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb42"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">skedastic</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/skedastic/man/breusch_pagan.html">breusch_pagan</a></span><span class="op">(</span>koenker <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["statistic"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["p.value"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["parameter"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["method"],"name":[4],"type":["chr"],"align":["left"]},{"label":["alternative"],"name":[5],"type":["chr"],"align":["left"]}],"data":[{"1":"174.2664612","2":"8.966294332e-36","3":"5","4":"Breusch-Pagan (non-studentised)","5":"greater"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
<div class="cell">
<div class="sourceCode" id="cb43"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">car</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/car/man/ncvTest.html">ncvTest</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; Non-constant Variance Score Test </span></span>
<span><span class="co">#&gt; Variance formula: ~ fitted.values </span></span>
<span><span class="co">#&gt; Chisquare = 9083.487491, Df = 1, p = &lt; 2.22045e-16</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb44"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine_2</span> <span class="op">|&gt;</span> <span class="fu">olsrr</span><span class="fu">::</span><span class="fu"><a href="https://olsrr.rsquaredacademy.com/reference/ols_test_breusch_pagan.html">ols_test_breusch_pagan</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;  Breusch Pagan Test for Heteroskedasticity</span></span>
<span><span class="co">#&gt;  -----------------------------------------</span></span>
<span><span class="co">#&gt;  Ho: the variance is constant            </span></span>
<span><span class="co">#&gt;  Ha: the variance is not constant        </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;                Data                </span></span>
<span><span class="co">#&gt;  ----------------------------------</span></span>
<span><span class="co">#&gt;  Response : msf_sc </span></span>
<span><span class="co">#&gt;  Variables: fitted values of msf_sc </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;            Test Summary            </span></span>
<span><span class="co">#&gt;  ----------------------------------</span></span>
<span><span class="co">#&gt;  DF            =    1 </span></span>
<span><span class="co">#&gt;  Chi2          =    72.18484543 </span></span>
<span><span class="co">#&gt;  Prob &gt; Chi2   =    1.959548349e-17</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section></section><section id="independence" class="level4" data-number="H.5.3.4"><h4 data-number="H.5.3.4" class="anchored" data-anchor-id="independence">
<span class="header-section-number">H.5.3.4</span> Independence</h4>
<div class="callout callout-style-simple callout-tip">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-body-container">
<dl>
<dt>Assumption 5</dt>
<dd>
<strong>Independence</strong>. The random variables <span class="math inline">\(Y_{1}, \dots , Y_{n}\)</span> are independent given the observed <span class="math inline">\(z_{1}, \dots , z_{n}\)</span> <span class="citation" data-cites="degroot2012a">(<a href="references.html#ref-degroot2012a" role="doc-biblioref">DeGroot &amp; Schervish, 2012, p. 737</a>)</span>.
</dd>
</dl>
<p>(Independence of the error terms <span class="citation" data-cites="hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019, p. 287</a>)</span>)</p>
</div>
</div>
</div>
<p><strong>Assumption 5</strong> is satisfied. Although the residuals show some autocorrelation, they fall within the acceptable range of the Durbin–Watson statistic (<span class="math inline">\(1.5\)</span> to <span class="math inline">\(2.5\)</span>). It’s also important to note that the observations for each predicted value are not related to any other prediction; in other words, they are not grouped or sequenced by any variable (by design) (see <span class="citation" data-cites="hair2019">Hair (<a href="references.html#ref-hair2019" role="doc-biblioref">2019, p. 291</a>)</span> for more information).</p>
<p>Many authors don’t consider autocorrelation tests for linear regression models, as they are more relevant for time series data. However, I include them here just for reference.</p>
<section id="visual-inspection-1" class="level5" data-number="H.5.3.4.1"><h5 data-number="H.5.3.4.1" class="anchored" data-anchor-id="visual-inspection-1">
<span class="header-section-number">H.5.3.4.1</span> Visual Inspection</h5>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb45"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/r/stats/residuals.html">residuals</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">forecast</span><span class="fu">::</span><span class="fu"><a href="https://pkg.robjhyndman.com/forecast/reference/tsdisplay.html">ggtsdisplay</a></span><span class="op">(</span></span>
<span>    lag.max <span class="op">=</span> <span class="fl">30</span>,</span>
<span>    theme <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/theme_get.html">theme_get</a></span><span class="op">(</span><span class="op">)</span></span>
<span>  <span class="op">)</span></span>
<span><span class="co">#&gt; Registered S3 methods overwritten by 'forecast':</span></span>
<span><span class="co">#&gt;   method  from </span></span>
<span><span class="co">#&gt;   head.ts stats</span></span>
<span><span class="co">#&gt;   tail.ts stats</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-independence-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-independence-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.8: Time series plot of the residuals along with its AutoCorrelation Function (ACF) and Partial AutoCorrelation Function (PACF).
</figcaption><div aria-describedby="fig-sm-8-diag-independence-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-52-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</section><section id="correlations" class="level5" data-number="H.5.3.4.2"><h5 data-number="H.5.3.4.2" class="anchored" data-anchor-id="correlations">
<span class="header-section-number">H.5.3.4.2</span> Correlations</h5>
<p><a href="#tbl-sm-8-residual-diag-independence-1" class="quarto-xref">Table&nbsp;<span>H.12</span></a> shows the relative importance of independent variables in determining the response variable. It highlights how much each variable uniquely contributes to the R-squared value, beyond what is explained by the other predictors.</p>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb46"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">olsrr</span><span class="fu">::</span><span class="fu"><a href="https://olsrr.rsquaredacademy.com/reference/ols_correlations.html">ols_correlations</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-residual-diag-independence-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-residual-diag-independence-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.12: Correlations between the dependent variable and the independent variables, along with the zero-order, part, and partial correlations.<code>&lt;br&gt;</code>{html}The zero-order correlation represents the Pearson correlation coefficient between the dependent and independent variables. Part correlations indicate how much the R-squared would decrease if a specific variable were removed from the model, while partial correlations reflect the portion of variance in the response variable that is explained by a specific independent variable, beyond the influence of other predictors in the model.
</figcaption><div aria-describedby="tbl-sm-8-residual-diag-independence-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["Zero-order"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["Partial"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["Part"],"name":[3],"type":["dbl"],"align":["right"]}],"data":[{"1":"-0.22584273344","2":"-0.2815936055","3":"-0.2806147806","_rn_":"age"},{"1":"0.03938797353","2":"0.1703121684","3":"0.1652667445","_rn_":"sexMale"},{"1":"-0.06810852464","2":"-0.1662400474","3":"-0.1612015368","_rn_":"longitude"},{"1":"-0.04936323563","2":"-0.1640753512","3":"-0.1590439835","_rn_":"ghi_month"},{"1":"-0.06008254515","2":"-0.1645150794","3":"-0.1594820684","_rn_":"latitude"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
</section><section id="newey-west-estimator" class="level5" data-number="H.5.3.4.3"><h5 data-number="H.5.3.4.3" class="anchored" data-anchor-id="newey-west-estimator">
<span class="header-section-number">H.5.3.4.3</span> Newey-West Estimator</h5>
<p>The <a href="https://en.wikipedia.org/wiki/Newey%E2%80%93West_estimator">Newey-West estimator</a> is a method used to estimate the <a href="https://en.wikipedia.org/wiki/Covariance_matrix">covariance matrix</a> of the coefficients in a regression model when the residuals are autocorrelated.</p>
<p>Learn more about the Newey-West estimator in: <span class="citation" data-cites="newey1987">Newey &amp; West (<a href="references.html#ref-newey1987" role="doc-biblioref">1987</a>)</span> and <span class="citation" data-cites="newey1994">Newey &amp; West (<a href="references.html#ref-newey1994" role="doc-biblioref">1994</a>)</span>.</p>
<div class="cell">
<div class="sourceCode" id="cb47"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">sandwich</span><span class="fu">::</span><span class="fu"><a href="https://sandwich.R-Forge.R-project.org/reference/NeweyWest.html">NeweyWest</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt;                (Intercept)              age          sexMale</span></span>
<span><span class="co">#&gt; (Intercept) 372183.6934257 -425.57968425539 -3422.9016991396</span></span>
<span><span class="co">#&gt; age           -425.5796843   11.64040001590    12.6056297067</span></span>
<span><span class="co">#&gt; sexMale      -3422.9016991   12.60562970672  3925.6778488357</span></span>
<span><span class="co">#&gt; longitude     3147.4694189   -0.78729929084   -12.4252016878</span></span>
<span><span class="co">#&gt; ghi_month      -37.0109194    0.02467725042     0.2877105044</span></span>
<span><span class="co">#&gt; latitude       943.6824329    5.87074507634    19.1869193147</span></span>
<span><span class="co">#&gt;                    longitude        ghi_month       latitude</span></span>
<span><span class="co">#&gt; (Intercept) 3147.46941886501 -37.010919402627 943.6824328946</span></span>
<span><span class="co">#&gt; age           -0.78729929084   0.024677250422   5.8707450763</span></span>
<span><span class="co">#&gt; sexMale      -12.42520168783   0.287710504404  19.1869193147</span></span>
<span><span class="co">#&gt; longitude     84.41505502739   0.010143834520 -35.6361017284</span></span>
<span><span class="co">#&gt; ghi_month      0.01014383452   0.005812213491  -0.3212740041</span></span>
<span><span class="co">#&gt; latitude     -35.63610172839  -0.321274004106  55.5690572263</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section><section id="durbin-watson-test" class="level5" data-number="H.5.3.4.4"><h5 data-number="H.5.3.4.4" class="anchored" data-anchor-id="durbin-watson-test">
<span class="header-section-number">H.5.3.4.4</span> Durbin-Watson Test</h5>
<p>The <a href="https://en.wikipedia.org/wiki/Durbin%E2%80%93Watson_statistic">Durbin-Watson test</a> is a statistical test used to detect the presence of autocorrelation at lag <span class="math inline">\(1\)</span> in the residuals from a regression analysis. The test statistic ranges from <span class="math inline">\(0\)</span> to <span class="math inline">\(4\)</span>, with a value of <span class="math inline">\(2\)</span> indicating no autocorrelation. Values less than <span class="math inline">\(2\)</span> indicate positive autocorrelation, while values greater than <span class="math inline">\(2\)</span> indicate negative autocorrelation <span class="citation" data-cites="fox2016">(<a href="references.html#ref-fox2016" role="doc-biblioref">Fox, 2016</a>)</span>.</p>
<p>A common rule of thumb in the statistical community is that a Durbin-Watson statistic between <span class="math inline">\(1.5\)</span> and <span class="math inline">\(2.5\)</span> suggests little to no autocorrelation.</p>
<p>Learn more about the Durbin-Watson test in <span class="citation" data-cites="durbin1950">Durbin &amp; Watson (<a href="references.html#ref-durbin1950" role="doc-biblioref">1950</a>)</span>, <span class="citation" data-cites="durbin1951">Durbin &amp; Watson (<a href="references.html#ref-durbin1951" role="doc-biblioref">1951</a>)</span>, and <span class="citation" data-cites="durbin1971">Durbin &amp; Watson (<a href="references.html#ref-durbin1971" role="doc-biblioref">1971</a>)</span>.</p>
<p><span class="math display">\[
\begin{cases}
\text{H}_{0}: \text{Autocorrelation of the disturbances is 0} \\
\text{H}_{a}: \text{Autocorrelation of the disturbances is not equal to 0}
\end{cases}
\]</span></p>
<div class="cell">
<div class="sourceCode" id="cb48"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">car</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/car/man/durbinWatsonTest.html">durbinWatsonTest</a></span><span class="op">(</span><span class="va">fit_engine</span><span class="op">)</span></span>
<span><span class="co">#&gt;  lag Autocorrelation D-W Statistic p-value</span></span>
<span><span class="co">#&gt;    1    0.0364739387   1.927029923       0</span></span>
<span><span class="co">#&gt;  Alternative hypothesis: rho != 0</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section><section id="ljung-box-test" class="level5" data-number="H.5.3.4.5"><h5 data-number="H.5.3.4.5" class="anchored" data-anchor-id="ljung-box-test">
<span class="header-section-number">H.5.3.4.5</span> Ljung-Box Test</h5>
<p>The Ljung–Box test is a statistical test used to determine whether any autocorrelations within a time series are significantly different from zero. Rather than testing randomness at individual lags, it assesses the “overall” randomness across multiple lags.</p>
<p>Learn more about the <a href="https://en.wikipedia.org/wiki/Ljung%E2%80%93Box_test">Ljung-Box test</a> in <span class="citation" data-cites="box1970">Box &amp; Pierce (<a href="references.html#ref-box1970" role="doc-biblioref">1970</a>)</span> and <span class="citation" data-cites="ljung1978">Ljung &amp; Box (<a href="references.html#ref-ljung1978" role="doc-biblioref">1978</a>)</span>.</p>
<p><span class="math display">\[
\begin{cases}
\text{H}_{0}: \text{Residuals are independently distributed} \\
\text{H}_{a}: \text{Residuals are not independently distributed}
\end{cases}
\]</span></p>
<div class="cell">
<div class="sourceCode" id="cb49"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/residuals.html">residuals</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/box.test.html">Box.test</a></span><span class="op">(</span>type <span class="op">=</span> <span class="st">"Ljung-Box"</span>, lag <span class="op">=</span> <span class="fl">10</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;  Box-Ljung test</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; data:  stats::residuals(fit_engine)</span></span>
<span><span class="co">#&gt; X-squared = 1088.8889, df = 10, p-value &lt; 2.2204e-16</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section></section><section id="collinearitymulticollinearity" class="level4" data-number="H.5.3.5"><h4 data-number="H.5.3.5" class="anchored" data-anchor-id="collinearitymulticollinearity">
<span class="header-section-number">H.5.3.5</span> Collinearity/Multicollinearity</h4>
<p>No high degree of collinearity was observed among the independent variables.</p>
<section id="variance-inflation-factor-vif" class="level5" data-number="H.5.3.5.1"><h5 data-number="H.5.3.5.1" class="anchored" data-anchor-id="variance-inflation-factor-vif">
<span class="header-section-number">H.5.3.5.1</span> Variance Inflation Factor (VIF)</h5>
<p>The <a href="https://en.wikipedia.org/wiki/Variance_inflation_factor">Variance Inflation Factor (VIF)</a> indicates the effect of other independent variables on the standard error of a regression coefficient. The VIF is directly related to the tolerance value (<span class="math inline">\(\text{VIF}_{i} = 1/\text{TO}L\)</span>). High VIF values (larger than <span class="math inline">\(\sim 5\)</span> <span class="citation" data-cites="struck2024">(<a href="references.html#ref-struck2024" role="doc-biblioref">Struck, 2024</a>)</span>) suggest significant collinearity or multicollinearity among the independent variables <span class="citation" data-cites="hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019, p. 265</a>)</span>.</p>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb50"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">diag_sum_plots</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">fit_engine_2</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">performance</span><span class="fu">::</span><span class="fu"><a href="https://easystats.github.io/performance/reference/check_model.html">check_model</a></span><span class="op">(</span>panel <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html">plot</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">rutils</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/rutils/reference/shush.html">shush</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">diag_sum_plots</span><span class="op">$</span><span class="va">VIF</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span></span>
<span>    title <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>    subtitle <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/theme_get.html">theme_get</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/theme.html">theme</a></span><span class="op">(</span></span>
<span>    legend.position <span class="op">=</span> <span class="st">"right"</span>,</span>
<span>    axis.title <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_text</a></span><span class="op">(</span>size <span class="op">=</span> <span class="fl">11</span>, colour <span class="op">=</span> <span class="st">"black"</span><span class="op">)</span>,</span>
<span>    axis.text <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_text</a></span><span class="op">(</span>colour <span class="op">=</span> <span class="st">"black"</span><span class="op">)</span>,</span>
<span>    axis.text.y <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_text</a></span><span class="op">(</span>size <span class="op">=</span> <span class="fl">9</span><span class="op">)</span>,</span>
<span>    legend.text <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_text</a></span><span class="op">(</span>colour <span class="op">=</span> <span class="st">"black"</span><span class="op">)</span></span>
<span>  <span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-collinearity-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-collinearity-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.9: Variance Inflation Factors (VIF) for each predictor variable. VIFs below <span class="math inline">\(5\)</span> are considered acceptable. Between <span class="math inline">\(5\)</span> and <span class="math inline">\(10\)</span>, the variable should be examined. Above <span class="math inline">\(10\)</span>, the variable must considered highly collinear.
</figcaption><div aria-describedby="fig-sm-8-diag-collinearity-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-57-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb51"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">olsrr</span><span class="fu">::</span><span class="fu"><a href="https://olsrr.rsquaredacademy.com/reference/ols_coll_diag.html">ols_vif_tol</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-residual-diag-collinearity-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-residual-diag-collinearity-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.13: Variance Inflation Factors (VIF) and tolerance values for each predictor variable.
</figcaption><div aria-describedby="tbl-sm-8-residual-diag-collinearity-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["Variables"],"name":[1],"type":["chr"],"align":["left"]},{"label":["Tolerance"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["VIF"],"name":[3],"type":["dbl"],"align":["right"]}],"data":[{"1":"age","2":"0.9908076932","3":"1.009277589"},{"1":"sexMale","2":"0.9983384318","3":"1.001664334"},{"1":"longitude","2":"0.6218250083","3":"1.608169480"},{"1":"ghi_month","2":"0.5026178903","3":"1.989582980"},{"1":"latitude","2":"0.3797575199","3":"2.633259245"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb52"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine_2</span> <span class="op">|&gt;</span> <span class="fu">performance</span><span class="fu">::</span><span class="fu"><a href="https://easystats.github.io/performance/reference/check_collinearity.html">check_collinearity</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-residual-diag-collinearity-2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-residual-diag-collinearity-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.14: Variance Inflation Factors (VIF) and tolerance values for each predictor variable.
</figcaption><div aria-describedby="tbl-sm-8-residual-diag-collinearity-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["Term"],"name":[1],"type":["chr"],"align":["left"]},{"label":["VIF"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["VIF_CI_low"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["VIF_CI_high"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["SE_factor"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["Tolerance"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["Tolerance_CI_low"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["Tolerance_CI_high"],"name":[8],"type":["dbl"],"align":["right"]}],"data":[{"1":"age","2":"1.004741928","3":"1.000932713","4":"1.024108033","5":"1.002368160","6":"0.9952804518","7":"0.97645948276045","8":"0.9990681559"},{"1":"sex","2":"1.000411193","3":"1.000000000","4":"48780.942790588","5":"1.000205575","6":"0.9995889765","7":"0.00002049980879","8":"1.0000000000"},{"1":"longitude","2":"1.609900297","3":"1.593178065","4":"1.627093944","5":"1.268818465","6":"0.6211564789","7":"0.61459266299008","8":"0.6276762288"},{"1":"ghi_month","2":"2.257148205","3":"2.230589121","4":"2.284280499","5":"1.502380846","6":"0.4430369249","7":"0.43777460791973","8":"0.4483120584"},{"1":"latitude","2":"2.245636664","3":"2.219252313","4":"2.272591966","5":"1.498544849","6":"0.4453080127","7":"0.44002619688030","8":"0.4506022115"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
</section><section id="condition-index" class="level5" data-number="H.5.3.5.2"><h5 data-number="H.5.3.5.2" class="anchored" data-anchor-id="condition-index">
<span class="header-section-number">H.5.3.5.2</span> Condition Index</h5>
<p>The <a href="https://en.wikipedia.org/wiki/Condition_number">condition index</a> is a measure of multicollinearity in a regression model. It is based on the <a href="https://en.wikipedia.org/wiki/Eigenvalues_and_eigenvectors">eigenvalues</a> of the correlation matrix of the predictors. A condition index of 30 or higher is generally considered indicative of significant collinearity <span class="citation" data-cites="belsley2004">(<a href="references.html#ref-belsley2004" role="doc-biblioref">Belsley et al., 2004, pp. 112–114</a>)</span>.</p>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb53"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">olsrr</span><span class="fu">::</span><span class="fu"><a href="https://olsrr.rsquaredacademy.com/reference/ols_coll_diag.html">ols_eigen_cindex</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-residual-diag-collinearity-3" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-residual-diag-collinearity-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.15: Condition indexes and eigenvalues for each predictor variable.
</figcaption><div aria-describedby="tbl-sm-8-residual-diag-collinearity-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["Eigenvalue"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["Condition Index"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["intercept"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["age"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["sexMale"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["longitude"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["ghi_month"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["latitude"],"name":[8],"type":["dbl"],"align":["right"]}],"data":[{"1":"5.241146518875","2":"1.000000000","3":"0.00008246834539","4":"0.00259871829829","5":"0.01047507698331","6":"0.0001723051452","7":"0.0001997838300","8":"0.0009961702073"},{"1":"0.617519661706","2":"2.913317083","3":"0.00005790755190","4":"0.00192134529209","5":"0.98458854902131","6":"0.0001317581510","7":"0.0001278995042","8":"0.0010106832863"},{"1":"0.076810668378","2":"8.260424732","3":"0.00017442748409","4":"0.24444198984704","5":"0.00327418289266","6":"0.0006595599094","7":"0.0079870629925","8":"0.2395906496318"},{"1":"0.058386171293","2":"9.474540397","3":"0.00334869893405","4":"0.72644426937190","5":"0.00114056947760","6":"0.0036826485351","7":"0.0236683795534","8":"0.0345827665676"},{"1":"0.004506196937","2":"34.104215695","3":"0.00029335460849","4":"0.00007860515982","5":"0.00003370540569","6":"0.6750617081050","7":"0.3383895591675","8":"0.6422260602343"},{"1":"0.001630782810","2":"56.691126630","3":"0.99604314307607","4":"0.02451507203086","5":"0.00048791621943","6":"0.3202920201543","7":"0.6296273149524","8":"0.0815936700728"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
</section></section><section id="measures-of-influence" class="level4" data-number="H.5.3.6"><h4 data-number="H.5.3.6" class="anchored" data-anchor-id="measures-of-influence">
<span class="header-section-number">H.5.3.6</span> Measures of Influence</h4>
<p>In this section, I check several measures of influence that can be used to assess the impact of individual observations on the model estimates.</p>
<div class="callout callout-style-simple callout-tip">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-body-container">
<dl>
<dt>Leverage points</dt>
<dd>
Leverage is a measure of the distance between individual values of a predictor and other values of the predictor. In other words, a point with high leverage has an <span class="math inline">\(x\)</span>-value far away from the other <span class="math inline">\(x\)</span>-values. Points with high leverage have the potential to influence the model estimates <span class="citation" data-cites="struck2024 hair2019 nahhas2024">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019, p. 262</a>; <a href="references.html#ref-nahhas2024" role="doc-biblioref">Nahhas, 2024</a>; <a href="references.html#ref-struck2024" role="doc-biblioref">Struck, 2024</a>)</span>.
</dd>
</dl>
</div>
</div>
</div>
<div class="callout callout-style-simple callout-tip">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-body-container">
<dl>
<dt>Influence points</dt>
<dd>
Influence is a measure of how much an observation affects the model estimates. If an observation with large influence were removed from the dataset, we would expect a large change in the predictive equation <span class="citation" data-cites="struck2024 nahhas2024">(<a href="references.html#ref-nahhas2024" role="doc-biblioref">Nahhas, 2024</a>; <a href="references.html#ref-struck2024" role="doc-biblioref">Struck, 2024</a>)</span>.
</dd>
</dl>
</div>
</div>
</div>
<section id="standardized-residuals" class="level5" data-number="H.5.3.6.1"><h5 data-number="H.5.3.6.1" class="anchored" data-anchor-id="standardized-residuals">
<span class="header-section-number">H.5.3.6.1</span> Standardized Residuals</h5>
<p>Standardized residuals are a rescaling of the residual to a common basis by dividing each residual by the standard deviation of the residuals <span class="citation" data-cites="hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019, p. 264</a>)</span>.</p>
<div class="cell">
<div class="sourceCode" id="cb54"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">rstandard</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt;              1              2              3              4              5 </span></span>
<span><span class="co">#&gt;  1.14466248868  1.91162017161 -0.05544017932  0.15285619139  0.86926612171 </span></span>
<span><span class="co">#&gt;              6 </span></span>
<span><span class="co">#&gt;  0.29009860510</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb55"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/tibble.html">tibble</a></span><span class="op">(</span></span>
<span>  x <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/seq.html">seq_len</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html">nrow</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span><span class="op">)</span>,</span>
<span>  std <span class="op">=</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">rstandard</a></span><span class="op">(</span><span class="va">fit_engine</span><span class="op">)</span></span>
<span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html">ggplot</a></span><span class="op">(</span></span>
<span>    <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">x</span>, y <span class="op">=</span> <span class="va">std</span>, ymin <span class="op">=</span> <span class="fl">0</span>, ymax <span class="op">=</span> <span class="va">std</span><span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_linerange.html">geom_linerange</a></span><span class="op">(</span>color <span class="op">=</span> <span class="st">"black"</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="fl">2</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"grey"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="op">-</span><span class="fl">2</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"grey"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="fl">3</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="op">-</span><span class="fl">3</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/scale_continuous.html">scale_y_continuous</a></span><span class="op">(</span>breaks <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/seq.html">seq</a></span><span class="op">(</span><span class="op">-</span><span class="fl">3</span>, <span class="fl">3</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span></span>
<span>    x <span class="op">=</span> <span class="st">"Observation"</span>,</span>
<span>    y <span class="op">=</span> <span class="st">"Standardized residual"</span></span>
<span>  <span class="op">)</span></span>
<span><span class="co">#&gt; ! The color grey was not found in the `_brand.yml` file.</span></span>
<span><span class="co">#&gt; ! The color grey was not found in the `_brand.yml` file.</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-influence-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-influence-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.10: Standardized residuals for each observation.
</figcaption><div aria-describedby="fig-sm-8-diag-influence-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-62-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</section><section id="studentized-residuals" class="level5" data-number="H.5.3.6.2"><h5 data-number="H.5.3.6.2" class="anchored" data-anchor-id="studentized-residuals">
<span class="header-section-number">H.5.3.6.2</span> Studentized Residuals</h5>
<p><a href="https://en.wikipedia.org/wiki/Studentized_residual">Studentized residuals</a> are a commonly used variant of the standardized residual. It differs from other methods in how it calculates the standard deviation used in standardization. To minimize the effect of any observation on the standardization process, the standard deviation of the residual for observation <span class="math inline">\(i\)</span> is computed from regression estimates omitting the <span class="math inline">\(i\)</span>th observation in the calculation of the regression estimates <span class="citation" data-cites="hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019, p. 264</a>)</span>.</p>
<div class="cell">
<div class="sourceCode" id="cb56"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">rstudent</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt;              1              2              3              4              5 </span></span>
<span><span class="co">#&gt;  1.14466518662  1.91165871961 -0.05543975944  0.15285505730  0.86926450791 </span></span>
<span><span class="co">#&gt;              6 </span></span>
<span><span class="co">#&gt;  0.29009658674</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb57"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/tibble.html">tibble</a></span><span class="op">(</span></span>
<span>  x <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/seq.html">seq_len</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html">nrow</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span><span class="op">)</span>,</span>
<span>  std <span class="op">=</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">rstudent</a></span><span class="op">(</span><span class="va">fit_engine</span><span class="op">)</span></span>
<span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html">ggplot</a></span><span class="op">(</span></span>
<span>    <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">x</span>, y <span class="op">=</span> <span class="va">std</span>, ymin <span class="op">=</span> <span class="fl">0</span>, ymax <span class="op">=</span> <span class="va">std</span><span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_linerange.html">geom_linerange</a></span><span class="op">(</span>color <span class="op">=</span> <span class="st">"black"</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="fl">2</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"grey"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="op">-</span><span class="fl">2</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"grey"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="fl">3</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="op">-</span><span class="fl">3</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/scale_continuous.html">scale_y_continuous</a></span><span class="op">(</span>breaks <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/seq.html">seq</a></span><span class="op">(</span><span class="op">-</span><span class="fl">3</span>, <span class="fl">3</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span></span>
<span>    x <span class="op">=</span> <span class="st">"Observation"</span>,</span>
<span>    y <span class="op">=</span> <span class="st">"Studentized residual"</span></span>
<span>  <span class="op">)</span></span>
<span><span class="co">#&gt; ! The color grey was not found in the `_brand.yml` file.</span></span>
<span><span class="co">#&gt; ! The color grey was not found in the `_brand.yml` file.</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-influence-2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-influence-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.11: Studentized residuals for each observation.
</figcaption><div aria-describedby="fig-sm-8-diag-influence-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-65-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb58"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">broom</span><span class="fu">::</span><span class="fu"><a href="https://generics.r-lib.org/reference/augment.html">augment</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html">mutate</a></span><span class="op">(</span></span>
<span>    std <span class="op">=</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">rstudent</a></span><span class="op">(</span><span class="va">fit_engine</span><span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html">ggplot</a></span><span class="op">(</span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span><span class="va">.pred</span>, <span class="va">std</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_point.html">geom_point</a></span><span class="op">(</span>color <span class="op">=</span> <span class="st">"black"</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="fl">2</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"grey"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="op">-</span><span class="fl">2</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"grey"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="fl">3</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_abline.html">geom_hline</a></span><span class="op">(</span>yintercept <span class="op">=</span> <span class="op">-</span><span class="fl">3</span>, color <span class="op">=</span> <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/scale_continuous.html">scale_y_continuous</a></span><span class="op">(</span>breaks <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/seq.html">seq</a></span><span class="op">(</span><span class="op">-</span><span class="fl">3</span>, <span class="fl">3</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span></span>
<span>    x <span class="op">=</span> <span class="st">"Predicted value"</span>,</span>
<span>    y <span class="op">=</span> <span class="st">"Studentized residual"</span></span>
<span>  <span class="op">)</span></span>
<span><span class="co">#&gt; ! The color grey was not found in the `_brand.yml` file.</span></span>
<span><span class="co">#&gt; ! The color grey was not found in the `_brand.yml` file.</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-influence-3" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-influence-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.12: Relation between studentized residuals and fitted values.
</figcaption><div aria-describedby="fig-sm-8-diag-influence-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-66-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb59"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plot</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">fit_engine</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">olsrr</span><span class="fu">::</span><span class="fu"><a href="https://olsrr.rsquaredacademy.com/reference/ols_plot_resid_lev.html">ols_plot_resid_lev</a></span><span class="op">(</span>threshold <span class="op">=</span> <span class="fl">2</span>, print_plot <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span></span>
<span><span class="va">plot</span><span class="op">$</span><span class="va">plot</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span></span>
<span>    title <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>    y <span class="op">=</span> <span class="st">"Studentized residual"</span></span>
<span>  <span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-influence-4" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-influence-4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.13: Relation between studentized residuals and their leverage points.
</figcaption><div aria-describedby="fig-sm-8-diag-influence-4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-67-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</section><section id="hat-values" class="level5" data-number="H.5.3.6.3"><h5 data-number="H.5.3.6.3" class="anchored" data-anchor-id="hat-values">
<span class="header-section-number">H.5.3.6.3</span> Hat Values</h5>
<p>The hat value indicates how distinct an observation’s predictor values are from those of other observations. Observations with high hat values have high leverage and may be, though not necessarily, influential.</p>
<p>There is no fixed threshold for what constitutes a “large” hat value; instead, the focus must be on observations with hat values significantly higher than the rest <span class="citation" data-cites="nahhas2024 hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019, p. 261</a>; <a href="references.html#ref-nahhas2024" role="doc-biblioref">Nahhas, 2024</a>)</span>.</p>
<div class="cell">
<div class="sourceCode" id="cb60"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">hatvalues</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt;                1                2                3                4 </span></span>
<span><span class="co">#&gt; 0.00011154803870 0.00005304235715 0.00001890244325 0.00028116437047 </span></span>
<span><span class="co">#&gt;                5                6 </span></span>
<span><span class="co">#&gt; 0.00011264323625 0.00010994216720</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb61"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/tibble.html">tibble</a></span><span class="op">(</span></span>
<span>  x <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/seq.html">seq_len</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html">nrow</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span><span class="op">)</span>,</span>
<span>  hat <span class="op">=</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">hatvalues</a></span><span class="op">(</span><span class="va">fit_engine</span><span class="op">)</span></span>
<span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html">ggplot</a></span><span class="op">(</span></span>
<span>    <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">x</span>, y <span class="op">=</span> <span class="va">hat</span>, ymin <span class="op">=</span> <span class="fl">0</span>, ymax <span class="op">=</span> <span class="va">hat</span><span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_linerange.html">geom_linerange</a></span><span class="op">(</span>color <span class="op">=</span> <span class="st">"black"</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span></span>
<span>    x <span class="op">=</span> <span class="st">"Observation"</span>,</span>
<span>    y <span class="op">=</span> <span class="st">"Hat value"</span></span>
<span>  <span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-influence-5" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-influence-5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.14: Hat values for each observation.
</figcaption><div aria-describedby="fig-sm-8-diag-influence-5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-72-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</section><section id="cooks-distance" class="level5" data-number="H.5.3.6.4"><h5 data-number="H.5.3.6.4" class="anchored" data-anchor-id="cooks-distance">
<span class="header-section-number">H.5.3.6.4</span> Cook’s Distance</h5>
<p>The <a href="https://en.wikipedia.org/wiki/Cook%27s_distance">Cook’s D</a> measures each observation’s influence on the model’s fitted values. It is considered one of the most representative metrics for assessing overall influence <span class="citation" data-cites="hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019</a>)</span>.</p>
<p>A common practice is to flag observations with a Cook’s distance of <span class="math inline">\(1.0\)</span> or greater. However, a threshold of <span class="math inline">\(4 / (n - k - 1)\)</span>, where <span class="math inline">\(n\)</span> is the sample size and <span class="math inline">\(k\)</span> is the number of independent variables, is suggested as a more conservative measure in small samples or for use with larger datasets <span class="citation" data-cites="hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019</a>)</span>.</p>
<p>Learn more about Cook’s D in <span class="citation" data-cites="cook1977">Cook (<a href="references.html#ref-cook1977" role="doc-biblioref">1977</a>)</span> and <span class="citation" data-cites="cook1979">Cook (<a href="references.html#ref-cook1979" role="doc-biblioref">1979</a>)</span>.</p>
<div class="cell">
<div class="sourceCode" id="cb62"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">cooks.distance</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt;                    1                    2                    3 </span></span>
<span><span class="co">#&gt; 0.000024362061633972 0.000032307087715705 0.000000009683317111 </span></span>
<span><span class="co">#&gt;                    4                    5                    6 </span></span>
<span><span class="co">#&gt; 0.000001095209567771 0.000014187579235604 0.000001542240395315</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb63"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plot</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">fit_engine</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">olsrr</span><span class="fu">::</span><span class="fu"><a href="https://olsrr.rsquaredacademy.com/reference/ols_plot_cooksd_bar.html">ols_plot_cooksd_bar</a></span><span class="op">(</span>type <span class="op">=</span> <span class="fl">2</span>, print_plot <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># The following procedure changes the plot aesthetics.</span></span>
<span><span class="va">q</span> <span class="op">&lt;-</span> <span class="va">plot</span><span class="op">$</span><span class="va">plot</span> <span class="op">+</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>title <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">q</span> <span class="op">&lt;-</span> <span class="va">q</span> <span class="op">|&gt;</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot_build.html">ggplot_build</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="va">q</span><span class="op">$</span><span class="va">data</span><span class="op">[[</span><span class="fl">5</span><span class="op">]</span><span class="op">]</span><span class="op">$</span><span class="va">label</span> <span class="op">&lt;-</span> <span class="st">""</span></span>
<span></span>
<span><span class="va">q</span> <span class="op">|&gt;</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot_gtable.html">ggplot_gtable</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu">ggplotify</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/ggplotify/man/as.ggplot.html">as.ggplot</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-influence-6" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-influence-6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.15: Cook’s distance for each observation along with a threshold line at <span class="math inline">\(4 / (n - k - 1)\)</span>.
</figcaption><div aria-describedby="fig-sm-8-diag-influence-6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-75-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb64"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">diag_sum_plots</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">fit_engine_2</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">performance</span><span class="fu">::</span><span class="fu"><a href="https://easystats.github.io/performance/reference/check_model.html">check_model</a></span><span class="op">(</span></span>
<span>    panel <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>    colors <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"blue"</span>, <span class="st">"black"</span>, <span class="st">"black"</span><span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html">plot</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">rutils</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/rutils/reference/shush.html">shush</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">plot</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">diag_sum_plots</span><span class="op">$</span><span class="va">OUTLIERS</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span></span>
<span>    title <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>    subtitle <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>    x <span class="op">=</span> <span class="st">"Leverage"</span>,</span>
<span>    y <span class="op">=</span> <span class="st">"Studentized residuals"</span></span>
<span>  <span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/theme_get.html">theme_get</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/theme.html">theme</a></span><span class="op">(</span></span>
<span>    legend.position <span class="op">=</span> <span class="st">"right"</span>,</span>
<span>    axis.title <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_text</a></span><span class="op">(</span>size <span class="op">=</span> <span class="fl">11</span>, colour <span class="op">=</span> <span class="st">"black"</span><span class="op">)</span>,</span>
<span>    axis.text <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_text</a></span><span class="op">(</span>colour <span class="op">=</span> <span class="st">"gray25"</span><span class="op">)</span>,</span>
<span>    axis.text.y <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_text</a></span><span class="op">(</span>size <span class="op">=</span> <span class="fl">9</span><span class="op">)</span></span>
<span>  <span class="op">)</span></span>
<span></span>
<span><span class="va">plot</span> <span class="op">&lt;-</span> <span class="va">plot</span> <span class="op">|&gt;</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot_build.html">ggplot_build</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># The following procedure changes the plot aesthetics.</span></span>
<span><span class="kw">for</span> <span class="op">(</span><span class="va">i</span> <span class="kw">in</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="fl">1</span><span class="op">:</span><span class="fl">9</span><span class="op">)</span><span class="op">)</span> <span class="op">{</span></span>
<span>  <span class="co"># "#1b6ca8" "#3aaf85"</span></span>
<span>  <span class="va">plot</span><span class="op">$</span><span class="va">data</span><span class="op">[[</span><span class="va">i</span><span class="op">]</span><span class="op">]</span><span class="op">$</span><span class="va">colour</span> <span class="op">&lt;-</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/case_when.html">case_when</a></span><span class="op">(</span></span>
<span>    <span class="va">plot</span><span class="op">$</span><span class="va">data</span><span class="op">[[</span><span class="va">i</span><span class="op">]</span><span class="op">]</span><span class="op">$</span><span class="va">colour</span> <span class="op">==</span> <span class="st">"blue"</span> <span class="op">~</span></span>
<span>      <span class="fu"><a href="https://rdrr.io/r/base/ifelse.html">ifelse</a></span><span class="op">(</span><span class="va">i</span> <span class="op">==</span> <span class="fl">4</span>, <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"grey"</span><span class="op">)</span>, <span class="fu">brandr</span><span class="fu">::</span><span class="fu"><a href="https://danielvartan.github.io/brandr/reference/get_brand_color.html">get_brand_color</a></span><span class="op">(</span><span class="st">"orange"</span><span class="op">)</span><span class="op">)</span>,</span>
<span>    <span class="va">plot</span><span class="op">$</span><span class="va">data</span><span class="op">[[</span><span class="va">i</span><span class="op">]</span><span class="op">]</span><span class="op">$</span><span class="va">colour</span> <span class="op">==</span> <span class="st">"#1b6ca8"</span> <span class="op">~</span> <span class="st">"black"</span>,</span>
<span>    <span class="va">plot</span><span class="op">$</span><span class="va">data</span><span class="op">[[</span><span class="va">i</span><span class="op">]</span><span class="op">]</span><span class="op">$</span><span class="va">colour</span> <span class="op">==</span> <span class="st">"darkgray"</span> <span class="op">~</span> <span class="st">"black"</span>,</span>
<span>    <span class="cn">TRUE</span> <span class="op">~</span> <span class="va">plot</span><span class="op">$</span><span class="va">data</span><span class="op">[[</span><span class="va">i</span><span class="op">]</span><span class="op">]</span><span class="op">$</span><span class="va">colour</span></span>
<span>  <span class="op">)</span></span>
<span><span class="op">}</span></span>
<span><span class="co">#&gt; ! The color grey was not found in the `_brand.yml` file.</span></span>
<span></span>
<span><span class="va">plot</span> <span class="op">|&gt;</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot_gtable.html">ggplot_gtable</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu">ggplotify</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/ggplotify/man/as.ggplot.html">as.ggplot</a></span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-influence-7" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-influence-7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.16: Relation between studentized residuals and their leverage points. The blue line represents the <a href="https://en.wikipedia.org/wiki/Cook%27s_distance">Cook’s distance</a>. Any points outside the contour lines are influential observations.
</figcaption><div aria-describedby="fig-sm-8-diag-influence-7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-76-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</section><section id="influence-on-prediction-dffits" class="level5" data-number="H.5.3.6.5"><h5 data-number="H.5.3.6.5" class="anchored" data-anchor-id="influence-on-prediction-dffits">
<span class="header-section-number">H.5.3.6.5</span> Influence on Prediction (DFFITS)</h5>
<p><a href="https://en.wikipedia.org/wiki/DFFITS">DFFITS</a> (Difference in Fits) is a standardized measure of how much the prediction for a given observation would change if it were deleted from the model. Each observation’s DFFITS is standardized by the standard deviation of fit at that point <span class="citation" data-cites="struck2024">(<a href="references.html#ref-struck2024" role="doc-biblioref">Struck, 2024</a>)</span>.</p>
<p>The best rule of thumb is to classify as influential any standardized values that exceed <span class="math inline">\(2 \sqrt{(p / n)}\)</span>, where <span class="math inline">\(p\)</span> is the number of independent variables <span class="math inline">\(+1\)</span> and <span class="math inline">\(n\)</span> is the sample size <span class="citation" data-cites="hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019, p. 261</a>)</span>.</p>
<p>Learn more about DDFITS in <span class="citation" data-cites="welsch1977">Welsch &amp; Kuh (<a href="references.html#ref-welsch1977" role="doc-biblioref">1977</a>)</span> and <span class="citation" data-cites="belsley2004">Belsley et al. (<a href="references.html#ref-belsley2004" role="doc-biblioref">2004</a>)</span>.</p>
<div class="cell">
<div class="sourceCode" id="cb65"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">dffits</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt;                1                2                3                4 </span></span>
<span><span class="co">#&gt;  0.0120902050791  0.0139230149062 -0.0002410373884  0.0025634273733 </span></span>
<span><span class="co">#&gt;                5                6 </span></span>
<span><span class="co">#&gt;  0.0092263296788  0.0030419259705</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb66"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plot</span> <span class="op">&lt;-</span> <span class="va">fit_engine</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">olsrr</span><span class="fu">::</span><span class="fu"><a href="https://olsrr.rsquaredacademy.com/reference/ols_plot_dffits.html">ols_plot_dffits</a></span><span class="op">(</span>print_plot <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span></span>
<span><span class="va">plot</span><span class="op">$</span><span class="va">plot</span> <span class="op">+</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>title <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-influence-8" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-influence-8-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.17: Standardized DFFITS (difference in fits) for each observation.
</figcaption><div aria-describedby="fig-sm-8-diag-influence-8-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-80-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</section><section id="influence-on-parameter-estimates-dfbetas" class="level5" data-number="H.5.3.6.6"><h5 data-number="H.5.3.6.6" class="anchored" data-anchor-id="influence-on-parameter-estimates-dfbetas">
<span class="header-section-number">H.5.3.6.6</span> Influence on Parameter Estimates (DFBETAS)</h5>
<p><a href="https://en.wikipedia.org/wiki/Influential_observation#:~:text=measures%20of%20influence">DFBETAS</a> are a measure of the change in a regression coefficient when an observation is omitted from the regression analysis. <strong>The value of the DFBETA is in terms of the coefficient itself</strong> <span class="citation" data-cites="hair2019">(<a href="references.html#ref-hair2019" role="doc-biblioref">Hair, 2019, p. 261</a>)</span>.</p>
<p>A cutoff for what is considered a large DFBETAS value is <span class="math inline">\(2 / \sqrt{n}\)</span>, where <span class="math inline">\(n\)</span> is the number of observations. <span class="citation" data-cites="struck2024">(<a href="references.html#ref-struck2024" role="doc-biblioref">Struck, 2024</a>)</span>.</p>
<p>Learn more about DFBETAS in <span class="citation" data-cites="welsch1977">Welsch &amp; Kuh (<a href="references.html#ref-welsch1977" role="doc-biblioref">1977</a>)</span> and <span class="citation" data-cites="belsley2004">Belsley et al. (<a href="references.html#ref-belsley2004" role="doc-biblioref">2004</a>)</span>.</p>
<div class="cell">
<div class="sourceCode" id="cb67"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">fit_engine</span> <span class="op">|&gt;</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">dfbeta</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt;      (Intercept)              age        sexMale        longitude</span></span>
<span><span class="co">#&gt; 1 -2.54519742992 -0.0104159688425 0.163678462807 -0.0321085605036</span></span>
<span><span class="co">#&gt; 2  1.95612939429 -0.0085954301495 0.243826899038 -0.0043306908770</span></span>
<span><span class="co">#&gt; 3 -0.03511379247  0.0002244927435 0.004780743688 -0.0001207820623</span></span>
<span><span class="co">#&gt; 4  0.42848993936 -0.0023788915306 0.034404027397  0.0072552713006</span></span>
<span><span class="co">#&gt; 5 -0.89702626365 -0.0083558333639 0.137754956560 -0.0112439580996</span></span>
<span><span class="co">#&gt; 6 -0.72956384607 -0.0016201389879 0.045784527016 -0.0081332881171</span></span>
<span><span class="co">#&gt;            ghi_month          latitude</span></span>
<span><span class="co">#&gt; 1  0.000239855929853 -0.01292882652087</span></span>
<span><span class="co">#&gt; 2 -0.000332969674368  0.00409279034750</span></span>
<span><span class="co">#&gt; 3  0.000003449967890  0.00008355341512</span></span>
<span><span class="co">#&gt; 4 -0.000003282864713 -0.00004161199299</span></span>
<span><span class="co">#&gt; 5  0.000086221225732 -0.01403962584853</span></span>
<span><span class="co">#&gt; 6  0.000066624264282 -0.00419501257285</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist">
<li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-2-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-1" role="tab" aria-controls="tabset-2-1" aria-selected="true">Intercept</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-2" role="tab" aria-controls="tabset-2-2" aria-selected="false">age</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-3" role="tab" aria-controls="tabset-2-3" aria-selected="false">sexMale</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-4" role="tab" aria-controls="tabset-2-4" aria-selected="false">longitude</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-5-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-5" role="tab" aria-controls="tabset-2-5" aria-selected="false">ghi_month</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-6-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-6" role="tab" aria-controls="tabset-2-6" aria-selected="false">latitude</a></li>
</ul>
<div class="tab-content">
<div id="tabset-2-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-2-1-tab">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb68"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plots</span><span class="op">$</span><span class="va">plots</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span> <span class="op">+</span></span>
<span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>title <span class="op">=</span> <span class="st">"Intercept coefficient"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-6-fm-coef-dfbetas-intercept" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-6-fm-coef-dfbetas-intercept-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.16: Standardized DFBETAS values for each observation concerning the <code>Intercept</code> coefficient.
</figcaption><div aria-describedby="tbl-sm-6-fm-coef-dfbetas-intercept-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-84-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
<div id="tabset-2-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-2-tab">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb69"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plots</span><span class="op">$</span><span class="va">plots</span><span class="op">[[</span><span class="fl">2</span><span class="op">]</span><span class="op">]</span> <span class="op">+</span></span>
<span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>title <span class="op">=</span> <span class="st">"age coefficient"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-6-fm-coef-dfbetas-age" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-6-fm-coef-dfbetas-age-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.17: Standardized DFBETAS values for each observation concerning the <code>age</code> coefficient.
</figcaption><div aria-describedby="tbl-sm-6-fm-coef-dfbetas-age-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-85-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
<div id="tabset-2-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-3-tab">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb70"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plots</span><span class="op">$</span><span class="va">plots</span><span class="op">[[</span><span class="fl">3</span><span class="op">]</span><span class="op">]</span> <span class="op">+</span></span>
<span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>title <span class="op">=</span> <span class="st">"sexMale coefficient"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-6-fm-coef-dfbetas-sexmale" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-6-fm-coef-dfbetas-sexmale-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.18: Standardized DFBETAS values for each observation concerning the <code>sexMale</code> coefficient.
</figcaption><div aria-describedby="tbl-sm-6-fm-coef-dfbetas-sexmale-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-86-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
<div id="tabset-2-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-4-tab">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb71"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plots</span><span class="op">$</span><span class="va">plots</span><span class="op">[[</span><span class="fl">4</span><span class="op">]</span><span class="op">]</span> <span class="op">+</span></span>
<span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>title <span class="op">=</span> <span class="st">"longitude coefficient"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-6-fm-coef-dfbetas-longitude" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-6-fm-coef-dfbetas-longitude-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.19: Standardized DFBETAS values for each observation concerning the <code>longitude</code> coefficient.
</figcaption><div aria-describedby="tbl-sm-6-fm-coef-dfbetas-longitude-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-87-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
<div id="tabset-2-5" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-5-tab">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb72"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plots</span><span class="op">$</span><span class="va">plots</span><span class="op">[[</span><span class="fl">5</span><span class="op">]</span><span class="op">]</span> <span class="op">+</span></span>
<span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>title <span class="op">=</span> <span class="st">"ghi_month coefficient"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-6-fm-coef-dfbetas-ghi-month" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-6-fm-coef-dfbetas-ghi-month-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.20: Standardized DFBETAS values for each observation concerning the <code>ghi_month</code> coefficient.
</figcaption><div aria-describedby="tbl-sm-6-fm-coef-dfbetas-ghi-month-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-88-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
<div id="tabset-2-6" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-6-tab">
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb73"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plots</span><span class="op">$</span><span class="va">plots</span><span class="op">[[</span><span class="fl">6</span><span class="op">]</span><span class="op">]</span> <span class="op">+</span></span>
<span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>title <span class="op">=</span> <span class="st">"latitude coefficient"</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-6-fm-coef-dfbetas-latitude" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-6-fm-coef-dfbetas-latitude-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.21: Standardized DFBETAS values for each observation concerning the <code>latitude</code> coefficient.
</figcaption><div aria-describedby="tbl-sm-6-fm-coef-dfbetas-latitude-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-89-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
</div>
</div>
</section><section id="hadis-measure" class="level5" data-number="H.5.3.6.7"><h5 data-number="H.5.3.6.7" class="anchored" data-anchor-id="hadis-measure">
<span class="header-section-number">H.5.3.6.7</span> Hadi’s Measure</h5>
<p>Hadi’s measure of influence is based on the idea that influential observations can occur in either the response variable, the predictors, or both.</p>
<p>Learn more about Hadi’s measure in <span class="citation" data-cites="chatterjee2012">Chatterjee &amp; Hadi (<a href="references.html#ref-chatterjee2012" role="doc-biblioref">2012</a>)</span>.</p>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb74"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plot</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">fit_engine</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">olsrr</span><span class="fu">::</span><span class="fu"><a href="https://olsrr.rsquaredacademy.com/reference/ols_plot_hadi.html">ols_plot_hadi</a></span><span class="op">(</span>print_plot <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span></span>
<span><span class="va">plot</span> <span class="op">+</span></span>
<span>  <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span></span>
<span>    title <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>    y <span class="op">=</span> <span class="st">"Hadi's measure"</span></span>
<span>  <span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-influence-12" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-influence-12-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.18: Hadi’s influence measure for each observation.
</figcaption><div aria-describedby="fig-sm-8-diag-influence-12-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-90-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb75"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">plot</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">fit_engine</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">olsrr</span><span class="fu">::</span><span class="fu"><a href="https://olsrr.rsquaredacademy.com/reference/ols_plot_resid_pot.html">ols_plot_resid_pot</a></span><span class="op">(</span>print_plot <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span></span>
<span><span class="va">plot</span> <span class="op">+</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>title <span class="op">=</span> <span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html">element_blank</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="fig-sm-8-diag-influence-13" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-fig" id="fig-sm-8-diag-influence-13-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;H.19: Potential-residual plot classifying unusual observations as high-leverage points, outliers, or a combination of both.
</figcaption><div aria-describedby="fig-sm-8-diag-influence-13-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure"><p><img src="supplementary-material-8_files/figure-html/unnamed-chunk-91-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</div>
</figure>
</div>
</section></section></section></section><section id="hypothesis-testing" class="level2" data-number="H.6"><h2 data-number="H.6" class="anchored" data-anchor-id="hypothesis-testing">
<span class="header-section-number">H.6</span> Hypothesis Testing</h2>
<dl>
<dt><strong>Hypothesis</strong></dt>
<dd>
<em>Latitude is associated with chronotype distributions</em>, with populations closer to the equator exhibiting, on average, a shorter or more morning-oriented circadian phenotype compared to those residing near the poles.
</dd>
</dl>
<p><span class="math display">\[
\begin{cases}
\text{H}_{0}: \Delta \ \text{Adjusted} \ \text{R}^{2} &lt; \text{MES} \\
\text{H}_{a}: \Delta \ \text{Adjusted} \ \text{R}^{2} \geq \text{MES}
\end{cases}
\]</span></p>
<section id="f-test" class="level3" data-number="H.6.1"><h3 data-number="H.6.1" class="anchored" data-anchor-id="f-test">
<span class="header-section-number">H.6.1</span> F-Test</h3>
<p>The results indicate that the F-test is significant (<span class="math inline">\(\alpha &lt; 0.05\)</span>), meaning that the model including the latitude variable differs from the model without it.</p>
<p><span class="math display">\[
\text{F} = \cfrac{\text{R}^{2}_{f} - \text{R}^{2}_{r} / (k_{f} - k_{R})}{(1 - \text{R}^{2}_{f}) / (\text{N} - k_{f} - 1)}
\]</span></p>
<div class="cell">
<div class="sourceCode" id="cb76"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">f_test</span> <span class="op">&lt;-</span> <span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/anova.html">anova</a></span><span class="op">(</span><span class="va">fit_engine_restricted</span>, <span class="va">fit_engine_full</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/print.html">print</a></span><span class="op">(</span><span class="va">f_test</span><span class="op">)</span></span>
<span><span class="co">#&gt; Analysis of Variance Table</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Model 1: msf_sc ~ age + sex + longitude + ghi_month</span></span>
<span><span class="co">#&gt; Model 2: msf_sc ~ age + sex + longitude + ghi_month + latitude</span></span>
<span><span class="co">#&gt;   Res.Df           RSS Df Sum of Sq        F          Pr(&gt;F)    </span></span>
<span><span class="co">#&gt; 1  65818 1355086391524                                          </span></span>
<span><span class="co">#&gt; 2  65817 1354318625681  1 767765843 37.31178 0.0000000010124 ***</span></span>
<span><span class="co">#&gt; ---</span></span>
<span><span class="co">#&gt; Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode" id="cb77"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">n</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/nrow.html">nrow</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span></span>
<span><span class="va">k_res</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/length.html">length</a></span><span class="op">(</span><span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/coef.html">coefficients</a></span><span class="op">(</span><span class="va">fit_engine_restricted</span><span class="op">)</span><span class="op">)</span> <span class="op">-</span> <span class="fl">1</span></span>
<span><span class="va">k_full</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/length.html">length</a></span><span class="op">(</span><span class="fu">stats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/stats/coef.html">coefficients</a></span><span class="op">(</span><span class="va">fit_engine_full</span><span class="op">)</span><span class="op">)</span> <span class="op">-</span> <span class="fl">1</span></span>
<span></span>
<span><span class="va">r_squared_restricted</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span><span class="op">(</span><span class="va">fit_engine_restricted</span><span class="op">)</span><span class="op">$</span><span class="va">r.squared</span></span>
<span><span class="va">r_squared_full</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span><span class="op">(</span><span class="va">fit_engine_full</span><span class="op">)</span><span class="op">$</span><span class="va">r.squared</span></span>
<span></span>
<span><span class="op">(</span><span class="op">(</span><span class="va">r_squared_full</span> <span class="op">-</span> <span class="va">r_squared_restricted</span><span class="op">)</span> <span class="op">/</span></span>
<span>    <span class="op">(</span><span class="va">k_full</span> <span class="op">-</span> <span class="va">k_res</span><span class="op">)</span><span class="op">)</span> <span class="op">/</span> <span class="op">(</span><span class="op">(</span><span class="fl">1</span> <span class="op">-</span> <span class="va">r_squared_full</span><span class="op">)</span> <span class="op">/</span> <span class="op">(</span><span class="va">n</span>  <span class="op">-</span> <span class="va">k_full</span> <span class="op">-</span> <span class="fl">1</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 37.31178436</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section><section id="effect-size" class="level3" data-number="H.6.2"><h3 data-number="H.6.2" class="anchored" data-anchor-id="effect-size">
<span class="header-section-number">H.6.2</span> Effect Size</h3>
<p>The results show that the <span class="math inline">\(\Delta \ \text{Adjusted} \ \text{R}^{2}\)</span> value is below the Minimal Effect Size (MES) threshold (<span class="math inline">\(\text{R}^{2} \approx 0.01961\)</span>), indicating that adding latitude does not meaningfully improve the model’s fit.</p>
<p><span class="math display">\[
\text{Cohen's } f^2 = \cfrac{\text{R}^{2}_{f} - \text{R}^{2}_{r}}{1 - \text{R}^{2}_{f}} = \cfrac{\Delta \text{R}^{2}}{1 - \text{R}^{2}_{f}}
\]</span></p>
<p><span class="math display">\[
\text{MES} = \text{Cohen's } f^2 \text{small threshold} = 0.02 \\
\]</span></p>
<p><span class="math display">\[
0.02 = \cfrac{\text{R}^{2}}{1 - \text{R}^{2}} \quad \text{or} \quad \text{R}^{2} = \cfrac{0.02}{1.02} \eqsim 0.01960784
\]</span></p>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb78"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">adj_r_squared_restricted</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">fit_engine_restricted</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">rutils</span><span class="fu">:::</span><span class="fu">summarize_r2</span><span class="op">(</span></span>
<span>    n <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/nrow.html">nrow</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span>,</span>
<span>    k <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/length.html">length</a></span><span class="op">(</span><span class="va">fit_engine_restricted</span><span class="op">$</span><span class="va">coefficients</span><span class="op">)</span> <span class="op">-</span> <span class="fl">1</span>,</span>
<span>    ci_level <span class="op">=</span> <span class="fl">0.95</span></span>
<span>  <span class="op">)</span></span>
<span></span>
<span><span class="va">adj_r_squared_restricted</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-hypothesis-test-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-hypothesis-test-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.22: Confidence interval for the adjusted R-squared of the <strong>restricted model</strong>. LCL correspond to the lower limit, and UCL to the upper limit.
</figcaption><div aria-describedby="tbl-sm-8-hypothesis-test-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["name"],"name":[1],"type":["chr"],"align":["left"]},{"label":["value"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["interpretation"],"name":[3],"type":["chr"],"align":["left"]},{"label":["rule"],"name":[4],"type":["chr"],"align":["left"]}],"data":[{"1":"R2","2":"0.085166284722","3":"weak","4":"cohen1988"},{"1":"SE","2":"0.002081010564","3":"NA","4":"NA"},{"1":"Lower CI","2":"0.081087578964","3":"weak","4":"cohen1988"},{"1":"Upper CI","2":"0.089244990479","3":"weak","4":"cohen1988"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb79"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">adj_r_squared_full</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">fit_engine_full</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">rutils</span><span class="fu">:::</span><span class="fu">summarize_r2</span><span class="op">(</span></span>
<span>    n <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/nrow.html">nrow</a></span><span class="op">(</span><span class="va">data</span><span class="op">)</span>,</span>
<span>    k <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/length.html">length</a></span><span class="op">(</span><span class="va">fit_engine_full</span><span class="op">$</span><span class="va">coefficients</span><span class="op">)</span> <span class="op">-</span> <span class="fl">1</span>,</span>
<span>    ci_level <span class="op">=</span> <span class="fl">0.95</span></span>
<span>  <span class="op">)</span></span>
<span></span>
<span><span class="va">adj_r_squared_full</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-hypothesis-test-2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-hypothesis-test-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.23: Confidence interval for the adjusted R-squared of the <strong>full model</strong>. LCL correspond to the lower limit, and UCL to the upper limit.
</figcaption><div aria-describedby="tbl-sm-8-hypothesis-test-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["name"],"name":[1],"type":["chr"],"align":["left"]},{"label":["value"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["interpretation"],"name":[3],"type":["chr"],"align":["left"]},{"label":["rule"],"name":[4],"type":["chr"],"align":["left"]}],"data":[{"1":"R2","2":"0.085684611880","3":"weak","4":"cohen1988"},{"1":"SE","2":"0.002086119198","3":"NA","4":"NA"},{"1":"Lower CI","2":"0.081595893385","3":"weak","4":"cohen1988"},{"1":"Upper CI","2":"0.089773330375","3":"weak","4":"cohen1988"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb80"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/tibble.html">tibble</a></span><span class="op">(</span></span>
<span>  name <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"adj_r_squared_res"</span>,</span>
<span>    <span class="st">"adj_r_squared_full"</span>,</span>
<span>    <span class="st">"diff"</span></span>
<span>  <span class="op">)</span>,</span>
<span>  value <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span></span>
<span>    <span class="va">adj_r_squared_restricted</span><span class="op">$</span><span class="va">value</span><span class="op">[</span><span class="fl">1</span><span class="op">]</span>,</span>
<span>    <span class="va">adj_r_squared_full</span><span class="op">$</span><span class="va">value</span><span class="op">[</span><span class="fl">1</span><span class="op">]</span>,</span>
<span>    <span class="va">adj_r_squared_full</span><span class="op">$</span><span class="va">value</span><span class="op">[</span><span class="fl">1</span><span class="op">]</span> <span class="op">-</span> <span class="va">adj_r_squared_restricted</span><span class="op">$</span><span class="va">value</span><span class="op">[</span><span class="fl">1</span><span class="op">]</span></span>
<span>  <span class="op">)</span></span>
<span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-hypothesis-test-3" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-hypothesis-test-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.24: Comparison between the coefficients of determination (<span class="math inline">\(\text{R}^2\)</span>) of the restricted and full models.
</figcaption><div aria-describedby="tbl-sm-8-hypothesis-test-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["name"],"name":[1],"type":["chr"],"align":["left"]},{"label":["value"],"name":[2],"type":["dbl"],"align":["right"]}],"data":[{"1":"adj_r_squared_res","2":"0.0851662847216"},{"1":"adj_r_squared_full","2":"0.0856846118803"},{"1":"diff","2":"0.0005183271587"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
<details class="code-fold"><summary>Code</summary><div class="sourceCode" id="cb81"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="va">effect_size</span> <span class="op">&lt;-</span> <span class="fu">rutils</span><span class="fu">:::</span><span class="fu">cohens_f_squared_summary</span><span class="op">(</span></span>
<span>  base_r_squared <span class="op">=</span> <span class="va">adj_r_squared_restricted</span>,</span>
<span>  new_r_squared <span class="op">=</span> <span class="va">adj_r_squared_full</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="va">effect_size</span> <span class="op">|&gt;</span> <span class="fu">rutils</span><span class="fu">:::</span><span class="fu">list_as_tibble</span><span class="op">(</span><span class="op">)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details><div id="tbl-sm-8-hypothesis-test-4" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure"><figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sm-8-hypothesis-test-4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;H.25: Effect size between the restricted and full models based on Cohen’s <span class="math inline">\(f^2\)</span>.
</figcaption><div aria-describedby="tbl-sm-8-hypothesis-test-4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell">
<div class="cell-output-display">
<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["name"],"name":[1],"type":["chr"],"align":["left"]},{"label":["value"],"name":[2],"type":["chr"],"align":["left"]}],"data":[{"1":"f_squared","2":"0.000566901930553285"},{"1":"lower_ci_limit","2":"0"},{"1":"upper_ci_limit","2":"0.00954240487679154"},{"1":"effect_size","2":"Negligible"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</figure>
</div>
</section></section><section id="conclusion" class="level2" data-number="H.7"><h2 data-number="H.7" class="anchored" data-anchor-id="conclusion">
<span class="header-section-number">H.7</span> Conclusion</h2>
<p>Based on the hypothesis test results and the methodology outlined in Supplementary Material <a href="supplementary-material-2.html" class="quarto-xref"><span>B</span></a>, <strong>we fail to reject the null hypothesis</strong>. Latitude does not meaningfully contribute to explaining the variance in chronotype.</p>


<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list" style="display: none">
<div id="ref-anderson1962" class="csl-entry" role="listitem">
Anderson, T. W. (1962). On the distribution of the two-sample <span class="nocase">Cramér-von Mises</span> criterion. <em>The Annals of Mathematical Statistics</em>, <em>33</em>(3), 1148–1159. <a href="https://doi.org/10.1214/aoms/1177704477">https://doi.org/10.1214/aoms/1177704477</a>
</div>
<div id="ref-anderson1952" class="csl-entry" role="listitem">
Anderson, T. W., &amp; Darling, D. A. (1952). Asymptotic theory of certain "goodness of fit" criteria based on stochastic processes. <em>The Annals of Mathematical Statistics</em>, <em>23</em>(2), 193–212. <a href="https://www.jstor.org/stable/2236446">https://www.jstor.org/stable/2236446</a>
</div>
<div id="ref-anderson1954" class="csl-entry" role="listitem">
Anderson, T. W., &amp; Darling, D. A. (1954). A test of goodness of fit. <em>Journal of the American Statistical Association</em>, <em>49</em>(268), 765–769. <a href="https://doi.org/10.1080/01621459.1954.10501232">https://doi.org/10.1080/01621459.1954.10501232</a>
</div>
<div id="ref-belsley2004" class="csl-entry" role="listitem">
Belsley, D. A., Kuh, E., &amp; Welsch, R. E. (2004). <em>Regression diagnostics: Identifying influential data and sources of collinearity</em> (Print ISBN: 9780471058564<br>
Online ISBN: 9780471725152). John Wiley &amp; Sons. <a href="https://doi.org/10.1002/0471725153">https://doi.org/10.1002/0471725153</a>
</div>
<div id="ref-bera1981" class="csl-entry" role="listitem">
Bera, A. K., &amp; Jarque, C. M. (1981). Efficient tests for normality, homoscedasticity and serial independence of regression residuals: <span>Monte Carlo</span> evidence. <em>Economics Letters</em>, <em>7</em>(4), 313–318. <a href="https://doi.org/10.1016/0165-1765(81)90035-5">https://doi.org/10.1016/0165-1765(81)90035-5</a>
</div>
<div id="ref-bonett2002" class="csl-entry" role="listitem">
Bonett, D. G., &amp; Seier, E. (2002). A test of normality with high uniform power. <em>Computational Statistics &amp; Data Analysis</em>, <em>40</em>(3), 435–445. <a href="https://doi.org/10.1016/S0167-9473(02)00074-9">https://doi.org/10.1016/S0167-9473(02)00074-9</a>
</div>
<div id="ref-box1970" class="csl-entry" role="listitem">
Box, G. E. P., &amp; Pierce, D. A. (1970). Distribution of residual autocorrelations in autoregressive-integrated moving average time series models. <em>Journal of the American Statistical Association</em>, <em>65</em>(332), 1509–1526. <a href="https://doi.org/10.1080/01621459.1970.10481180">https://doi.org/10.1080/01621459.1970.10481180</a>
</div>
<div id="ref-breusch1979" class="csl-entry" role="listitem">
Breusch, T. S., &amp; Pagan, A. R. (1979). A simple test for heteroscedasticity and random coefficient variation. <em>Econometrica</em>, <em>47</em>(5), 1287–1294. <a href="https://doi.org/10.2307/1911963">https://doi.org/10.2307/1911963</a>
</div>
<div id="ref-chatterjee2012" class="csl-entry" role="listitem">
Chatterjee, S., &amp; Hadi, A. S. (2012). <em>Regression analysis by example</em> (5th ed.). Wiley.
</div>
<div id="ref-cook1977" class="csl-entry" role="listitem">
Cook, R. D. (1977). Detection of influential observation in linear regression. <em>Technometrics</em>, <em>19</em>(1), 15–18. <a href="https://doi.org/10.1080/00401706.1977.10489493">https://doi.org/10.1080/00401706.1977.10489493</a>
</div>
<div id="ref-cook1979" class="csl-entry" role="listitem">
Cook, R. D. (1979). Influential observations in linear regression. <em>Journal of the American Statistical Association</em>, <em>74</em>(365), 169–174. <a href="https://doi.org/10.1080/01621459.1979.10481634">https://doi.org/10.1080/01621459.1979.10481634</a>
</div>
<div id="ref-cramer1928" class="csl-entry" role="listitem">
Cramér, H. (1928). On the composition of elementary errors: <span>First</span> paper: <span>Mathematical</span> deductions. <em>Scandinavian Actuarial Journal</em>, <em>1928</em>(1), 13–74. <a href="https://doi.org/10.1080/03461238.1928.10416862">https://doi.org/10.1080/03461238.1928.10416862</a>
</div>
<div id="ref-dagostino1971" class="csl-entry" role="listitem">
D’Agostino, R. B. (1971). An omnibus test of normality for moderate and large size samples. <em>Biometrika</em>, <em>58</em>(2), 341–348. <a href="https://doi.org/10.1093/biomet/58.2.341">https://doi.org/10.1093/biomet/58.2.341</a>
</div>
<div id="ref-dagostino1990" class="csl-entry" role="listitem">
D’Agostino, R. B., &amp; Belanger, A. (1990). A suggestion for using powerful and informative tests of normality. <em>The American Statistician</em>, <em>44</em>(4), 316–321. <a href="https://doi.org/10.2307/2684359">https://doi.org/10.2307/2684359</a>
</div>
<div id="ref-dagostino1973" class="csl-entry" role="listitem">
D’Agostino, R. B., &amp; Pearson, E. S. (1973). Tests for departure from normality. <span>Empirical</span> results for the distributions of b2 and √b1. <em>Biometrika</em>, <em>60</em>(3), 613–622. <a href="https://doi.org/10.1093/biomet/60.3.613">https://doi.org/10.1093/biomet/60.3.613</a>
</div>
<div id="ref-dallal1986" class="csl-entry" role="listitem">
Dallal, G. E., &amp; Wilkinson, L. (1986). An analytic approximation to the distribution of <span>Lilliefors</span>’s test statistic for normality. <em>The American Statistician</em>, <em>40</em>(4), 294–296. <a href="https://doi.org/10.1080/00031305.1986.10475419">https://doi.org/10.1080/00031305.1986.10475419</a>
</div>
<div id="ref-degroot2012a" class="csl-entry" role="listitem">
DeGroot, M. H., &amp; Schervish, M. J. (2012). <em>Probability and statistics</em> (OCLC: ocn502674206) (4th ed.). Addison-Wesley.
</div>
<div id="ref-durbin1950" class="csl-entry" role="listitem">
Durbin, J., &amp; Watson, G. S. (1950). Testing for serial correlation in least squares regression. <span>I</span>. <em>Biometrika</em>, <em>37</em>(3–4), 409–428. <a href="https://doi.org/10.1093/biomet/37.3-4.409">https://doi.org/10.1093/biomet/37.3-4.409</a>
</div>
<div id="ref-durbin1951" class="csl-entry" role="listitem">
Durbin, J., &amp; Watson, G. S. (1951). Testing for serial correlation in least squares regression. <span>II</span>. <em>Biometrika</em>, <em>38</em>(1–2), 159–178. <a href="https://doi.org/10.1093/biomet/38.1-2.159">https://doi.org/10.1093/biomet/38.1-2.159</a>
</div>
<div id="ref-durbin1971" class="csl-entry" role="listitem">
Durbin, J., &amp; Watson, G. S. (1971). Testing for serial correlation in least squares regression. <span>III</span>. <em>Biometrika</em>, <em>58</em>(1), 1–19. <a href="https://doi.org/10.1093/biomet/58.1.1">https://doi.org/10.1093/biomet/58.1.1</a>
</div>
<div id="ref-fox2016" class="csl-entry" role="listitem">
Fox, J. (2016). <em>Applied regression analysis and generalized linear models</em> (3rd ed.). Sage.
</div>
<div id="ref-greener2020" class="csl-entry" role="listitem">
Greener, R. (2020, August 4). <em>Stop testing for normality</em>. Medium. <a href="https://towardsdatascience.com/stop-testing-for-normality-dba96bb73f90">https://towardsdatascience.com/stop-testing-for-normality-dba96bb73f90</a>
</div>
<div id="ref-hair2019" class="csl-entry" role="listitem">
Hair, J. F. (2019). <em>Multivariate data analysis</em> (8th ed.). Cengage.
</div>
<div id="ref-jarque1980" class="csl-entry" role="listitem">
Jarque, C. M., &amp; Bera, A. K. (1980). Efficient tests for normality, homoscedasticity and serial independence of regression residuals. <em>Economics Letters</em>, <em>6</em>(3), 255–259. <a href="https://doi.org/10.1016/0165-1765(80)90024-5">https://doi.org/10.1016/0165-1765(80)90024-5</a>
</div>
<div id="ref-jarque1987" class="csl-entry" role="listitem">
Jarque, C. M., &amp; Bera, A. K. (1987). A test for normality of observations and regression residuals. <em>International Statistical Review</em>, <em>55</em>(2), 163–172. <a href="https://doi.org/10.2307/1403192">https://doi.org/10.2307/1403192</a>
</div>
<div id="ref-koenker1981" class="csl-entry" role="listitem">
Koenker, R. (1981). A note on studentizing a test for heteroscedasticity. <em>Journal of Econometrics</em>, <em>17</em>(1), 107–112. <a href="https://doi.org/10.1016/0304-4076(81)90062-2">https://doi.org/10.1016/0304-4076(81)90062-2</a>
</div>
<div id="ref-kolmogorov1933" class="csl-entry" role="listitem">
Kolmogorov, A. (1933). Sulla determinazione empirica di una legge di distribuzione. <em>Giornale dell’Istituto Italiano degli Attuari</em>, <em>4</em>.
</div>
<div id="ref-kozak2018" class="csl-entry" role="listitem">
Kozak, M., &amp; Piepho, H.-P. (2018). What’s normal anyway? <span>Residual</span> plots are more telling than significance tests when checking <span>ANOVA</span> assumptions. <em>Journal of Agronomy and Crop Science</em>, <em>204</em>(1), 86–98. <a href="https://doi.org/10.1111/jac.12220">https://doi.org/10.1111/jac.12220</a>
</div>
<div id="ref-lilliefors1967" class="csl-entry" role="listitem">
Lilliefors, H. W. (1967). On the <span>Kolmogorov-Smirnov</span> test for normality with mean and variance unknown. <em>Journal of the American Statistical Association</em>, <em>62</em>(318), 399–402. <a href="https://doi.org/10.1080/01621459.1967.10482916">https://doi.org/10.1080/01621459.1967.10482916</a>
</div>
<div id="ref-ljung1978" class="csl-entry" role="listitem">
Ljung, G. M., &amp; Box, G. E. P. (1978). On a measure of lack of fit in time series models. <em>Biometrika</em>, <em>65</em>(2), 297–303. <a href="https://doi.org/10.1093/biomet/65.2.297">https://doi.org/10.1093/biomet/65.2.297</a>
</div>
<div id="ref-massey1951" class="csl-entry" role="listitem">
Massey, F. J. (1951). The <span>Kolmogorov-Smirnov</span> test for goodness of fit. <em>Journal of the American Statistical Association</em>, <em>46</em>(253), 68–78. <a href="https://doi.org/10.1080/01621459.1951.10500769">https://doi.org/10.1080/01621459.1951.10500769</a>
</div>
<div id="ref-nahhas2024" class="csl-entry" role="listitem">
Nahhas, R. W. (2024). <em>Introduction to regression methods for public health using <span>R</span></em>. <a href="https://www.bookdown.org/rwnahhas/RMPH/">https://www.bookdown.org/rwnahhas/RMPH/</a>
</div>
<div id="ref-newey1987" class="csl-entry" role="listitem">
Newey, W. K., &amp; West, K. D. (1987). A simple, positive semi-definite, heteroskedasticity and autocorrelation consistent covariance matrix. <em>Econometrica</em>, <em>55</em>(3), 703–708. <a href="https://doi.org/10.2307/1913610">https://doi.org/10.2307/1913610</a>
</div>
<div id="ref-newey1994" class="csl-entry" role="listitem">
Newey, W. K., &amp; West, K. D. (1994). Automatic lag selection in covariance matrix estimation. <em>The Review of Economic Studies</em>, <em>61</em>(4), 631–653. <a href="https://doi.org/10.2307/2297912">https://doi.org/10.2307/2297912</a>
</div>
<div id="ref-pearson1900" class="csl-entry" role="listitem">
Pearson, K. (1900). X. <span>On</span> the criterion that a given system of deviations from the probable in the case of a correlated system of variables is such that it can be reasonably supposed to have arisen from random sampling. <em>The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science</em>, <em>50</em>(302), 157–175. <a href="https://doi.org/10.1080/14786440009463897">https://doi.org/10.1080/14786440009463897</a>
</div>
<div id="ref-ramsey1969" class="csl-entry" role="listitem">
Ramsey, J. B. (1969). Tests for specification errors in classical linear least-squares regression analysis. <em>Journal of the Royal Statistical Society. Series B (Methodological)</em>, <em>31</em>(2), 350–371. <a href="https://doi.org/10.1111/j.2517-6161.1969.tb00796.x">https://doi.org/10.1111/j.2517-6161.1969.tb00796.x</a>
</div>
<div id="ref-schucany2006" class="csl-entry" role="listitem">
Schucany, W. R., &amp; Ng, H. K. T. (2006). Preliminary goodness-of-fit tests for normality do not validate the one-sample <span>Student</span> t. <em>Communications in Statistics - Theory and Methods</em>, <em>35</em>(12), 2275–2286. <a href="https://doi.org/10.1080/03610920600853308">https://doi.org/10.1080/03610920600853308</a>
</div>
<div id="ref-shapiro1972" class="csl-entry" role="listitem">
Shapiro, S. S., &amp; Francia, R. S. (1972). An approximate analysis of variance test for normality. <em>Journal of the American Statistical Association</em>, <em>67</em>(337), 215–216. <a href="https://doi.org/10.1080/01621459.1972.10481232">https://doi.org/10.1080/01621459.1972.10481232</a>
</div>
<div id="ref-shapiro1965" class="csl-entry" role="listitem">
Shapiro, S. S., &amp; Wilk, M. B. (1965). An analysis of variance test for normality (complete samples)†. <em>Biometrika</em>, <em>52</em>(3–4), 591–611. <a href="https://doi.org/10.1093/biomet/52.3-4.591">https://doi.org/10.1093/biomet/52.3-4.591</a>
</div>
<div id="ref-shatz2024" class="csl-entry" role="listitem">
Shatz, I. (2024). Assumption-checking rather than (just) testing: <span>The</span> importance of visualization and effect size in statistical diagnostics. <em>Behavior Research Methods</em>, <em>56</em>(2), 826–845. <a href="https://doi.org/10.3758/s13428-023-02072-x">https://doi.org/10.3758/s13428-023-02072-x</a>
</div>
<div id="ref-smirnov1948" class="csl-entry" role="listitem">
Smirnov, N. (1948). Table for estimating the goodness of fit of empirical distributions. <em>Annals of Mathematical Statistics</em>, <em>19</em>, 279–281.
</div>
<div id="ref-struck2024" class="csl-entry" role="listitem">
Struck, J. (2024). <em>Regression <span>Diagnostics</span> with <span>R</span></em>. University of Wisconsin-Madison. <a href="https://sscc.wisc.edu/sscc/pubs/RegDiag-R/">https://sscc.wisc.edu/sscc/pubs/RegDiag-R/</a>
</div>
<div id="ref-thode2002" class="csl-entry" role="listitem">
Thode, H. C. (2002). <em>Testing for normality</em>. Marcel Dekker.
</div>
<div id="ref-welsch1977" class="csl-entry" role="listitem">
Welsch, R., &amp; Kuh, E. (1977). <em>Linear regression diagnostics</em> (Working Paper 0173; p. 44). National Bureau of Economic Research. <a href="https://doi.org/10.3386/w0173">https://doi.org/10.3386/w0173</a>
</div>
</div>
</section></main><!-- /main --><script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp("https:\/\/danielvartan\.github\.io\/mastersthesis\/");
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script><nav class="page-navigation"><div class="nav-page nav-page-previous">
      <a href="../qmd/supplementary-material-7.html" class="pagination-link" aria-label="Hypothesis Test A">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">G</span>&nbsp; <span class="chapter-title">Hypothesis Test A</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../qmd/supplementary-material-9.html" class="pagination-link" aria-label="Comparative Assessment of QualoCEP and Google Geocoding API Performance">
        <span class="nav-page-text"><span class="chapter-number">I</span>&nbsp; <span class="chapter-title">Comparative Assessment of QualoCEP and Google Geocoding API Performance</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



<footer class="footer"><div class="nav-footer"><div class="nav-footer-center"><div class="toc-actions d-sm-block d-md-none"><ul><li><a href="https://github.com/danielvartan/mastersthesis/edit/main/qmd/supplementary-material-8.qmd" class="toc-action"><i class="bi bi-github"></i>Edit this page</a></li><li><a href="https://github.com/danielvartan/mastersthesis/issues/" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></div></div></footer></body></html>